---
format: docx
#format:
#   latex: 
#      cite-method: natbib
include-in-header:
  text: |
    \usepackage{blkarray}
editor: visual
editor_options:
  markdown:
    wrap: sentence
bibliography: translocation.bib
csl: pnas.csl
---

## Title: Reintroduction of resistant frogs facilitates landscape-scale recovery in the presence of a lethal fungal disease

### Authors:

Roland A. Knapp^a,b,^\*, Mark Q. Wilber^c,1^, Allison Q. Byrne^d,e,1^, Maxwell B. Joseph^f,g^, Thomas C. Smith^a,b^, Andrew P. Rothstein^d,e,h^, Robert L. Grasso^i^, Erica Bree Rosenblum^d,e^

### Affiliations:

^a^Sierra Nevada Aquatic Research Laboratory, University of California, Mammoth Lakes, CA, 93546

^b^Earth Research Institute, University of California, Santa Barbara, CA, 93106-3060

^c^School of Natural Resources, University of Tennessee Institute of Agriculture, Knoxville, TN, 37996

^d^Department of Environmental Science, Policy, and Management, University of California - Berkeley, Berkeley, CA, 94720-3114

^e^Museum of Vertebrate Zoology, University of California - Berkeley, Berkeley, CA, 94720-3160

^f^Earth Lab, University of Colorado, Boulder, CO, 80303

^g^Current affiliation: Planet, San Francisco, CA, 94107

^h^Current affiliation: Ginkgo Bioworks, Boston, MA, 02210

^i^Resources Management and Science, Yosemite National Park, El Portal, CA, 95318

\*Corresponding author: Sierra Nevada Aquatic Research Laboratory, 1016 Mount Morrison Road, Mammoth Lakes, CA 93546, phone: 760.647.0034, email: roland.knapp\@ucsb.edu

^1^These authors contributed equally to this work.

**Classification:** Biological Sciences - Ecology

**Key words:** amphibians, disease, evolution, reintroduction, population recovery

**Author contributions**: All authors designed research; R.A.K., M.Q.W., A.Q.B., M.B.J., T.C.S., A.P.R., and R.L.G. performed research; M.B.J. contributed new analytic tools; R.A.K., M.Q.W., A.Q.B., A.P.R., and E.B.R. analyzed data; and R.A.K., M.Q.W., A.Q.B., M.B.J., T.C.S., and E.B.R. wrote paper.

**Competing interests:** This project was supported by grants from the National Park Service (to R.A.K.), Yosemite Conservancy (to R.A.K.), and National Science Foundation (EF-0723563, to C. Briggs; DEB-1557190, to C. Briggs; DEB-2133401, to M.Q.W; and DBI-2120084, to C. Richards-Zawacki).
The authors declare no additional competing interests.

### Abstract

Vast alteration of the biosphere by humans is causing a sixth mass extinction, driven in part by an increase in infectious diseases.
The emergence of the lethal fungal pathogen (*Batrachochytrium dendrobatidis*; "Bd") has devastated global amphibian biodiversity, with hundreds of species experiencing declines or extinctions.
Lacking any broadly applicable methods to reverse these impacts, the future of many amphibians appears grim.
The once-common mountain yellow-legged (MYL) frog is emblematic of amphibians threatened by Bd.
Although most MYL frog populations are extirpated following disease outbreaks, some persist and eventually recover.
Frogs in these recovering populations have increased resistance against Bd infection, consistent with evolution of resistant genotypes.
We conducted a 15-year landscape-scale reintroduction study and show that frogs collected from recovering populations and reintroduced to vacant habitats can reestablish populations despite the presence of Bd.
In addition, the likelihood of establishment is influenced by site, population, and frog attributes.
Results from viability modeling suggest that many reintroduced populations have a low probability of extinction over 50 years.
To better understand the role of evolution in frog resistance, we compared the genomes of MYL frogs from Bd-naive and recovering populations.
We found numerous differences between these categories, including in immune function loci associated with disease resistance, consistent with evolutionary changes in response to Bd exposure.
These results provide a rare example of how reintroduction of resistant individuals can allow the landscape-scale recovery of disease-impacted species, and have broad implications for the many taxa worldwide that are threatened with extinction by novel pathogens.

### Significance

Understanding how species persist despite accelerating global change is critical for the conservation of biodiversity.
We used reintroductions of disease-resistant individuals in an effort to recover a once-common frog species driven to near-extinction by a novel disease that has decimated amphibians worldwide.
Reintroductions allowed establishment of viable populations in the presence of disease.
In addition, resistance may be at least partially the result of natural selection at specific immune function genes.
The evolution of resistance and reintroduction of resistant individuals could play an important role in biodiversity conservation in our rapidly changing world.

### Introduction

Human activities are increasingly impacting global biodiversity [@ceballos2015], with important implications for ecosystem resilience and human welfare [@naeem2009].
One consequence of human alteration of the biosphere is an increase in emerging infectious diseases [@jones2008; @fisher2012].
Such diseases pose a severe threat to wildlife populations [@daszak2000], and have caused dramatic declines and extinctions in a wide range of taxa, including echinoderms, mammals, birds, and amphibians [@hewson2014; @samuel2015; @scheele2019; @cunningham2021].
Amphibians are experiencing particularly devastating impacts of disease due to the recent emergence and global spread of the highly virulent amphibian chytrid fungus, *Batrachochytrium dendrobatidis* (Bd) [@luedtke2023; @scheele2019].
By one estimate, hundreds of species have experienced Bd-caused declines, and numerous susceptible taxa are extinct in the wild [@scheele2019].
These impacts to global biodiversity may be unprecedented, and highlight the importance of understanding mechanisms of species persistence in the presence of emerging diseases [@russell2020].

Evidence of natural recovery in the many Bd-impacted amphibian populations is surprisingly rare [for notable exceptions, see @scheele2017; @voyles2018; @knapp2016], suggesting that disease-caused declines will be difficult to reverse.
This apparent low resilience to disease effects may be due to the limited ability of many amphibians to develop Bd resistance and/or tolerance, which in turn, could also lessen the effectiveness of potential Bd mitigation strategies.
Following pathogen arrival in a host population, resistance (ability to limit pathogen burden) and tolerance (ability to limit the harm caused by a particular burden) are key mechanisms to reduce disease impacts [@raberg2009] and facilitate population persistence and recovery [@brannelly2021].
Host immunity and evolution both play important roles in the development of resistance and tolerance, and an enhanced understanding of these processes is important for developing effective strategies to mitigate disease impacts in the wild [@garner2016; @woodhams2011].
However, several aspects of the amphibian-Bd system present difficult obstacles for the development of mitigation efforts, including (i) the general inability of amphibians to mount an effective immune response against Bd infection [@rosenblum2012; @fites2013; @grogan2018a], and (ii) the apparent rarity of evolution of more resistant/tolerant genotypes [but see @savage2016; @grogan2018b].
These factors suggest that reintroduction of amphibians into sites to reestablish populations extirpated by Bd will often result not in population recovery, but instead in the rapid reinfection and mortality of the introduced animals and/or their progeny [@hammond2021; @knapp2022; @stockwell2008; @scheele2021].
If true, the future of many amphibian species threatened by Bd appears bleak.

The mountain yellow-legged (MYL) frog, composed of the sister species *Rana muscosa* and *Rana sierrae* [@vredenburg2007], is emblematic of the global declines of amphibians caused by Bd [@scheele2019].
Once the most common amphibian in the high elevation portion of California's Sierra Nevada mountains [USA, @grinnell1924], during the past century this frog has disappeared from more than 90% of its historical range [@vredenburg2007].
Due to the severity of its decline and the increasing probability of extinction, both species are now listed as "endangered" under the U.S.
Endangered Species Act.
In the Sierra Nevada, this decline was initiated by the introduction of non-native trout into the extensive historically-fishless region [@bradford1989; @knapp2000] starting in the late 1800s.
The arrival of Bd in the mid-1900s and its subsequent spread [@vredenburg2019] caused additional large-scale population extirpations [@vredenburg2010; @rachowicz2006].
These Bd-caused declines are fundamentally different from the fish-caused declines because fish eradication is feasible [@knapp1998] and results in the rapid recovery of frog populations [@knapp2007; @vredenburg2004].
In contrast, Bd appears to persist in habitats even in the absence of amphibian hosts [@walker2007], and therefore represents a long-term alteration of invaded ecosystems that amphibians will need to overcome to reestablish populations.

Despite the catastrophic impact of Bd on MYL frogs, wherein most Bd-naive populations are extirpated following Bd arrival [@vredenburg2010], some populations have persisted after epizootics [during which Bd infection intensity on frogs is very high, @briggs2010] and are now recovering (@fig-recovery-model) [@knapp2016].
Frogs in these recovering populations show reduced susceptibility to Bd infection [@knapp2016], with infection intensity ("load") on adults consistently in the low-to-moderate range [@briggs2010; @knapp2011; @joseph2018].
This reduced susceptibility is evident even under controlled laboratory conditions [@knapp2016], indicative of host resistance against Bd infection and not simply an effect of factors external to individual frogs (e.g., environmental conditions).
In addition to frogs from recovering populations having higher resistance to Bd infection than those from naive populations, they could also have higher tolerance, but no data are currently available to evaluate this possibility.
Therefore, we focus on resistance throughout this paper.
The observed resistance of MYL frogs could be the result of several mechanisms that are not mutually exclusive, including natural selection for more resistant genotypes [@savage2016; @grogan2018b], acquired immunity [@grogan2018a], and/or inherent between-population differences that pre-date Bd exposure.
The possible evolution of MYL frog resistance and subsequent population recovery is consistent with that expected under "evolutionary rescue", whereby rapid evolutionary change increases the frequency of adaptive alleles and restores positive population growth [@carlson2014; @searle2020].
This intriguing possibility also suggests an opportunity to expand recovery beyond the spatial scale possible under natural recovery by using resistant frogs from recovering populations in reintroductions to vacant habitats (@fig-recovery-model) [@joseph2018; @mendelson2019].

In the current study, we had three primary objectives.
First, determine whether the reintroduction of resistant MYL frogs, obtained from populations recovering from Bd-caused declines, allows the reestablishment of populations despite ongoing disease (@fig-recovery-model).
We accomplished this using a 15-year large-scale reintroduction study.
Second, extend our inferences of population recovery beyond the temporal bounds of our reintroduction study to estimate the probability of persistence for the reintroduced populations over a multi-decadal period (@fig-recovery-model).
We did this using a stage-structured matrix model.
Third, given the importance of resistance for frog survival, population establishment, and long-term viability (this study), determine whether MYL frogs in recovering populations show genomic patterns consistent with selection for resistance (@fig-recovery-model).
We achieved this with a comparative study using exome capture methods applied to naive and recovering populations.

### Results

#### Frog population recovery

```{r}
#| include: false
#| code-fold: true

library(readr)
library(dplyr)
library(magrittr)
library(stringr)
library(lubridate)
d1 <- read_csv(here::here("data", "clean", "frog_translocation_final.csv"))
d2 <- read_csv(here::here("data", "clean", "frog_counts_table.csv"))
d3 <- read_csv(here::here("data", "clean", "frog_captures_table.csv"))

transno_min <- d1 %>% 
  distinct(site_id, date) %>% 
  group_by(site_id) %>% 
  count() %>% 
  ungroup() %>% 
  summarize(min(n))

transno_max <- d1 %>% 
  distinct(site_id, date) %>% 
  group_by(site_id) %>% 
  count() %>% 
  ungroup() %>% 
  summarize(max(n))

transfrogs_max <- d1 %>% 
  group_by(site_id, date) %>% 
  count() %>% 
  ungroup() %>% 
  summarize(max(n))

transfrogs_min <- d1 %>% 
  group_by(site_id, date) %>% 
  count() %>% 
  ungroup() %>% 
  summarize(min(n))

transfrogs_median <- d1 %>% 
  group_by(site_id, date) %>% 
  count() %>% 
  ungroup() %>% 
  summarize(median(n))

bdload_min <- d1 %>% 
  filter(!is.na(bd_load)) %>% 
  group_by(site_id) %>% 
  mutate(bdload_ave = median(log10(bd_load + 1))) %>% 
  ungroup() %>% 
  summarize(min(bdload_ave))

bdload_max <- d1 %>% 
  filter(!is.na(bd_load)) %>% 
  group_by(site_id) %>% 
  mutate(bdload_ave = median(log10(bd_load + 1))) %>% 
  ungroup() %>% 
  summarize(max(bdload_ave))

bdprev_max <- d1 %>% 
  filter(!is.na(bd_load)) %>% 
  mutate(bdload01 = if_else(bd_load > 0, 1, 0)) %>% 
  group_by(site_id) %>% 
  summarize(prev = sum(bdload01)/n()) %>% 
  ungroup() %>% select(prev) %>% 
  slice_max(prev, n = 1, with_ties = FALSE)

bdprev_min <- d1 %>% 
  filter(!is.na(bd_load)) %>% 
  mutate(bdload01 = if_else(bd_load > 0, 1, 0)) %>% 
  group_by(site_id) %>% 
  summarize(prev = sum(bdload01)/n()) %>% 
  ungroup() %>% select(prev) %>% 
  slice_min(prev, n = 1, with_ties = FALSE)

files <- fs::dir_ls(here::here("data", "raw", "cmr-analysis", "survey"))  
surveys <- read_csv(files, id = "path") %>% 
  filter(survey_date < "2023-01-01") %>% 
  select(-path)
rm(files)

surveys_yr_median <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr") %>% 
  distinct(site_id, primary_period, .keep_all = TRUE) %>% 
  group_by(site_id) %>% 
  count(yr) %>% ungroup() %>% 
  summarize(median(n))

surveys_yr_max <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr") %>% 
  distinct(site_id, primary_period, .keep_all = TRUE) %>% 
  group_by(site_id) %>% 
  count(yr) %>% ungroup() %>% 
  summarize(max(n))

surveys_yr_min <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr") %>% 
  distinct(site_id, primary_period, .keep_all = TRUE) %>% 
  group_by(site_id) %>% 
  count(yr) %>% ungroup() %>% 
  summarize(min(n))

robust_year_median <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr" & secondary_period > 1) %>% 
  group_by(site_id) %>% 
  summarize(max_2period = max(secondary_period)) %>% ungroup() %>% 
  summarize(median(max_2period))

robust_year_min <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr" & secondary_period > 1) %>% 
  group_by(site_id) %>% 
  summarize(max_2period = max(secondary_period)) %>% ungroup() %>% 
  summarize(min(max_2period))

robust_year_max <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr" & secondary_period > 1) %>% 
  group_by(site_id) %>% 
  summarize(max_2period = max(secondary_period)) %>% ungroup() %>% 
  summarize(max(max_2period))

surveyyrs <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr") %>% 
  distinct(site_id, yr) %>% 
  count(site_id)

surveydates <- surveys %>% 
  filter(survey_type == "cmr") %>% 
  mutate(moday = format(as.Date(survey_date), "%b-%d"))

translocationdates <- surveys %>% 
  filter(survey_type == "translocation") %>% 
  mutate(moday = format(as.Date(survey_date), "%b-%d"))

frog_counts_gt0_nrow <- d2 %>% 
  filter(pyears_tadsub_present > 0) %>% 
  nrow()

frog_counts_gt0_median <- d2 %>% 
  filter(pyears_tadsub_present > 0) %>% 
  summarize(median(pyears_tadsub_present)) %>% 
  round(2)

frog_counts_gt0_min <- d2 %>% 
  filter(pyears_tadsub_present > 0) %>% 
  summarize(min(pyears_tadsub_present)) %>% 
  round(2)

frog_counts_gt0_max <- d2 %>% 
  filter(pyears_tadsub_present > 0) %>% 
  summarize(max(pyears_tadsub_present))

frog_captures_gt0_nrow <- d3 %>% 
  filter(pyears_recruits_present > 0) %>% 
  nrow()

frog_captures_gt0_median <- d3 %>% 
  filter(pyears_recruits_present > 0) %>% 
  summarize(median(pyears_recruits_present)) %>% 
  round(2)

frog_captures_gt0_min <- d3 %>% 
  filter(pyears_recruits_present > 0) %>% 
  summarize(min(pyears_recruits_present)) %>% 
  round(2)

frog_captures_gt0_max <- d3 %>% 
  filter(pyears_recruits_present > 0) %>% 
  summarize(max(pyears_recruits_present))
```

To determine whether MYL frogs from recovering populations can be used to reestablish extirpated populations, we conducted `r n_distinct(d1$site_id, d1$date)` reintroductions in Yosemite National Park (`r min(d1$year)`--`r max(d1$year)`).
Each of the reintroductions involved collection of adult frogs from 1 of `r n_distinct(d1$donor)` recovering, Bd-positive "donor" populations and translocating them to 1 of `r n_distinct(d1$site_id)` nearby recipient sites (@fig-yosemap SI).
The 3 donor populations are among the largest MYL frog populations in Yosemite, and included 2 of the 5 recovering populations used in the frog evolution study referenced above (see **Results - Frog evolution in response to Bd:** population 1 and 4).
In addition, this subset of 2 donor populations provided frogs for 20 of the `r n_distinct(d1$site_id, d1$date)` translocations.
Following translocation, we estimated adult survival and recruitment of new adults from capture-mark-recapture (CMR) surveys and obtained counts of tadpoles and juveniles from visual encounter surveys (VES).
Across all translocation sites, the duration of survey time series was `r min(surveyyrs$n)`--`r max(surveyyrs$n)` years (median = `r (median(surveyyrs$n))`).

Of the `r nrow(d2)` reintroduced populations, `r frog_counts_gt0_nrow` (`r frog_counts_gt0_nrow / nrow(d2)`) showed evidence of successful reproduction in subsequent years, as indicated by the presence of tadpoles and/or juveniles.
For these `r frog_counts_gt0_nrow` populations, one or both life stages were detected in nearly all survey-years following translocation (proportion of survey-years: median = `r frog_counts_gt0_median`, range = `r frog_counts_gt0_min`--`r frog_counts_gt0_max`).
These same populations were also those in which recruitment of new adults (i.e., progeny of translocated individuals) was detected.
As with early life stages, recruits were detected in the majority of post-translocation survey-years (proportion of survey-years: median = `r frog_captures_gt0_median`, range = `r frog_captures_gt0_min`--`r frog_captures_gt0_max`).
In summary, survey results indicate that translocations resulted in the establishment of reproducing MYL frog populations at most recipient sites despite the ongoing presence of Bd.

Bd loads were fairly consistent before versus after translocation, and loads were nearly always well below the level indicative of severe chytridiomycosis (i.e., the disease caused by Bd) and associated frog mortality (@fig-bdload-beforeafter SI) [@joseph2018; @vredenburg2010].
Although it is possible that the observed relatively small changes in load are a consequence of individuals with high Bd loads dying and therefore being unavailable for sampling during the post-translocation period, the fact that there was little difference in pre- versus post-translocation Bd loads even in those populations that had very high frog survival (70556, 74976 - see below; @fig-bdload-beforeafter SI) suggests a true lack of substantial change in Bd load.

The ultimate measure of reintroduction success is the establishment of a self-sustaining population.
Given that it can take years or even decades to determine the self-sustainability of a reintroduced population [for an example in MYL frogs, see @joseph2018], the use of proxies is essential for providing shorter-term insights into reintroduction success and the factors driving it.
Results from our CMR surveys allowed us to accurately estimate frog survival, including over the entire CMR time series for each site and during only the 1-year period immediately following translocation.
These estimates were made using site-specific models analyzed using the mrmr package.
We use these estimates to describe general patterns of frog survival in all translocated cohorts, and in an among-site meta-analysis of frog survival to identify important predictors of 1-year frog survival (e.g., Bd load).

Estimates of 1-year frog survival indicate that survival was highly variable between recipient sites, but relatively constant within recipient sites (for the subset of sites that received multiple translocations; @fig-translocation-survival).
These patterns indicate an important effect of site characteristics on frog survival.
In addition, 1-year survival was higher for frogs translocated later in the study period than earlier: 5 of the 7 populations translocated after 2013 had estimated survival $\ge$ 0.5, compared to only 1 of 5 populations translocated prior to 2013.
We suggest this resulted primarily from our improved ability to choose recipient sites with higher habitat quality for *R. sierrae* (see **Materials and Methods - Frog population recovery - Field methods** for details).
This increased survival has direct implications for population viability (see **Results - Long-term population viability**).

The goal of our meta-analysis was to identify important predictors of 1-year frog survival.
We were particularly interested in whether Bd load had a negative effect on adult survival, as would be expected if frogs were highly susceptible to Bd infection.
This analysis was conducted in a Bayesian framework and included a diversity of site, cohort, and individual-level characteristics as predictors and 1-year frog survival (@fig-translocation-survival) as the response variable.
The best model of 1-year frog survival identified several important predictors, but Bd load at the time of translocation was not among them (@fig-survival-postdens SI).
Instead, important predictors included winter severity in the year following translocation (snow_t1), site elevation, and donor population (@fig-cond-effects, @fig-survival-postdens SI).
Males had somewhat higher survival than females, but this effect was small (@fig-cond-effects, @fig-survival-postdens SI).
The absence of Bd load as an important predictor of frog survival is consistent with frogs in recovering populations having sufficient resistance to suppress Bd loads below harmful levels.

In summary, results from our frog translocation study indicate that (i) translocations produced relatively high 1-year survival of translocated adults, as well as reproduction and recruitment, at the majority of recipient sites, (ii) 1-year survival of adults is influenced by site characteristics, weather conditions, and donor population (but not Bd load), and (iii) based on the relatively small changes in Bd load after translocation, loads appear more strongly influenced by frog characteristics (e.g., resistance) than site characteristics.
Together, these results indicate that frogs translocated from recovering populations can maintain the benefits of resistance in non-natal habitats.
In addition, in 3 locations where longer CMR time series allowed us to assess the survival of new adults recruited to the population, naturally-recruited adults had equivalent or higher survival probabilities than the originally translocated adults (@fig-compare_surv_probs SI).
This suggests that frog resistance is maintained across generations.
All of the conditions described above are supportive of population establishment and long-term population growth.

#### Long-term population viability

Results from the frog translocation study indicated that most populations showed evidence of successful reproduction and recruitment, and that adult survival was often relatively high (described above).
Although suggestive of population establishment, a decade or more of surveys may be necessary to confirm that populations are in fact self-sustaining [@joseph2018].
To extend our inferences of population establishment beyond those possible from the site-specific CMR data, we developed a population viability model.
Specifically, to test whether the observed yearly adult survival probabilities in translocated populations were sufficient for long-term viability, we built a stage-structured matrix model that captured known frog demography and included demographic and environmental stochasticity.
We parameterized the model using CMR data from translocated populations and known life history values in this system (@tbl-param_values SI).

Given observed yearly adult survival probabilities of translocated frogs (from site-specific mrmr CMR models; provided in legend of @fig-viability B) and a yearly survival probability of the year-1 juvenile class ($\sigma_{J_1}$) greater than 0.09, at least six of twelve translocated populations should experience a long-run growth rate $\lambda$ greater than 1 in the presence of Bd (@fig-viability A; median predicted $\lambda$ ranges from 1.19-1.40 for these six populations).
These six populations all had observed yearly adult survival greater than 0.5.
As year-1 juvenile survival probability increased above 0.2, the deterministic long-run growth rate of eight of twelve population was greater than 1 (@fig-viability A).

Even when incorporating (i) demographic stochasticity and (ii) environmental stochasticity in year-1 juvenile survival and recruitment (the transition that we expect to be the most subject to environmental variability in the presence of Bd), populations with high adult survival are likely to persist over a 50 year time horizon.
Our model predicted that, following a single introduction of 40 adult individuals into a population, the six populations with the highest adult survival probabilities ($\sigma_{A_R} > 0.5$) had 50-year extinction probabilities of less than 0.5 when the average year-1 juvenile survival was greater than 0.10 (@fig-viability B).
This indicates strong potential for long-term persistence in the presence of Bd and environmental variability in survival and recruitment.
In contrast, for the six populations where yearly adult survival probability $\sigma_{A_R} < 0.5$, extinction probability over 50 years was always predicted to be \> 50% regardless of the value of mean year-1 juvenile survival between 0 and 0.25.
To test the validity of our model predictions, we demonstrated that our stochastic model could describe the general recovery trajectory of our translocated population with the longest survey history (@fig-viability C; population 70550, surveyed for 16 years).

In summary, our model demonstrates that given observed yearly adult survival probabilities of translocated frogs, 50% of our translocated populations have a high probability of population growth and long-term viability in the presence of Bd.
This is likely a conservative estimate because there is evidence that naturally-recruited adults have higher survival probability than translocated adults (@fig-compare_surv_probs SI), but we considered these probabilities to be equal in all but three of our populations where we had sufficient data to distinguish these different probabilities.

#### Frog genome evolution

Results from the preceding sections indicate the critical role of intrinsic factors (such as frog resistance) in post-translocation frog survival, population growth, and population viability.
As such, identifying the mechanisms underlying resistance would fill a key gap in our understanding of the factors that promote population resilience in the presence of disease.
Although natural selection for more resistant frog genotypes, and evolutionary rescue, may be foundational to the ability of frogs to recover despite ongoing Bd infection, for MYL frogs the role of disease-mediated selection in these processes remains unknown.

To determine whether MYL frog populations show genomic patterns consistent with an evolutionary response to Bd, we compared frog exomes (i.e., coding region of a genome) between populations with contrasting histories of Bd exposure.
Specifically, we compared frog genomes sampled in 4 populations that have not yet experienced a Bd-caused epizootic ("naive") [@zhou2015] versus in 5 populations that experienced a Bd epizootic during the past several decades and have since recovered to varying degrees ("recovering"; @fig-allelefrequencies) [@knapp2016; @vredenburg2010].
Bd-exposure histories of the 9 study populations are based on 10-20 years of VES and Bd surveillance using skin swabbing [e.g., @knapp2016; @zhou2015; @wilber2022].
Naive populations are characterized by large numbers of adults (i.e, typically 1000s), Bd prevalence that is generally 0% except during occasional Bd failed invasions [during which Bd loads remain very low, @wilber2022], and no history of Bd epizootics since we first surveyed these populations in the late 1990s and early 2000s [@zhou2015].
In contrast, recovering populations exist in an enzootic state [@briggs2010], characterized by smaller numbers of adults (generally \< 500), high Bd prevalence [often \> 80%, @knapp2011], and, in adults, moderate Bd loads that are typically well below the level expected to cause mortality [@vredenburg2010].
Naive and recovering populations can be identified unambiguously using these differences in Bd prevalence and load.
Finally, there is no potential for frog dispersal between the 9 study populations due to intervening distances and topography, as well as the presence of introduced (predatory) fish and fish-induced habitat fragmentation.

It is important to note that our genomic analysis is necessarily limited by the small number of naive populations remaining on the landscape (nearly all of which we sampled as part of the current study) and their clumped distribution ([@fig-allelefrequencies]).
These characteristics of naive populations precluded a replicated analyses among pairs of geographically proximate naive and recovering populations.
Instead, to obtain adequate power, we conducted a pooled analysis across population types to identify genomic regions showing patterns consistent with selection.
Our comparative approach cannot provide a direct assessment of the strength or timing of natural selection, but does allow us to identify specific regions of the genome that merit further study for their potential contribution to frog resistance in recovering populations.
(See **Supporting Information - Frog evolution in response to Bd - Study design** for additional details regarding the study design.)

As a first step in our pooled analysis, we conducted a principal component analysis (PCA) of the genomic data to describe the relationships between sampled populations.
We then used two complementary approaches to identify regions of the genome that differed between naive and recovering populations (i.e., regions possibly under selection).
First, we used a multivariate linear mixed model to evaluate associations between population type (i.e., naive versus recovering) and individual variants, including single nucleotide polymorphisms (SNPs) and insertions/deletions (INDELS), while accounting for population structure.
Second, we used a splined window analysis to identify larger genomic regions showing differences between population types in *F~ST~* and nucleotide diversity ($\pi_{diff}$ = $\pi_{naive}$ - $\pi_{recovering}$).

Individual frogs clustered into 3 separate groups in principal component space (@fig-selectionresults A SI), and clusters reflected the species split (i.e., *R. muscosa* versus *R. sierrae*) and the strong signature of isolation-by-distance that is characteristic of MYL frogs [@rothstein2020; @poorten2017; @byrne2023].
Importantly, each cluster contained at least one population from both the naive and recovering groups, allowing us to distinguish allelic associations of individuals sampled in the 2 population types versus allelic associations resulting from population structure and genetic drift.

Results from the individual variant and splined window analyses show that recovering populations show patterns consistent with selection on multiple regions of the genome.
The analysis of individual variants identified 11 "outlier" SNPs (i.e., showing significantly different allele frequencies between naive versus recovering populations) from 7 distinct genes across 4 contigs (@fig-selectionresults B, C SI).
One of the 7 identified genes (LOC108802036) does not have an associated annotation.
For the outlier SNPs, frequency differences between the naive and recovering populations ranged from 0.41 to 0.86.
Most of these SNPs showed frequency differences in only a subset of the sampled populations (@fig-allelefrequencies A, B), but the SNP in the RIN3 gene showed consistent differences in frequencies across all populations (@fig-allelefrequencies C).
This is suggestive of parallel selection at this locus across multiple populations.
The other 6 outlier variants showed less consistent frequency differences across the study populations, but for these we still found statistically significant differences in 2 of the 3 genetic clusters (containing populations 5--9; @fig-allelefrequencies, @fig-selectionresults A SI).
Therefore, although some outlier variant associations have a more limited geographic extent than RIN3, they still describe results that are consistent with parallel evolutionary changes following Bd exposure.
<!--# Add text citing new figure -->

The splined window analysis identified 33 outlier regions for $\pi_{diff}$ and 58 outlier regions for *F~ST~* (@fig-spline-manhattan A, B).
Of these, 9 regions were outliers for both metrics ("shared regions") and 2 of these shared regions also contained one or more of the outlier SNPs described above.
A total of 35 annotated genes were found in the 9 shared regions.
Given this large number of genes, here we focus on those with the strongest signal of selection and/or immune-related functions.
The largest $\pi_{diff}$, suggestive of directional selection, occurred in a 163kb region on Contig19, 12.9Mb upstream of the RIN3 outlier SNP (@fig-spline-manhattan C).
This region contains approximately 500 SNPs and one annotated gene called "interferon-induced very large GTPase 1-like" (GVINP1).
Additionally, a shared outlier region on Contig1 contained two complement factor genes (C6 and C7).
Interestingly, this region had a large negative $\pi_{diff}$ value, consistent with balancing selection.
Finally, one shared outlier region on Contig8 contained one outlier SNP (TCF19) and was within 360kb of another outlier SNP (VARS) (@fig-spline-manhattan D, @fig-synteny-plot SI).
This region (854kb from the beginning of the outlier window to the VARS SNP) contained a total of 8 annotated genes.
In *Xenopus*, five of these genes occur in the extended major histocompatibility complex (MHC) Class I region (FLOT1, TUBB, MDC1, CCHCR1, TCF19) and three occur in the extended MHC Class III region (HSP70, LSM2, VARS) [@ohta2006].
Therefore, this region is part of the extended MHC Class I and III complex and shows synteny with other amphibian genomes.

Although the joint processes of Bd-caused population declines and selection in response to Bd exposure could affect genetic diversity of recovering populations, we found no consistent differences in individual-level heterozygosity or population-level $\pi$ between naive and recovering populations (see **Supporting Information - Frog evolution in response to Bd - Genetic diversity** for details).
Thus, despite patterns consistent with localized selection in particular regions of the genome, we did not find evidence for reduced genetic diversity across the genome in recovering populations.
In addition, no gene ontology (GO) biological functions, molecular functions, or cellular processes were over-represented in either the outlier variants or the 35 genes located in the overlapping *F~ST~* and $\pi_{diff}$ splined windows (see **Supporting Information - Frog evolution in response to Bd** **- GO analysis** for details).

In summary, our genomic results indicate that frogs from naive and recovering populations show differences at several immune-function loci, consistent with selection following Bd exposure.

### Discussion

Disease-induced population declines are decimating global biodiversity [@daszak2000], but broadly-applicable strategies to recover affected species are generally lacking [e.g., @garner2016].
Here, we tested the possibility that populations of resistant individuals from naturally recovering populations can be used to reestablish extirpated populations of the endangered MYL frog in the presence of a highly virulent fungal pathogen (Bd).
Our results indicate (i) the capacity of reintroduced populations to become established and eventually recover despite ongoing disease, (ii) that 50% of the recovering populations are likely to persist over a 50-year period, (iii) that there are genomic differences between naive and recovering MYL frog populations, consistent with evolutionary change in frogs following Bd exposure, and (iv) that some of the genomic differences are in regions that contain genes related to disease resistance.
Collectively, these results (@fig-recovery-model) provide a rare example of amphibian recovery in the presence of Bd, and have important implications for the conservation and recovery of amphibians and other taxa worldwide that are endangered by escalating impacts from emerging infectious diseases.
In light of the generally low success rate of amphibian reintroduction efforts [@dodd2005], our success in reestablishing MYL frog populations via translocation of resistant individuals is striking, and even more so given that MYL frogs were driven to near-extinction by Bd.

In the following discussion, we follow the sequence of frog recovery described in @fig-recovery-model to structure our key points.
Previous field studies in MYL frogs show that frog-Bd dynamics and frog survival in the presence of Bd are fundamentally different between naive and recovering populations.
Following the arrival and establishment of Bd in previously-naive populations, adult frogs develop high Bd loads that lead to mass die-offs [@vredenburg2010].
In contrast, in recovering populations adult frogs typically have low-to-moderate and relatively constant Bd loads and mass die-offs are not observed [@briggs2010; @knapp2011, see also @fig-bdload-beforeafter SI].
The differences in Bd load of frogs from naive and recovering populations are also observed in controlled laboratory studies [see Figure 4 in @knapp2016], and clearly indicate that frogs from recovering populations exhibit resistance against Bd infection.
This resistance could be due at least in part to natural selection for more resistant genotypes, but until now evidence to evaluate the possible role of evolution was lacking.

Our genomics study used a pooled design that compared MYL frogs from naive and recovering populations distributed across a similar latitudinal range.
The study design produced frog genetic clusters that each contained at least one naive and one recovering population, allowing us to infer selection while attempting to minimize the confounding effects of population structure.
We identified multiple specific alleles and genomic regions showing differences between naive and recovering MYL frog populations, consistent with selection following Bd exposure.
In addition, we did not find a reduction in overall genetic variation in recovering populations, suggesting that despite possible localized selection in the genome, some of these populations likely retain adequate genetic diversity for long-term persistence.

Importantly, some genomic regions that we identified as possibly under selection are associated with cellular and immunological mechanisms known to contribute to disease resistance, including in amphibians [@zamudio2020].
For example, the MHC plays an important role in immunity.
In our study, we identified a region that shows evidence of selection in recovering populations and contains eight genes associated with either the MHC Class I or Class III regions.
These results corroborate numerous previous studies linking MHC genes to amphibian resistance against Bd [e.g., @savage2011; @bataille2015].
Similarly, the region with the strongest pattern of possible directional selection (as measured by $\pi_{diff}$) contains the interferon-related gene GVINP1.
Several previous studies of amphibians have found this gene to be differentially expressed during Bd infection [e.g., @grogan2018b; @ellison2014] and in populations differing in Bd susceptibility [@grogan2018b].
This gene is also strongly linked to disease in salmon, explaining a notable 20% of the resistance phenotype [i.e., gill damage and amoebic load as heritable host resistance traits, @robledo2020; @robledo2018].
We also identified a region, characterized by high *F~ST~* and low $\pi_{diff}$, that contained the complement genes C6 and C7.
The complement system plays an important role in innate immunity [@rieraromo2016], and our results could indicate that balancing selection is acting in this region of the genome to favor a diverse set of alleles, as is known for C6 in humans [@soejima2005].
Based on the analysis of individual outlier variants, the RIN3 gene showed a consistent pattern of allele frequency differences across all nine of the frog populations sampled in this study, suggesting possible consistent selection in populations distributed across a wide geographic area.
This gene is associated with immune response and in *Xenopus* is expressed during appendage regeneration [@fukazawa2009].
Finally, the outlier variant with the lowest p-value was the uncharacterized gene LOC108802036.
In the genome of another frog species, this gene is located adjacent to a type I interferon gene (Np-IFNi2) [@gan2018], and together with GVINP1 further suggests the importance of interferon-related genes in this system.
Collectively, the genes associated with these genomic differences may confer at least some degree of resistance against Bd infection, an attribute that may be critically important to population reestablishment and recovery in the presence of Bd.

Reintroduction of resistant MYL frogs was remarkably successful in reestablishing viable populations in the presence of Bd.
Of the 12 translocated populations, approximately 80% showed evidence of both successful reproduction and recruitment of new adults.
Year-1 survival for 12 of the 24 translocated cohorts exceeded 50%, and \> 70% of translocated cohorts had survival above this 50% level when the earliest translocations are excluded (i.e., translocations conducted when methods were still being refined; see **Materials and Methods - Frog population recovery - Field methods** for a brief description of these refinements).
The fact that the relatively low Bd loads and correspondingly high frog survival was maintained when frogs were moved from donor populations to recipient sites indicates that these characteristics of naturally-recovering populations were not solely an effect of site characteristics, but were also strongly influenced by resistance inherent in the frogs.
Although it could be argued that the relatively invariant Bd loads before versus after translocation are a consequence of similar pathogen pressure in the donor and translocated populations, this is at odds with the fact that in the first year after translocation frog densities are typically 1-2 orders of magnitude lower in the translocated versus donor populations and pathogen pressure should follow a similar pattern.
In addition to the maintenance of Bd load and frog survival between natal and translocation sites, the relatively high survival of translocated frogs was maintained in their progeny, as expected if resistance has a genetic basis.

Results from the population viability model were also encouraging.
In particular, translocated populations with \> 50% survival in the first year post-translocation were predicted to have a low probability of extinction over 50 years (probability of extinction $<$ 0.5 when year-1 juvenile survival probability was greater than 0.10).
The viability model highlighted the important role of frog survival in affecting long-term population viability, and allowed us to extend the temporal scale of our study beyond the years covered by our post-translocation surveys.
These long-term forecasts are important, given that reintroduced MYL frog populations may often take decades to achieve our ultimate goal of self-sustainability [@joseph2018].
Making well-supported projections about the long-term outcome of reintroduction efforts from shorter-term information is critically important to the process of adaptive management of species reintroduction programs [@seddon2007], including the one we are carrying out for MYL frogs.
Specifically, the combined results from our reintroduction study and viability modeling indicate that survival of frogs in the first year following translocation is an effective proxy of longer-term survival and population viability.
In addition, given the repeatability of frog survival at a site, 1-year frog survival also serves as an effective proxy of site quality (i.e., the ability of a site to support high frog survival and a viable frog population over the long term).
This proxy of site quality is important in the MYL frog system because accurately predicting the ability of a site to support a viable frog population *a priori* remains difficult, even after conducting 24 translocations over 16 years.

The results of the meta-analysis indicate the important influence of several variables on adult survival, and by extension on population viability.
Although we assume that the effects of winter severity and frog sex are direct effects of the variables themselves, the elevation and donor population variables are likely serving as proxies for a range of factors.
For example, elevation is correlated with several attributes that may influence frog survival, including the presence of boulder habitat (positive correlation) that is important for frog overwintering, abundance of frog predators including garter snakes (negative correlation), and frog developmental rates (negative correlation).
The latter can influence the duration of frog life stages that have differential susceptibility to chytridiomycosis (see below).
The duration of the tadpole and juvenile stages increases with elevation, but the fact that long-run population growth rate is relatively insensitive to variation in these durations suggests that developmental rates of early frog life stages have little effect on the predicted long-term viability of reintroduced populations.
Nonetheless, a clearer understanding of the mechanistic components of the elevation and donor population variables will require additional research.

Despite the demonstrated resistance of adult MYL frogs against Bd infection, individual and population-level impacts of Bd are still evident.
In an earlier study of 2 of our 12 translocated populations [@joseph2018], Bd infection and load had detectable effects on the survival of adults and may have influenced population establishment (sites referred to as "Alpine" and "Subalpine" in @joseph2018 are identified as "70550" and "70505" in the current study).
Applying similar analyses to all 12 of our translocated populations would likely provide a broader perspective of the ongoing effect of Bd.
In addition to these important but relatively subtle effects of Bd on adults, the impacts on younger life stages are more apparent.
MYL frogs immediately following metamorphosis ("metamorphs") are highly susceptible to Bd infection [@ellison2019] and as a result experience high mortality [@rachowicz2006].
This high susceptibility of metamorphs is documented in numerous species of anurans, and may result from the poorly developed immune system characteristic of this life stage [@humphries2022].
In naturally recovering and translocated MYL frog populations, we suggest that the high mortality of metamorphs is an important limitation on subsequent recruitment of new adults.
Therefore, although adult MYL frogs appear relatively resistant, Bd infection continues to have important limiting effects on recovering populations [see also @hollanders2022].

The recent emergence of Bd worldwide has contributed to the decline of hundreds of amphibian species, some of which are now extinct in the wild [@scheele2019].
This extraordinary impact on global amphibian biodiversity is compounded by the lack of any effective and broadly applicable strategies to reverse these impacts [@garner2016; @knapp2022].
Importantly, in addition to the natural recovery documented for MYL frogs [@knapp2016], other amphibian species are also showing evidence of post-epizootic recovery in the presence of Bd [@scheele2017; @voyles2018] and suggest the possibility of also using animals from these recovering populations to reestablish extirpated populations.
As with MYL frogs, the feasibility and long-term success of such efforts will depend on the availability of robust donor populations containing individuals that have the adaptive alleles necessary to allow frog survival and population growth in the presence of Bd.
Despite the hopeful example of successful reestablishment of MYL frogs despite ongoing Bd infection, the challenge of recovering hundreds of Bd-impacted amphibian species globally is a daunting prospect.
Although we now have a proven strategy to reestablish extirpated MYL frog populations, recovery across their large historical range will require substantial resources over many decades.
The results of this study provide a hopeful starting point for that endeavor and other future efforts worldwide.

In our rapidly changing world, evolution is likely to play in important role in facilitating the resilience of wildlife populations.
Whether the documented disease resistance in MYL frogs and concurrent recovery of decimated populations provides an airtight example of evolutionary rescue will likely always be uncertain (given that we can never have a perfect understanding of the past).
Regardless, we provide an example from the wild that suggests that evolution might produce individuals that harbor adaptive alleles and allow population recovery in a novel (i.e., Bd-positive) environment, and show conclusively that individuals from these recovering populations can be used to reestablish extirpated populations and expand the scale of natural recovery (@fig-recovery-model).
We expect that similar species recovery actions will be an essential tool in wildlife conservation in an era of accelerating global change.

### Materials and Methods

#### Frog population recovery

##### Field methods

For the `r n_distinct(d1$site_id, d1$date)` translocations we conducted, we identified donor populations from which adult frogs ($\geq$ 40 mm snout-vent length) could be collected using several years of VES and skin swab collections [@knapp2016], and results from population genetic analyses [@poorten2017].
The populations that we selected contained hundreds of *R. sierrae* adults and thousands of tadpoles.
These relatively high abundances were the result of recent increases following previous Bd-caused declines [@knapp2016].
As is typical for recovering MYL frog populations, Bd prevalence in the donor populations was high (`r round(bdprev_min, 2)`--`r round(bdprev_max, 2)`) and Bd load (median log~10~(load) = `r round(bdload_min, 2)`--`r round(bdload_max, 2)` ITS copies) was two or more orders of magnitude below the level at which frog mortality is expected (log~10~(load) $\approx$ 5.78 copies)[@vredenburg2010; @joseph2018].
Recipient sites to which frogs were translocated were chosen based on previous *R. sierrae* presence (determined from VES and/or museum records) or characteristics that suggested high quality habitat for this species [@knapp2005].
At the beginning of this study, we had a relatively limited understanding of the factors that affect habitat quality.
In subsequent years, we improved our site selection process by incorporating new information about important habitat features, in particular, overwinter habitats such as submerged boulders and overhanging banks.
*R. sierrae* were absent from all recipient sites prior to the first translocation.

We conducted `r transno_min`--`r transno_max` translocations per site (@fig-translocation-survival, @fig-yosemap SI) and each translocated cohort included `r transfrogs_min` to `r transfrogs_max` frogs (median = `r transfrogs_median`).
In preparation for each translocation, adult frogs were collected from the donor population and measured, weighed, swabbed, and PIT tagged.
Frogs were transported to the recipient site either on foot or via helicopter.
Following release, we visited translocated populations approximately once per month during the summer active season and conducted diurnal CMR surveys and VES (summer active season is generally July-August but can start as early as May and end as late as September; range of survey dates = May-25 to Sep-29, range of translocation dates = Jun-28 to Sep-02; median number of visits per summer = `r round(surveys_yr_median)`, range = `r surveys_yr_min`--`r surveys_yr_max`).
CMR surveys allowed estimation of adult survival, recruitment of new adults, and adult population size, and VES provided estimates of tadpole and juvenile abundance.
During 2006-2012, we conducted CMR surveys on a single day (primary period) per site visit, during which we searched all habitats repeatedly for adult frogs.
Frogs were captured using handheld nets, identified via their PIT tag (or tagged if they were untagged), measured, weighed, swabbed, and released at the capture location.
During 2013-2022, we generally used a robust design in which all habitats were searched during several consecutive days (median number of secondary periods per primary period = `r round(robust_year_median)`; range = `r robust_year_min`--`r robust_year_max`), and frogs were processed as described above.
However, when the number of frogs detected on the first survey day was zero or near zero, we typically conducted only a single-day CMR survey.
When using a robust design, within a primary period, frogs that were captured during more than one secondary period were measured, weighed, and swabbed during the first capture, and during subsequent captures were only identified and released.

During each site visit, we conducted VES either immediately before CMR surveys or during the first day of CMR surveys.
VES was conducted by walking the entire water body perimeter, first 100 m of each inlet and outlet stream, and any fringing ponds and wetlands, and counting all *R. sierrae* tadpoles and juveniles.
These *R. sierrae* life stages have high detectability, and counts are highly repeatable and provide estimates of relative abundance [@knapp2000].

##### Frog counts and reproductive success

For each of the translocated populations, we used the presence of tadpoles and/or juveniles from VES and counts of new recruits (i.e, untagged adults) in CMR surveys to provide two measures of successful reproduction.
To calculate the proportion of years in which tadpoles/juveniles were present at a site, we excluded surveys conducted in the year of the initial translocation to that site.
This exclusion accounted for the fact that all translocations were conducted after the breeding period and reproduction would therefore not occur until the following year.
Similarly, to calculate the proportion of years in which new recruits were present at a site, we excluded surveys conducted during the 3 years following the initial translocation.
This accounted for the multi-year tadpole and juvenile stages in MYL frogs (@tbl-param_values SI).

##### Estimation of frog survival and abundance

For each translocation site, we estimated survival of translocated frogs, recruitment of new frogs into the adult population, and adult population size using a site-specific Bayesian open-population Jolly-Seber CMR model with known additions to the population (i.e., translocated cohorts), as implemented by the mrmr package [@joseph2019] and using R Statistical Software [v4.4.4, @rsoftware2022] (see **Supporting Information - Frog population recovery - CMR model structure** for details).
Briefly, the model tracks the states of *M* individuals that comprise a superpopulation made up of real and pseudo-individuals [see @joseph2018 for details].
The possible states of individuals include "not recruited", "alive", and "dead".
The possible observations of individuals include "detected" and "not detected".
We assume that individuals that are in the "not recruited" or "dead" states are never detected (i.e., there are no mistakes in the individual PIT tag records).
We also assume that new recruits were the result of within-site reproduction and not immigration from adjacent populations.
This assumption is justified by the fact that no *R. sierrae* populations were present within several kilometers of the translocation sites.
For all models, we used mrmr defaults for priors, number of chains (4), and warmup and post-warmup iterations (2000 for each).
We evaluated convergence of the Markov chain Monte Carlo (MCMC) algorithm using trace plots and Gelman-Rubin statistics (Rhat).

##### Predictors of post-translocation frog survival

To identify important predictors of frog survival following translocation, we used multilevel Bayesian models [@gelman2013; @gabry2019].
Included predictor variables describe characteristics of sites, translocated cohorts, and individuals (Bd load, sex, frog size, site elevation, winter severity in the year of translocation, winter severity in the year following translocation, donor population, day of year on which a translocation was conducted, and translocation order).
We used 1-year post-translocation survival estimates from CMR models as the response.
Estimated survival was rounded to integer values to produce a binary outcome, and modeled with a Bernoulli distribution.
Group-level (random) effects included site_id, translocation_id, or translocation_id nested within site_id.
We performed all analyses with the rstanarm package [@rstanarm2022] and R Statistical Software [v4.4.4, @rsoftware2022].
For all models, we used default, weakly informative priors, four chains, and 5000 iterations each for warmup and post-warmup.
We checked MCMC convergence using trace plots and Rhat, and evaluated model fit using leave-one-out cross-validation [@vehtari2016], as implemented by the loo package [@vehtari2022].
(See **Supporting Information - Frog population recovery - Among-site survival modeling** for details.)

##### Changes in Bd load following translocation

We analyzed skin swabs using standard Bd DNA extraction and qPCR methods [@boyle2004, see **Supporting Information - Frog population recovery - Laboratory methods** for details].
To assess the magnitude of changes in Bd load on frogs following translocation, we compared Bd loads measured before versus after translocation.
Before-translocation loads were quantified using skin swabs collected from all to-be-translocated frogs at the donor site on the day before or the day of the translocation.
After-translocation Bd loads were based on all swabs collected from translocated frogs at the recipient site in the year of and the year following translocation.
Individual frogs and their associated Bd loads were included in the dataset only if frogs were captured at the recipient site at least once during the 1-year period following translocation.

Code to replicate all of the analyses described above is available from the following GitHub repositories: <https://github.com/SNARL1/translocation>; <https://github.com/SNARL1/cmr-analysis>.

#### Population viability modeling

##### Model description

To determine the implications of observed 1-year adult survival on the long-term viability of populations established via translocation, we developed a population model for MYL frogs.
Our central question was: How does the magnitude and variation in observed adult survival probability across translocated populations affect the long-term persistence probability of populations?
We developed a model that tracked seven state variables of a frog population: density of translocated adults ($A_T$), density of adults naturally recruited into the population ($A_R$), density of first-year tadpoles ($L_1$), density of second-year tadpoles ($L_2$), density of third-year tadpoles ($L_3$), density of first-year juveniles ($J_1$), and density of second-year juveniles ($J_2$).
We divided adults into two classes $A_T$ and $A_R$ because there is evidence that the survival probability of translocated adults and naturally recruited adults differs ([@fig-compare_surv_probs] SI).

We modeled the dynamics of these seven state variables using a discrete-time, stage-structured model where a time step is one year.
The dynamics are given by

$$
\begin{bmatrix}
L_1 \\
L_2 \\
L_3 \\
J_1 \\
J_2 \\
A_R \\ 
A_T
\end{bmatrix}(t + 1) = 
\begin{bmatrix}
  0 & 0 & 0 & 0 & 0 & \sigma_{A_R} p_F F & \sigma_{A_T} p_F F \\
  \sigma_{L1} p_{L1} & 0 & 0 & 0 & 0 & 0 & 0 \\
  0 & \sigma_{L2} p_{L2} & 0 & 0 & 0 & 0 & 0\\
  \sigma_{L1} (1 - p_{L1}) & \sigma_{L2} (1 - p_{L2}) & \sigma_{L3} & 0 & 0 & 0 & 0 \\
  0 & 0 & 0 & \sigma_{J1} p_{J1} & 0 & 0 & 0 \\
  0 & 0 & 0 & \sigma_{J1} (1 - p_{J1}) & \sigma_{J2}  & \sigma_{A_R} & 0 \\
  0 & 0 & 0 & 0 & 0 & 0 & \sigma_{A_T} 
\end{bmatrix} \begin{bmatrix}
L_1 \\
L_2 \\
L_3 \\
J_1 \\
J_2 \\
A_R \\
A_T \\
\end{bmatrix}(t) 
$$

The parameters in this model are yearly survival probability $\sigma_{\cdot}$ (the subscript "$\cdot$" indicates a particular state variable), probability that a female frog reproduces in a given year $p_F$, number of eggs produced by a female frog in a year that successfully hatch $F$, probability of a first-year tadpole remaining as a tadpole $p_{L1}$, probability of a second-year tadpole remaining as a tadpole $p_{L2}$, and probability of a first-year juvenile remaining as a juvenile $p_{J1}$.
First-year juvenile survival and recruitment $\sigma_{J1}$ is the parameter that we think is most influenced by environmental stochasticity.

In this model we ignore density-dependent recruitment because we were interested in the growth of the population from an initial reintroduction and whether this growth was sufficient to prevent extinction over 50 years following the introduction.
We also did not directly consider the dynamics of Bd in this model.
We made this decision because (i) translocated populations are infected with Bd at high prevalence [@joseph2018], and (ii) host density does not seem to play a significant role in multi-year Bd infection dynamics in this system [@wilber2022].
Thus, ignoring Bd infection dynamics and instead assuming all host vital rates are in the presence of high Bd prevalence significantly simplifies the model without much loss of realism.
Additional details are provided in **Supporting Information - Population viability modeling - Incorporating yearly variability in survival rates** and **Estimating model parameters**.

##### Model analysis

After parameterizing our model with CMR-estimated adult frog survival probabilities and other known vital rates ([@tbl-param_values]), we performed four analyses.
First, we compute the long-run growth rate $\lambda$ for each of our 12 translocated populations to determine if the populations were deterministically predicted to grow or decline in the long-run.
Second, we compute the elasticity and sensitivity of $\lambda$ to nine model parameters to quantify how much changes in these parameters affected the long-run growth rate (@fig-viability-supp SI).
This also helped us determine where in the model environmental variation in juvenile survival and recruitment would have the largest effects on population dynamics.
Third, we included demographic stochasticity and environmental stochasticity in $\sigma_{J_1}$ in our model and simulated the 50-year viability (i.e., 1 - extinction probability) of populations given an introduction of 40 adult individuals into an unoccupied habitat.
Finally, we fit our model to our longest translocation trajectory to confirm that our model could reasonably reproduce the observed recovery trajectories of MYL frogs following reintroductions.
Additional details are provided in **Supporting Information - Population viability modeling - Model analysis and simulation**.
Code to replicate the analyses can be found at <https://github.com/SNARL1/translocation>.

#### Frog genome evolution

##### Sampling and sequencing

We collected DNA samples via buccal swabbing [@broquet2007] from 53 *Rana muscosa*/*Rana sierrae* individuals: 24 from 4 naive populations, and 29 from 5 recovering populations.
These populations are located in the southern Sierra Nevada, from northern Yosemite National Park to northern Sequoia National Park (@fig-allelefrequencies).
Samples were collected from 5-6 frogs per population.
To minimize potential confounding effects caused by variation in frog genotypes across latitude [@byrne2023], we selected sampling sites such that both population types were represented across similar latitudinal ranges.
DNA was extracted following Qiagen DNEasy manufacturer's protocols.
We sequenced the samples using an exome capture assay as described in @byrne2023.
Briefly, genomic libraries were prepared and captured using a custom Nimblegen capture pool.
Capture baits were designed based on the coding regions of the *R. muscosa* transcriptome (GenBank accession GKCT00000000).
Captured libraries were then pooled and sequenced on a NovaSeq 6000 150PE Flow Cell S1 at the Vincent J. Coates Genomics Sequencing Lab at UC Berkeley.
Raw sequencing reads are available from NCBI SRA (PRJNA870451).

##### Data pre-processing and cleaning

Raw reads were filtered for adapters and contaminants using fastp [@chen2018] and aligned to the *Rana muscosa* genome (NCBI SRA: SRS6533475 [@hon2020]), with repetitive elements masked using bwa ("mem" mode) [@li2013].
Exact PCR duplicates were marked using Picard.
Variants were then called following GATK best practices (v.4.2.0.0 [@van2020]).
Briefly, raw variants were called for each sample using HaplotypeCaller and combined using CombineGVCFs.
Next, genotypes were jointly called using GenotypeGVCFs.
Variants were then hard filtered using gatk VariantFiltration using the following parameters to remove low-quality sites: QD \< 2.0, FS \> 40.0, SOR \> 3.0, MQ \< 50.0, MQRankSum \> 3.0, MQRankSum \< -3.0, ReadPosRankSum \> 3.0, ReadPosRankSum \< -3.0.
This initial filter resulted in 1,595,206 variant sites across 53 individuals.

We then further filtered our dataset at the individual and variant level.
First, we trimmed our variants to only include those with minor allele frequency \> 0.03, a maximum depth of 250 and minimum depth of 5, a minimum genotype quality of 20, and a maximum missing proportion of 0.5.
This filter resulted in 427,038 sites, of which 353,172 were SNPs and 73,866 were INDELS.
Finally, we trimmed samples with an average depth across filtered sites \< 7x (n = 3).
Our final dataset included 50 samples, 23 from naive and 27 from recovering populations, with an average depth of 16.7x (range = 7.4x -- 26.1x).

##### Data analysis

To visualize the genomic relationships of our populations, we conducted a PCA using the glPCA function in the adegenet R package [@jombart2008].
To detect regions of the genome that differed between naive and recovering populations, we used two approaches: (1) a multivariate linear mixed model to evaluate individual variants (SNPs and INDELs), and (2) a splined window analysis to evaluate larger genomic regions.
For the variant analysis, we first used a stringent data filter to include only variants with \< 5% missing data (missing for no more than 2 individuals), and then calculated the likelihood ratio statistic for the resulting set of 148,307 high quality variants across 127 contigs using GEMMA [@zhou2014].
GEMMA calculates and incorporates a relatedness matrix for input samples, allowing us to account for relatedness and population structure when calculating likelihood ratio statistics.
We identified variants showing different allele frequencies between naive versus recovering populations ("outliers") using a Bonferroni-corrected significance level of 0.01.
We visualized the results using a Manhattan plot and qqplot.
We developed a more liberal set of outlier variants using a Bonferroni-corrected significance level of 0.05 and used this set solely for the gene ontology (GO) analysis (see below and **Supporting Information - Frog evolution in response to Bd - GO analysis;** Dataset S1, S2).

In the analysis of individual variants, for each outlier variant we determined whether the variant was synonymous (protein sequence the same for each variant) or non-synonymous (protein sequence differs between variants), and where in the gene it was located.
To do this, we first extracted the reference genome sequence surrounding the variant using the bedtools "getfasta" function [@quinlan2010].
Next, we re-annotated each sequence using BLAST to get the predicted gene location based on the closest annotated reference [@altschul1997].
We then translated each variant to amino acids and aligned this translation to that of the gene annotation to ensure proper frame of reference using Geneious [@kearse2012].
After ensuring proper translation, we characterized variants as within or outside the coding sequence of the gene and as either synonymous or non-synonymous.

In the splined window analysis, we identified outlier regions using *F~ST~* and differences in nucleotide diversity ($\pi_{diff}$) between naive and recovering populations.
First, we calculated per-site *F~ST~* between the naive and recovering individuals for all bi-allelic SNPs in the 30 largest contigs (98% of all SNPs) using VCFtools [@danecek2011].
Next, we calculated per-site nucleotide diversity $\pi$ separately for individuals from the naive and recovering populations using VCFtools, then calculated $\pi_{diff}$ for each population ($\pi_{diff} = \pi_{naive} - \pi_{recovering}$).
We concatenated the values for *F~ST~* and $\pi_{diff}$ in order of size-sorted chromosome number and adjusted the SNP position based on the relative position in the genome (for more efficient data processing and to better contextualize the strength of the outlier signals in different regions of the genome).
We then used the GenWin R package [@beissinger2015] to conduct a splined discrete window analysis for *F~ST~* and $\pi_{diff}$.
This method calculates where non-overlapping window boundaries should occur by identifying inflection points in the spline fitted to *F~ST~* and $\pi_{diff}$ values along the genome, therefore balancing false positive and false negative results that occur using other window-based methods [@beissinger2015].
This method also calculates a W-statistic allowing for outlier identification.
We identified outliers as those with a W-statistic greater than 4 standard deviations above the mean for *F~ST~* or above/below the mean for $\pi_{diff}$.
These standard deviations represent strict criteria to select only the top \~ 0.3% of windows.
Shared outliers were then identified as those that were outliers in both analyses, meaning that they showed (i) high differentiation between naive and recovering populations, and (ii) differential patterns of nucleotide diversity in the same region.
Finally, we extracted gene transcripts mapped within each region and retrieved annotation for that region using BLAST (**Supporting Information**: Datasets S3, S4).

Code used for genomic analyses and to create figures is available from the following GitHub repository: <https://github.com/allie128/mylf-selection>.

### Acknowledgments

We thank the following for important contributions to this study: E.
Hegeman and A.
Lindauer (project and data management, field and laboratory work); A.
Barbella and K.
Rose-(laboratory work); C.
Kamoroff and numerous summer technicians (field work); staff at Sequoia-Kings Canyon and Yosemite National Parks, Inyo and Sierra National Forests, California Department of Fish and Wildlife, U.S. Fish and Wildlife Service, University of California-Santa Barbara Institutional Animal Care and Use Committee, and Sierra Nevada Aquatic Research Laboratory (research permits, logistical support, and/or field assistance).
This project was supported by grants from the National Park Service (to R.A.K.), Yosemite Conservancy (to R.A.K.), and National Science Foundation (EF-0723563, to C. Briggs; DEB-1557190, to C. Briggs; DEB-2133401, to M.Q.W; and DBI-2120084, to C. Richards-Zawacki).

### References

See combined references at end of document (temporary placeholder).

\

### Figures

![For MYL frogs, a conceptual model depicting the Bd-caused decline and subsequent natural recovery (black text), facilitated recovery via reintroductions, and the linkages between these two pathways. Rectangles and hexagons represent outcomes and processes, respectively. Blue text indicates components that are included in the current study. We quantified frog survival and population viability following reintroductions directly using field studies and models, and evaluated "disease-mediated selection" indirectly using a comparison of frog exomes in naive and recovering populations. The timeline shows the general sequence of the components, with the dashed portion indicating a projection into the future.](figures/conceptual_model.png){#fig-recovery-model}

\newpage

![Median 1-year survival for each cohort of translocated frogs at the 12 recipient sites, as estimated for each site from the mrmr CMR model. Error bars show the 95% uncertainty intervals. Sites are arranged along the x-axis using the average of the median 1-year survival per translocation at each site. Dot colors indicate the donor population from which frogs in each translocated cohort were collected. When multiple translocations were conducted to a site, points and error bars are slightly offset to avoid overlap.](figures/translocation_survival_bysiteid.png){#fig-translocation-survival width="450"}

\newpage

![Results from the among-site meta-analysis showing conditional effects of the important predictors of 1-year frog survival (expressed as a probability): (A) winter severity in the year following translocation, (B) elevation of recipient site, (C) donor population, and (D) sex. In (A) and (B), blue lines are medians and gray ribbons are 95% uncertainty intervals. In (C) and (D), box plots show medians, first and third quartiles, largest and smallest values within 1.5x interquartile range, and values outside the 1.5x interquartile range.](figures/cond_effects_plot.png){#fig-cond-effects width="550"}

\newpage

![Results from population viability analysis: (A) Predicted long-run growth rate $\lambda$ for different values of yearly adult survival probability $\sigma_{A_R}$ and year-1 juvenile survival and recruitment probability $\sigma_{J_1}$, given the parameterized, deterministic model. Colored points show the predicted $\lambda$ values for the twelve translocated populations when year-1 juvenile survival probability is $\sigma_{J_1} = 0.09$ (indicated by the dashed line). The red line shows where $\lambda = 1$. Note that the point for 70413 is mostly hidden behind other points. (B) Predicted 50-year extinction probabilities of the 12 translocated populations, given demographic stochasticity, environmental variability in $\sigma_{J_1}$, and different mean values of $\sigma_{J_1}$. There are 6 lines at extinction probability = 1, 5 of which (70414--70619) are hidden beneath the line for 70505 when $\sigma_{J_1} < 0.10$. (C) 100 simulated trajectories (black lines) from the population viability model that most closely matched the observed abundance trajectory of adult amphibians at site 70550 (light orange).](figures/pop_viability_figures_for_manuscript.jpg){#fig-viability width="650"}

\newpage

![Genomic patterns consistent with selection on individual variants in recovering MYL frog populations at the landscape scale (based on a linear mixed model analysis). For each of the 9 naive and recovering MYL frog populations (indicated by numbered points), adjacent pie charts show allele frequencies for the 11 outlier SNPs from 7 distinct genes: (A) LOC108802036, (B) TCF19, VARS, MAP1S, TYRP1, and SRFBP1, and (C) RIN3. Charts are superimposed on a map of the Sierra Nevada study area, with Yosemite, Kings Canyon, and Sequoia National Parks (from north to south) shown in dark gray, and the range boundary for MYL frogs shown in light gray. The inset map locates the study area and range boundary in California. Sites 1 and 4 (site id = 72996 and 70567, respectively) were also used as sources of frogs in the frog population recovery study.](figures/allele_maps.png){#fig-allelefrequencies width="600"}

\newpage

![Patterns consistent with selection on genomic regions in recovering MYL frog populations based on a splined window analysis. Note that several key candidate genes (RIN3, TCF19, VARS) are outliers in both the individual variant analysis [@fig-allelefrequencies] and splined window analysis (this figure). Manhattan plots show outlier regions for the difference in (A) nucleotide diversity $\pi_{diff}$ and (B) *F~ST~*. In (A), outlier regions are shown above the upper red dashed line and below the lower red dashed line. In (B), outlier regions are shown above the single dashed red line. Outlier regions for either $\pi_{diff}$ or *F~ST~* are shown in blue and outlier regions for both $\pi_{diff}$ and *F~ST~* are shown in red. (C) Magnified Contig19 from (A) showing two adjacent outlier regions for $\pi_{diff}$ 12.9Mb upstream of the RIN3 outlier SNP (indicated with a dashed vertical blue line). (D) Magnified Contig8 from from (B) showing the *F~ST~* outlier region that includes the outlier SNPs TCF19 and VARS. This region of the genome contains 8 annotated genes known to occur in the extended MHC Class I and III regions.](figures/splinewindow_manhattan.png){#fig-spline-manhattan width="550"}

\newpage

## Supporting Information

### Frog population recovery

#### Laboratory methods

Swab extracts were analyzed using standard Bd DNA extraction and qPCR methods [@boyle2004], and extracts were analyzed singly instead of in triplicate [@kriger2006].
For analysis of swabs collected during 2005--2014, we used standards developed from known concentrations of zoospores [@boyle2004], and after 2014, we used standards based on single ITS1 PCR amplicons [@longo2013].
Based on paired comparisons between samples analyzed using both types of standards, Bd in the study area has an average of 60 ITS1 copies per zoospore.
To express all qPCR results as the number of ITS1 copies, starting quantities obtained using the zoospore standard (measured as "zoospore equivalents") were multiplied by 60.
In addition, all qPCR quantities (regardless of standard) were multiplied by 80 to account for the fact that DNA extracts from swabs were diluted 80-fold during extraction and PCR [@vredenburg2010].

#### CMR model structure

We estimated survival and recruitment for each site using open population CMR models based on @joseph2018.
For each individual in a superpopulation of real and psuedo-individuals, $i=1, ..., M$ on each survey $j=1, ..., n_j$: $o_{i, j} = 1$ if the individual was not detected, and $o_{i, j}=2$ if the individual was detected.
Capture histories of $M$ individuals are modeled, although only $N_s$ individuals were captured.
This parameter expanded data augmentation allows us to capture the possibility that undetected individuals may have recruited into the adult population [@royle2012].
Here, $M$ was chosen to be three times the number of observed individuals ($3N_s$) to be considerably greater than our prior guess of $N_s$.

We denote the true state of individual $i$ as $u_{i, t}$ for primary period $t = 1,..., n_t$.
The states that we consider are: $u_{i, t} = 1$ for individuals that have not recruited, $u_{i, t} =2$ for live adults, and $u_{i, t} = 3$ for dead adults.
Each survey $j=1, ..., n_j$ occurs in one of the $n_t$ primary periods, and we denote the primary period in which survey $j$ takes place as $t_j$.
Multiple primary periods can occur during a summer active season (typically July-August, but in some years as early as May and as late as September).
We set the year containing the first primary period to $y_{t = 1} = 1$, and generally $y_t$ represents the year containing primary period $t$.
Years increment by one until the final year of the mark recapture efforts, which we denote $n_y$: $y \in \{1, 2, ..., n_y\}$.
We assume that within a primary period, the state of each individual does not change (i.e., individuals do not recruit into the adult population, gain or lose Bd infection, or die).
This assumption is justified by the short time intervals between surveys within primary periods, in cases where primary periods contain multiple secondary periods.

Live individuals are detected with probability $p_j$, which is modeled as:

$$p_j = \text{logit}^{-1}(X_j^{(p)} \beta^{(p)}),$$

where $X_j^{(p)}$ is a known row vector and $\beta^{(p)}$ an unknown parameter vector.
Not recruited and dead individuals are never captured.
We bundle these assumptions about the observation probabilities for survey $j$ into an emission matrix $\Omega_j$:

$$
\Omega_j =
\begin{blockarray}{ccc}
  \text{Not detected} & \text{Detected} \\
\begin{block}{(cc)c}
  1 & 0 & \text{Not recruited} \\
  1 - p_j & p_j & \text{Alive} \\
  1 & 0 & \text{Dead} \\
\end{block}
\end{blockarray}
$$

The state transition matrix $\Psi_{t, i}$ contains the probabilities of individual $i$ transitioning from state $u_{i, t}$ (rows) to $u_{i, t+1}$ (columns) between primary period $t$ and $t+1$.
For non-introduced (i.e., naturally recruited) individuals, this matrix is given by:

$$
\Psi_{t, i} =
\begin{blockarray}{cccc}
  \text{Not recruited} & \text{Alive} & \text{Dead} \\
\begin{block}{(ccc)c}
  1 - \lambda_t & \lambda_t & 0 & \text{Not recruited} \\
  0 & \phi_t & 1 - \phi_t & \text{Alive} \\
  0 & 0 & 1 & \text{Dead} \\
\end{block}
\end{blockarray},
$$ where $\lambda_t$ is the probability of recruiting in time $t$ and $\phi_t$ is the probability of survival in time $t$.

For introduced individuals, which have deterministic recruitment (i.e., they recruit when introduced), the state transition matrix is given by:

$$
\Psi_{t, i} =
\begin{blockarray}{cccc}
  \text{Not recruited} & \text{Alive} & \text{Dead} \\
\begin{block}{(ccc)c}
  1 - I_{t, i} & I_{t, i} & 0 & \text{Not recruited} \\
  0 & \phi_t & 1 - \phi_t & \text{Alive} \\
  0 & 0 & 1 & \text{Dead} \\
\end{block}
\end{blockarray},
$$ where $I_{t, i}$ is a known indicator function for whether individual $i$ was introduced in primary period $t$.

We allow recruitment probabilities to vary in time via random effects, such that:

$$\lambda_t = \text{logit}^{-1}(\alpha^{(\lambda)} + \epsilon^{(\lambda)}_t),$$

where $\alpha^{(\lambda)}$ is an intercept parameter and $\epsilon^{(\lambda)}_t$ is an adjustment for time $t$.

Survival probabilities also vary in time, and as a function of known covariates:

$$\phi_t = \text{logit}^{-1}(X^{(\phi)}_t \beta^{(\phi)} + \epsilon^{(\phi)}_t),$$

where $X^{(\phi)}_t$ is a row vector of known covariates, $\beta^{(\phi)}$ is a column vector of unknown coefficients, and $\epsilon^{(\phi)}_t$ is an adjustment for time $t$.

To complete the specification of the Bayesian model, we specify priors for all unknown parameters.
The recruitment parameter priors were specified as follows:

$$\alpha^{(\lambda)} \sim N(0,1),$$ $$\sigma^{(\lambda)} \sim N_+(0,1),$$ $$\epsilon^{(\lambda)}_t \sim N(0, \sigma^{(\lambda)}),$$ for periods $t=1, ..., T$.
Here $N$ represents the normal distribution and $N_+$ the half normal distribution with positive support.

Survival parameter priors were specified similarly as:

$$\beta^{(\phi)} \sim N(0, 1),$$ $$\sigma^{(\phi)} \sim N_+(0, 1),$$ $$\epsilon^{(\phi)}_t \sim N(0, \sigma^{(\phi)}),$$ for time $t=1, ..., T$.

The detection model coefficient vector also received a standard normal prior $\beta^{(p)} \sim N(0, 1)$.

We computed the likelihood of each individual capture history using the forward algorithm, and we estimated the latent states using the forward-backward algorithm [@zucchini2009; @joseph2018].

All of the code to specify and fit the model in Stan is available in the open source mrmr package [@joseph2019].

The joint distribution of the resulting model can be written as follows:

```{=tex}
\begin{multline*}
[\alpha^{(\lambda)}, \sigma^{(\lambda)}, \epsilon^{(\lambda)}_{1:T}, \beta^{(\phi)}, \sigma^{(\phi)}, \epsilon^{(\phi)}_{1:T}, \beta^{(p)} \mid \pmb Y] \propto \prod_{i=1}^M [Y_i \mid \alpha^{(\lambda)}, \epsilon^{(\lambda)}_{1:T}, \beta^{(\phi)}, \epsilon^{(\phi)}_{1:T}, \beta^{(p)}] \times \\
\prod_{t=1}^T [\epsilon_t^{(\lambda)} \mid \sigma^{(\lambda)}] [\epsilon_t^{(\phi])} \mid \sigma^{(\phi)}] [\sigma^{(\lambda)}] [\sigma^{(\phi)}] [\alpha^{(\lambda)}] [\beta^{(\phi)}] [\beta^{(p)}],
\end{multline*}
```
where $\pmb Y$ is an $M \times T$ detection matrix, and $Y_i$ the capture history of individual $i$.

#### Among-site survival modeling

The objective of this analysis is to describe the influence of site, cohort, and individual level characteristics on post-translocation frog survival.
By modeling survival estimates obtained from site-specific mrmr CMR analyses, we are in effect conducting an among-site meta-analysis.
Although it would theoretically be possible to estimate survival covariate effects in a joint CMR model that integrates capture histories across all sites, this was impractical due to computational requirements of the CMR models (namely, run time and memory).

We used Bayesian generalized linear mixed models to investigate predictors of survival among sites.
The response $y_i$ is binary, representing a point estimate of whether individual $i$ survived in the year following translocation.
We generated these point estimates by rounding the posterior median of 1-year post-introduction survival for each individual (from site-specific mrmr CMR models) and modeled the data using a Bernoulli distribution:

$$y_i \sim \text{Bernoulli}(p_i),$$

where $p_i$ is the probability of survival.

We modeled variation in probabilities as follows:

$$\text{logit}(p_i) = \alpha + X_i \pmb \beta + \pmb \nu_{g[1:N]},$$

where $\alpha$ is an intercept, $X_i$ is a length $K$ row vector of predictors, $\pmb \beta$ is a column vector of predictor effects, and $\pmb \nu$ a vector of group level random effects.
Here $g[i]$ refers to the group $g$ containing individual $i$, and we estimate an adjustment for each of the $G$ groups ($\nu_1, ..., \nu_G$).

These models were fit using the `stan_glmer()` function in the rstanarm package, with default priors described below [@rstanarm2022].
These priors are vague, but include data-dependent scaling as follows to account for different input variable scales.
However, because we standardized all predictor variables similarly to have equal variance (by centering and dividing by twice the sample standard deviation), the resulting priors are identical.
Specifically, we have:

$$\alpha \sim \text{Normal}(0, 2.5),$$ and

$$\beta_k \sim \text{Normal}(0, 5),$$

for $k=1, ..., K$ where $K$ is the number of predictor variables.

The default prior for group level adjustments $\nu_1, ..., \nu_G$ in rstanarm is a zero-mean Gaussian, where the covariance matrix is constructed from a correlation matrix with an LKJ prior, and a vector of variance parameters -- the decomposition of variance prior with unit regularization, concentration, shape, and scale parameters [@lewandowski2009].

We drew posterior samples using Dynamic Hamiltonian Monte Carlo in Stan, with four parallel chains, each run for 10,000 iterations, discarding the first half of each chain as warm-up draws [@rstanarm2022].
We used Rhat statistics and trace plots to verify convergence.
We considered models with different subsets of fixed and random effects, and used approximate leave-one-out cross validation to identify the best model [@vehtari2016].

### Population viability modeling

#### Incorporating yearly variability in vital rates

We computed yearly survival probabilities for translocated adults $\sigma_{A_T}$ and naturally recruited adults $\sigma_{A_R}$ from the posterior distribution of individual state trajectories derived from mrmr CMR models.
Although we observed yearly variability in adult survival within a population, the magnitude of this variability was small compared to among-population variability (@fig-translocation-survival).
Thus, we did not include yearly within-population variability in adult survival in this analysis.
However, within a population there was substantial yearly variability in the successful recruitment of adults, greater than what we would expect from Poisson variability around a mean value.
Therefore, we allowed for yearly variability in the probability of year-1 juvenile survival and recruitment (additional details provided in **Estimating model parameters** below).
We also could have included environmental stochasticity in year-2 juvenile survival and recruitment $\sigma_{J_2}$, but our elasticity and sensitivity analysis (see below, **Model analysis and simulation**) showed that this parameter had little effect on host growth rate relative to $\sigma_{J_1}$ (@fig-viability-supp SI).

#### Estimating model parameters

The baseline parameter values for the model and how they were estimated are given in @tbl-param_values.
Parameters $\sigma_{A_T}$ and $\sigma_{A_R}$ were extracted directly from our CMR models (see **Materials and Methods - Frog populaton recovery - Estimation of frog survival and abundance** and **Supporting Information - Frog population recovery - CMR model structure** for details).
For populations where we had a sufficient number of PIT-tagged, naturally-recruited adults, we observed that $\sigma_{A_T}$ and $\sigma_{A_R}$ could be notably different, with $\sigma_{A_R} > \sigma_{A_T}$ (@fig-compare_surv_probs SI).
For populations lacking sufficient numbers of naturally-recruited adults, we were unable to directly estimate $\sigma_{A_R}$, and instead set $\sigma_{A_R} = \sigma_{A_T}$.

To estimate the $\sigma_{A_R}$, we used the posterior distribution of predicted true states for naturally-recruited individuals (1=not recruited, 2=alive, 3=dead, as described in **Supporting Information - Frog population recovery - CMR model structure**), then calculated the posterior probability of individuals surviving between consecutive primary periods, conditional on being alive in the first primary period (e.g., given a value of 2 (alive) in the first primary period, how often was the value still 2 (alive) in the next primary period compared to 3 (dead)) .
This yielded posterior distributions for survival probabilities between primary periods.
However, because the time interval between primary periods differed, the survival probabilities between different consecutive primary periods were not directly comparable.
To address this, we converted the survival probabilities between each consecutive primary period to per day death rates, propagating the uncertainty from the posterior distributions.
We then took a weighted average of these death rates, weighted by the time interval between primary periods, to get the average per day death rate over the entire CMR survey.
We converted this per day death rate $d$ back to a yearly survival probability using $\exp(-d \times 365 \text{ days})$.
We used the same procedure for $\sigma_{A_T}$ such that our estimates of average yearly survival probability were comparable between $\sigma_{A_R}$ and $\sigma_{A_T}$.

#### Model analysis and simulation

We performed four analyses on our model.
First, we considered a deterministic version of our model with no yearly heterogeneity in year-1 juvenile survival and recruitment probability $\sigma_{J_1}$, and calculated the predicted long-run growth rate $\lambda$ of a population for different values of $\sigma_{A_R}$ and $\sigma_{J_1}$.
We then fixed $\sigma_{J_1} = 0.09$ and calculated the predicted growth rate of our 12 populations.

Second, we performed a local elasticity and sensitivity analysis on $\lambda$ with respect to parameters $\sigma_{L_1}$, $\sigma_{L_2}$, $\sigma_{L_3}$, $\sigma_{J_1}$, $\sigma_{J_2}$, $\sigma_{A_R}$, $F$, $p_{L_2}$, $p_{J_1}$ to determine how small changes in these parameters could influence the long-run deterministic growth rate of populations (@fig-viability-supp SI).
Note that $\lambda$ is equally or more sensitive to changes in adult survival ($\sigma_{A_R}$) and survival of year-1 juveniles ($\sigma_{J_1}$) than it is to other model parameters, providing additional justification for focusing on variation in $\sigma_{A_R}$ and $\sigma_{J_1}$ in our viability analyses.

Third, we defined a version of the model with demographic and environmental stochasticity, where environmental stochasticity was represented by among-year variability in $\sigma_{J_1}$.
We used this model to simulate a one-time introduction of 40 translocated adult frogs.
We ran this simulation 1000 times for each population and computed the probability of a population becoming extinct after 50 years given the observed parameter values and environmental stochasticity in $\sigma_{J_1}$.
We varied the mean recruitment probability $\sigma_{J_1}$ from 0 and 0.25 and drew values of $\sigma_{J_1}$ each year from a beta distribution with a dispersion parameter of $\phi = 2$ (when $\sigma_{J_1} = 0.5$ and $\phi = 2$ the beta distribution is uniform between 0 and 1).
Using different values of $\phi$ does not qualitatively change the existence of distinct extinction dynamics between populations with $\sigma_{A_R} < 0.5$ and those with $\sigma_{A_R} > 0.5$.
However, increasing yearly variability in $\sigma_{J_1}$ increases extinction risk for all populations.
For example, if we set $\phi = 0.001$, such that in a given year essentially either all year-1 juveniles survive or all of them die, populations with $\sigma_{A_R} > 0.5$ need to have $\sigma_{J_1}$ greater than 0.2 to have a 50-year extinction probability of less 50%.
Because we do not pit tag juveniles, we do not have CMR estimates for $\sigma_{J_1}$ or $\phi$.
However, based on qualitative and semi-quantitative field observations over 25 years, a value of $\sigma_{J_1} = 0.25$ in the presence of Bd is probably a reasonable estimate for many populations.
Thus, we expect our model predictions to be conservative with regards to population recovery.

Finally, we assessed whether our stochastic model could reproduce observed trajectories of population recovery.
We focused on population 70550 because this was our longest CMR time series for a translocated population and because this population shows evidence of substantial post-translocation increases in adult abundance associated with population establishment and recovery.
We simulated our model for 16 years, repeating the simulation 50,000 times.
For each run and each year, we drew $\sigma_{J_1}$ from a uniform distribution between 0 and 1 (or equivalently a beta distribution with mean 0.5 and $\phi = 2$).
Using Approximate Bayesian Computing and rejection sampling [@kosmala2016], we identified the top 2% of trajectories (i.e., 1000 trajectories) that minimized the sum of squared errors between the observed and predicted data.
The yearly $\sigma_{J_1}$ values associated with these "best" trajectories represented an approximate posterior distribution [@beaumont2010].
Using these best fit trajectories, we assessed whether our model could qualitatively describe the patterns of recovery in the observed data for population 70550.

### Frog genome evolution

#### Study design

To gain insights into the role of evolution in the development of resistance by MYL frogs, we compared frog exomes sampled in naive versus recovering populations.
Comparing populations with different infection histories allowed larger sample sizes and replication across the landscape.
The alternative approach of comparing samples from the same populations before and after Bd exposure isn't feasible in this system because Bd arrived in most MYL frog populations decades ago and population persistence/recovery is rare and unpredictable.
As a result, samples from recovering populations collected before and after Bd exposure are not available and are unlikely to be available in the future.

Our study design, in which we compared frog genomes in naive and recovering populations, required sampling populations across the range of MYL frogs in the southern Sierra Nevada.
This is due to the fact that very few MYL frog populations remain in the naive state, and those that do are scattered across a wide latitudinal range, from Yosemite National Park in the north to southern Kings Canyon National Park in south.
To minimize potential confounding effects caused by known variation in frog genotypes across latitude [@byrne2023], we selected sampling sites such that both population types were represented across similar latitudinal ranges (@fig-allelefrequencies).

#### GO analysis

In **Results-Frog evolution in response to Bd**, we describe the stringent set of outlier variants (identified using a Bonferroni-corrected p-value of 0.01).
A liberal set of outlier variants, identified using a Bonferroni-corrected p-value of 0.05, included 38 outliers (35 SNPs and 3 INDELS) from 30 distinct genes across 16 contigs.
We used this liberal set to determine if any GO biological functions, molecular functions, or cellular processes were overrepresented.
To do this, we retrieved the BLAST hits and mapped GO terms for each gene in our targeted transcriptome.
We then conducted a statistical overrepresentation test (Fisher's exact test) using Blast2GO [@conesa2005] to compare the 30 unique outlier genes to the complete set of genes in our target transcriptome.
We repeated this process for the set of 35 genes located in the 9 shared regions of the *F~ST~* and $\pi_{diff}$ splined windows.

#### Genetic diversity

To characterize general patterns of genetic diversity between naive and recovering populations, we conducted three analyses.
First, we calculated heterozygosity for each sampled frog using VCFtools [@danecek2011].
Second, to characterize genome-wide patterns of diversity, we used VCFtools to calculate nucleotide diversity ($\pi$) in 100kb sliding windows along the genome for each population.
Third, we calculated average $\pi$ per population within each of the 9 outlier windows identified in the splined window analysis.
We compared average $\pi$ per population between naive and recovering population types, and also for each naive and recovering population in Yosemite.
The latter comparison focused on Yosemite because these populations are the most relevant to the translocated populations described in the paper (see results below for additional details).

Average individual-level heterozygosity and genome-wide population-level $\pi$ were similar between the naive and recovering groups (@fig-violinplot-heterozy SI; @fig-boxplot-genomewide-pi-by-pop SI).
Within each of the 9 outlier windows, average $\pi$ shows considerable variation between populations and no obvious patterns between the naive and recovering populations (@fig-boxplot-pi-by-windowpop SI).
For the four populations in Yosemite, within each of the 9 outlier windows, average $\pi$ per population was generally higher in the 3 recovering populations than in the 1 naive population (@fig-ave-pi-per-site-yose SI).
ADD CONCLUDING SENTENCE OR TWO THAT SUMMARIZE THE SIGNIFICANCE OF THESE RESULTS?

\newpage

### Figures

\newpage

![Map showing the locations of translocated and donor MYL frog populations in Yosemite National Park (park boundary indicated by gray polygon). Symbol shapes indicate the donor population used for each translocation site, and 5-digit numbers identify each donor and translocation site. To obscure the exact locations of populations, random noise was added to all point coordinates. Inset map shows the location of Yosemite within California. In both maps, elevation is indicated by the colored hillshade layer (dark green = lowest elevation, white = highest elevation).](figures/map_translocation_points.png){#fig-yosemap width="400"}

\newpage

![For frogs translocated to each of the 12 recipient sites, Bd loads for the period immediately prior to translocation versus during the 1-year period after translocation. Bd loads are expressed as the number of ITS1 copies per skin swab, as estimated by qPCR of the Bd ITS1 region. Box plots show medians, first and third quartiles, largest and smallest values within 1.5x interquartile range, and values outside the 1.5x interquartile range. Samples sizes are provided immediately above the x-axis. The red dashed horizontal line indicates the approximate load above which severe disease is typically observed [@vredenburg2010]. This threshold (5.8 ITS1 copies - on a log~10~ scale) is commonly exceeded during Bd epizootics, but is exceeded only rarely in recovering populations [@briggs2010].](figures/bdload_beforeafter.png){#fig-bdload-beforeafter width="630"}

\newpage

![Results from the among-site meta-analysis, showing that Bd load is not an important predictor of post-translocation frog survival. Depicted distributions are the estimated posterior density curves and shaded 95% uncertainty intervals for the intercept and all predictor variables from the best model. In the Bayesian framework in which the model was developed, variables are considered important predictors if the associated uncertainty interval does not overlap zero (indicated by the dashed red line). Predictor variables shown on the y-axis are defined as follows: length = frog size, snow_t = winter severity in the year of translocation (measured on April 1), snow_t1 = winter severity in the year following translocation (measured on April 1), day = day of year on which a translocation was conducted, bd_load = Bd load, elevation = site elevation, order_first = within-site translocation order, sex_male = frog sex, donor_70567 and donor_72996 = donor population.](figures/mcmc_areas_m1d.png){#fig-survival-postdens width="500"}

\newpage

![Comparison of average yearly adult survival probabilities for adults translocated to each of 3 sites versus adults that were naturally recruited at each site (as a result of reproduction by translocated frogs). In contrast to @fig-translocation-survival, these are not survival probabilities from the first year following translocation, but instead represent averaged survival probabilities across multiple years and cohorts. Points are median estimates and error bars give the 95% uncertainty intervals around the estimates, accounting for yearly variation in survival probabilities. All estimates were derived using the mrmr package, and the methods for calculation are described in **Supporting Information - Population viability modeling - Estimating model parameters**.](figures/compare_surv_probs.jpg){#fig-compare_surv_probs width="580"}

\newpage

![Sensitivity analysis of the stage-structured MYL frog model. A. Elasticity of $\lambda$ with changes in nine parameters: $\sigma_{A_R}$ (yearly survival probability of naturally recruited adults), $F$ (number of eggs produced by a female frog in a year that successfully hatch), $\sigma_{J_1}$ (yearly probability of survival of year-1 juveniles that also affects recruitment), $\sigma_{J_2}$ (yearly probability of survival and recruitment of year-2 juveniles), $p_{L_2}$ (probability that year-2 tadpoles metamorph into juveniles), $p_{J_1}$ (probability that year-1 juveniles mature to adults), $\sigma_{L_1}$ (survival probability of year-1 tadpoles), $\sigma_{L_2}$ (survival probability of year-2 tadpoles), and $\sigma_{L_3}$ (survival probability of year-3 tadpoles). B. Sensitivity of $\lambda$ with changes in the same nine parameters. Note that $F$ has different units than the other parameters, which is why its sensitivity is low. Sensitivities of all other parameters are comparable. We did not include $p_{L_1}$ as this parameter is generally 1 across the populations in this study (Table S1) -- tadpoles spend at least a year as tadpoles before metamorphosing. Elasticity and sensitivity is calculated at the default parameter values for each population and $\sigma_{J_1}$ = 0.09.](figures/pop_viability_figures_for_supp.jpg){#fig-viability-supp width="650"}

\newpage

![Results from the analysis of individual variants, showing putative signatures of selection in recovering MYL frog populations. (A) PCA calculated from binary SNPs showing the genomic relationship of samples. Numeric labels and colors match those in @fig-allelefrequencies. Populations 1-7 are *R. sierrae* and populations 8 and 9 are *R. muscosa*. (B) qqplot showing observed and expected p-values for 148,307 SNPs and INDELS. Dashed line shows p-value that identifies outliers. (C) Manhattan plot showing p-value for each SNP. SNPs are sorted by genomic position and contigs are sorted by size. Red line shows p-value that identifies outliers. Outlier SNPs above this threshold are highlighted and labeled. Bold labels indicate the presence of at least one non-synonymous SNP in that gene.](figures/pca_qq_manhattan.png){#fig-selectionresults width="550"}

\newpage

![Synteny plot showing conserved gene order in *Xenopus tropicalis* and *Rana muscosa* for the outlier region containing MHC Class I Classical and MHC Class III gene regions. The plot was created with SimpleSynteny [@veltri2016] using *Xenopus tropicalis* Chromosome 8 (NC_030684.2, genbank accession GCA_000004195.4) and *Rana muscosa* Contig19. Asterices indicate the location of SNP outliers in the TCF19 and VARS1 genes. Gap sizes for each contig representation are labeled.](figures/synteny_figure.png){#fig-synteny-plot width="650"}

\newpage

![Violin plots showing individual heterozygosity for the Bd-naive and recovering populations included in the frog evolution study. Individual data points are represented by their corresponding site number (from @fig-allelefrequencies).](figures/violin_plot_heterozy_by_group.png){#fig-violinplot-heterozy width="500"}

\newpage

![For each study population in the frog evolution study (from @fig-allelefrequencies), boxplots showing nucleotide diversity ($\pi$) within 100kb sliding windows along the genome. The y-axis has been log~10~-transformed for display purposes. Sites are color-coded as "Bd-naive" or "recovering", and vertical dashed lines show the genetic breaks between frog populations as described in @fig-selectionresults A SI).](figures/boxplot_genomewide_pi_by_pop.png){#fig-boxplot-genomewide-pi-by-pop width="550"}

\newpage

![For each of the 9 outlier windows identified by the overlapping splined windows for *F~ST~* and $\pi$, boxplots showing average $\pi$ for naive and recovering populations. The corresponding gene annotations for each window are as follows: (1) C6, C7, (2) DDX10, ZBTB24, (3) SULTR-like, TRAF3IP2, VGLL3, EXOC1, (4) FLOT1, TUBB, MDC1, CCHCR1, TCF19, HSP70, LSM2, VARS1, (5) GCC2, CFAP251, PEG10, (6) ERO1A, GVINP1, (7) PPP1R12A, TSPAN4, PAWR, MFRP, MAX, PPP6R3, (8) C6H5ORF22, PKS6, BSPRY, MPV17, and (9) CAD, ATRAID, GPN1.](figures/boxplot_pi_by_windowpop.png){#fig-boxplot-pi-by-windowpop width="500"}

\newpage

![For each study population located in Yosemite National Park (populations 1-4 shown in Site Map), average $\pi$ for the 9 outlier windows (panels A-I) identified in the splined window analysis. Populations 1 and 4 were the source of frogs for 20 of the 24 translocations and the donor population for the remaining four translocations is located adjacent to populations 3 and 4. Bars are color-coded by population type (naive, recovering), and gene annotations within each window are listed above each panel. Gene regions described in the paper are in panel A (Complement C6,C7), panel E (GVINP1), and panel F (extended MHC region that includes two SNPs identified by the GEMMA analysis in genes TCF19 and VARS1, indicated by an asterisk). In the Site Map, the four populations included in panels A-I are color-coded by population type, and dashed lines delineate the genetic clusters as defined in [@fig-selectionresults SI, panel A]. <!--# Update caption based on feedback from Allie and Bree -->](figures/ave_pi_per_site_yose.png){#fig-ave-pi-per-site-yose width="650"}

\newpage

### Tables

| **Parameter**                                                             | **Value**            | **Source**                                                                        |
|:-------------------------|:------------------|:--------------------------|
| $\sigma_{L_1}$, Yearly survival probability of year-1 tadpoles            | 0.7                  | Estimated from field data, observations, natural history knowledge                |
| $\sigma_{L_2}$, Yearly survival probability of year-2 tadpoles            | 0.7                  | Estimated from field data, observations, natural history knowledge                |
| $\sigma_{L_3}$, Yearly survival probability of year-3 tadpoles            | 0.7                  | Estimated from field data, observations, natural history knowledge                |
| $\sigma_{J_1}$, Yearly survival probability of year-1 juveniles           | Varies yearly        | Varies. Bounds estimated from field data, observations, natural history knowledge |
| $\sigma_{J_2}$, Yearly survival probability of year-2 juveniles           | 0.5                  | Estimated from field data, observations, natural history knowledge                |
| $\sigma_{A_R}$, Yearly survival probability of naturally recruited adults | Varies by population | Estimated from CMR studies                                                        |
| $\sigma_{A_T}$, Yearly survival probability of translocated adults        | Varies by population | Estimated from CMR studies                                                        |
| $p_{L_1}$, Probability of a year-1 tadpoles remaining as a tadpoles       | 1                    | Estimated from field data, observations, natural history knowledge                |
| $p_{L_2}$, Probability of a year-2 tadpoles remaining as a tadpoles       | 0.25                 | Estimated from field data, observations, natural history knowledge                |
| $p_{J_1}$, Probability of a year-1 juvenile remaining as a juvenile       | 0.25                 | Estimated from field data, observations, natural history knowledge                |
| $p_F$, Probability of a adult female reproducing in a year                | 0.5                  | Could be as high at 1, based on field observations                                |
| $F$, Number of surviving eggs produced by an adult female                 | 100                  | From observations of captive frogs                                                |

: Description and values of parameters used in the model. All survival probabilities are in the presence of the fungal pathogen Bd. {#tbl-param_values}

\newpage

### Datasets

Dataset S1.
Set of liberal SNP and INDEL outlier variants (Bonferroni corrected p-value \< 0.05), identified via GEMMA.
file: gemma_outliers_all.csv

Dataset S2.
Set of strict SNP and INDEL outlier variants (Bonferroni corrected p-value \< 0.01), identified via GEMMA.
Additional information includes variant location within the gene (predicted_gene_loc) and whether the variant is synonymous or nonsynonymous (predicted_effect_AA).
file: gemma_outliers_strict_freq.csv

Dataset S3.
Description of overlapping *F~ST~* and $\pi_{diff}$ outlier windows, as identified in the splined window analysis.
The outlier window on chromosome 19 (column name = chr_num_sorted) is included twice because one *F~ST~* outlier window overlapped with two $\pi_{diff}$ outlier windows.
file: spline_window_shared_outliers.csv

Dataset S4.
Annotation information for all genes within each of the overlapping outlier windows in dataset S3.
file: spline_window_gene_details

### References
