---
format: docx
editor: visual
bibliography: translocation.bib
csl: pnas.csl
---

<!--# Intended outlet is PNAS. Preferred length of research reports is 6 pages, but up to 12 pages is allowed. A standard 6-page article is approximately 4,000 words, 50 references, and 4 medium-size graphical elements (i.e., figures and tables) -->

## Title: Reintroduction of resistant frogs allows recovery in the presence of a lethal fungal disease

<!--# Title should be less than 135 characters -->

### Authors:

Roland A. Knapp^1,2^\*, Mark Q. Wilber^3^, Allison Q. Byrne^4,5^, Maxwell B. Joseph^6,7^, Thomas C. Smith^1,2^, Andrew P. Rothstein^4,5^, Rob L. Grasso^8^, Erica Bree Rosenblum^4,5^

<!--# Exact author order to be determined -->

### Affiliations:

^1^Sierra Nevada Aquatic Research Laboratory, University of California, Mammoth Lakes, CA, 93546

^2^Earth Research Institute, University of California, Santa Barbara, CA, 93106-3060

^3^School of Natural Resources, University of Tennessee Institute of Agriculture, Knoxville, TN, 37996

^4^Department of Environmental Science, Policy, and Management, University of California - Berkeley, Berkeley, CA, 94720-3114

^5^Museum of Vertebrate Zoology, University of California - Berkeley, Berkeley, CA, 94720-3160

^6^Earth Lab, University of Colorado, Boulder, CO, 80303

^7^NCX, San Francisco, CA, 94115

^8^Resources Management and Science, Yosemite National Park, El Portal, CA, 95318

\*Corresponding author: Sierra Nevada Aquatic Research Laboratory, 1016 Mount Morrison Road, Mammoth Lakes, CA 93546, phone: 760.647.0034, email: roland.knapp\@ucsb.edu

**Preprint Server:**

**Classification:** Biological Sciences - Ecology

### Abstract

<!--# No more than 250 words -->

### Significance Statement

<!--# Less than 120 words. Explain the significance of the research at a level understandable to an undergraduate-educated scientist outside their field of specialty.  -->

### Introduction

Emerging infectious diseases are increasing [@daszak2000] and pose a severe threat to human, livestock, and wildlife populations.
In wildlife populations, disease is responsible for dramatic declines, extirpations, and even extinctions in mammals, birds, amphibians, echinoderms, and other taxa [@hewson2014; @samuel2015; @scheele2019; @cunningham2021].
Although the dramatic and immediate nature of disease-induced declines in wildlife has led to intensive focus on documenting and understanding disease impacts [@langwig2015], this information is generally insufficient to allow the development of effective methods to recover disease-impacted species [@scheele2021; @fellers2007].
Importantly, some populations previously decimated by disease have subsequently recovered to varying degrees [@newell2013; @voyles2018; @knapp2016] and might provide critical insights for species conservation efforts.
Unfortunately, given the nascent nature of wildlife recoveries in the presence of disease, we know far less about the mechanisms driving population recovery [@brannelly2021; @russell2020] and how these mechanisms can be leveraged to re-establish host populations across large spatial scales [@mendelson2019].
At a time when novel infectious diseases are emerging at an unprecedented rate [@fisher2012], this knowledge gap has far-reaching consequences for global biodiversity.

Host evolution following disease-induced declines is one mechanism by which hosts can recover [@carlson2014; @searle2020].
Evolutionary rescue describes natural host evolution whereby rapid evolutionary changes originating from variation within a host population reverse ongoing population declines [@carlson2014].
In host-pathogen systems, evolutionary rescue is more likely to occur when hosts experience repeated selection events (e.g., through re-infections), there is a persistent pathogen reservoir or frequency-dependent infection dynamics, and populations experience increased per capita birth rates at small population sizes [@golas2021].
Evolutionary changes following disease-induced declines are known from a diversity wildlife populations [@savage2016; @epstein2016; @gignoux-wolfsohn2021; @holland2022], but linking these selective responses directly to population recoveries, as necessary for evolutionary rescue, remains rare.
Important for conservation action, selection that promotes recovery of focal populations can also provide sources of individuals with increased resistance and/or tolerance (hereafter, "resistance") to reestablish populations extirpated following disease emergence [e.g., @mendelson2019].
To date, the use of resistant individuals for species recovery is rare [see @joseph2018 for a notable example], and studies that evaluate its potential utility are critically needed.
<!--# Other examples in addition to Joseph and Knapp (2018)? In other taxa besides amphibians? -->

There are four conditions that will often determine the feasibility of using individuals from populations that are recovering following disease-induced declines to reestablish extirpated populations.
First, populations that survived following disease arrival have evolved increased resistance.
Second, increased resistance contributes to higher individual fitness in the presence of disease and allows positive population growth.
Third, the fitness benefits gained through selection for increased resistance manifest beyond the natal environment and are maintained across generations, allowing for the establishment of populations reintroduced into novel environments.
And fourth, reintroduced populations show long-term growth and persistence in a landscape of disease.
To date, few of these conditions are documented for any single species impacted by disease, and as a result the reintroduction of resistant hosts as a means to recover species of conservation concern remains largely an untested idea [@scheele2021].

The mountain yellow-legged (MYL) frog, comprised of the sister species *Rana muscosa* and *Rana sierrae* [@vredenburg2007], is emblematic of the unprecedented global declines of amphibians caused by the recently-emerged amphibian chytrid fungus [*Batrachochytrium dendrobatidis,* @scheele2019].
Once the most common amphibian in the high elevation portion of the Sierra Nevada mountains [California, USA, @grinnell1924], this frog has disappeared from more than 90% of its historical range during the past century [@vredenburg2007].
Due to the severity of its decline and the increasing probability of extinction, both species are now listed as "endangered" under the U.S.
Endangered Species Act.
Although this decline was initiated by the introduction of non-native trout into the once-fishless portion of the Sierra Nevada [@bradford1989; @knapp2000], the arrival of *B. dendrobatidis* (Bd) in the 1960s and its subsequent spread [@vredenburg2019] caused additional widespread population extirpations [@vredenburg2010; @rachowicz2006].
These Bd-caused declines are fundamentally different from the fish-caused declines because removal of fish populations is feasible and results in the rapid recovery of frogs [@knapp2007; @vredenburg2004].
In contrast, Bd appears to persist in habitats even in the absence of amphibian hosts [@walker2007], and therefore represents a permanent alteration of invaded ecosystems that amphibians will need to overcome to reestablish populations.

Despite the catastrophic impact of Bd on MYL frogs in which most Bd-naive populations are driven to extirpation following Bd arrival [@vredenburg2010], some populations have persisted after epizootics [@briggs2010] and are now recovering [@knapp2016].
Frogs in these recovering populations show reduced susceptibility to Bd infection, consistent with selection for more resistant genotypes [@knapp2016].
Furthermore, exploratory efforts to reintroduce frogs collected from recovering populations to sites from which frogs were previously extirpated indicate that population establishment is possible despite ongoing Bd infection [@joseph2018].
Here, we report the results of (i) a greatly expanded MYL frog reintroduction effort, (ii) population viability modeling, and (iii) an analysis to evaluate genomic signatures of selection following MYL frog exposure to Bd.
Collectively, the results show that MYL frogs generally meet the four conditions for reintroduction feasibility outlined above, and provide a novel example of how the reintroduction of resistant frogs can allow the large-scale recovery of this disease-impacted amphibian.
These results may have broad implications for the many species worldwide that are threatened with extinction by emerging diseases.

### Results

#### Frog evolution in response to Bd

To identify putative genes under selection and potentially related to increased frog resistance to Bd, we used exome capture methods to compare genomes of frogs in populations with different histories of Bd exposure.
Specifically, we compared genomes of frogs sampled in four populations that have never experienced a Bd-caused epizootic ("naive") versus in five populations that experienced a Bd epizootic during the past several decades and have since recovered to varying degrees (@fig-selectionresults A: SI) [@knapp2016; @vredenburg2010].
These "recovering" populations persist in an enzootic state [@briggs2010], characterized by high Bd prevalence and Bd infection intensities ("load") in adults that are typically well below the level expected to cause frog mortality [@knapp2011, @vredenburg2010].
<!--# Modify text/remove text here/elsewhere to avoid repetition of this. --> Bd-exposure histories of these populations are based on a decade or more of data from VES and Bd surveillance using skin swabbing [e.g., @knapp2016].
The comparison of frog genomes between naive and recovering populations is not as powerful as comparing samples collected from populations before and after Bd exposure, but because Bd arrived in most MYL frog populations decades ago and population persistence/recovery is rare and unpredictable, no such samples are available and are unlikely to be available in the future.
As such, the current study design is the best possible under the circumstances.
To detect regions of the genome under selection, we used a multivariate linear mixed model to compare genomes of frogs from naive and recovering populations, and created "liberal" and "stringent" sets of outlier variants using a Bonferroni-corrected significance level of 0.05 and 0.01, respectively.

The samples clustered into three separate groups in principal component space, each containing at least one population from both the naïve and persistent groups (@fig-selectionresults B: SI).
The liberal set of variants contained 38 outliers (35 SNPs and 3 structural variants) from 30 distinct genes across 16 contigs (@fig-selectionresults C, D: SI).
One gene in this group does not have an associated annotation.
No biological functions, molecular functions, or cellular processes were significantly overrepresented in the set of 29 annotated outlier genes.
The stringent set contained 11 outlier SNPs from 7 distinct genes across 4 contigs (@fig-selectionresults C, D: SI).
For these SNPs, frequency differences between the persistent and recovering groups ranged from 41% to 86%.
Most SNPs in the stringent set showed frequency differences in only a subset of the sampled populations (@fig-allelefrequencies A, B) and the patterns may be confounded by frog relatedness and/or population structure despite the methods we used to account for these factors.
However, the outlier SNP in the RIN3 gene showed consistent differences in frequencies across all populations (@fig-allelefrequencies C), suggestive of strong selection at this locus associated with exposure of frogs to Bd.
<!--# Allie should add a brief description of the function of RIN3 here. Also, emphasize key messages related to selection, resistance. And, of course, rework any of the text as necessary. -->

<!--# Figure edits:  selectionresults: Correctly locate population 7, capitalize panel labels, label liberal and stringent outlier sets. outlier map: Rearrange panels to put RIN3 last? Relocate pie charts to avoid the perception that all recovering populations are west of naive populations, add A, B, C labels to panels. -->

#### Frog translocations

```{r}
#| include: false
#| code-fold: true

library(readr)
library(dplyr)
library(magrittr)
library(stringr)
library(lubridate)
d1 <- read_csv(here::here("data", "clean", "frog_translocation_final.csv"))

transno_min <- d1 %>% 
  distinct(site_id, date) %>% 
  group_by(site_id) %>% 
  count() %>% 
  ungroup() %>% 
  summarize(min(n))

transno_max <- d1 %>% 
  distinct(site_id, date) %>% 
  group_by(site_id) %>% 
  count() %>% 
  ungroup() %>% 
  summarize(max(n))

transfrogs_max <- d1 %>% 
  group_by(site_id, date) %>% 
  count() %>% 
  ungroup() %>% 
  summarize(max(n))

transfrogs_min <- d1 %>% 
  group_by(site_id, date) %>% 
  count() %>% 
  ungroup() %>% 
  summarize(min(n))

bdload_min <- d1 %>% 
  filter(!is.na(bd_load)) %>% 
  group_by(site_id) %>% 
  mutate(bdload_ave = median(log10(bd_load + 1))) %>% 
  ungroup() %>% 
  summarize(min(bdload_ave))

bdload_max <- d1 %>% 
  filter(!is.na(bd_load)) %>% 
  group_by(site_id) %>% 
  mutate(bdload_ave = median(log10(bd_load + 1))) %>% 
  ungroup() %>% 
  summarize(max(bdload_ave))

bdprev_max <- d1 %>% 
  filter(!is.na(bd_load)) %>% 
  mutate(bdload01 = if_else(bd_load > 0, 1, 0)) %>% 
  group_by(site_id) %>% 
  summarize(prev = sum(bdload01)/n()) %>% 
  ungroup() %>% select(prev) %>% 
  slice_max(prev, n = 1, with_ties = FALSE)

bdprev_min <- d1 %>% 
  filter(!is.na(bd_load)) %>% 
  mutate(bdload01 = if_else(bd_load > 0, 1, 0)) %>% 
  group_by(site_id) %>% 
  summarize(prev = sum(bdload01)/n()) %>% 
  ungroup() %>% select(prev) %>% 
  slice_min(prev, n = 1, with_ties = FALSE)

files <- fs::dir_ls(here::here("data", "raw", "cmr-analysis", "survey"))  
surveys <- read_csv(files, id = "path") %>% 
  filter(survey_date < "2023-01-01") %>% 
  select(-path)
rm(files)

surveys_yr_median <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr") %>% 
  distinct(site_id, primary_period, .keep_all = TRUE) %>% 
  group_by(site_id) %>% 
  count(yr) %>% ungroup() %>% 
  summarize(median(n))

surveys_yr_max <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr") %>% 
  distinct(site_id, primary_period, .keep_all = TRUE) %>% 
  group_by(site_id) %>% 
  count(yr) %>% ungroup() %>% 
  summarize(max(n))

surveys_yr_min <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr") %>% 
  distinct(site_id, primary_period, .keep_all = TRUE) %>% 
  group_by(site_id) %>% 
  count(yr) %>% ungroup() %>% 
  summarize(min(n))

robust_year_median <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr" & secondary_period > 1) %>% 
  group_by(site_id) %>% 
  summarize(max_2period = max(secondary_period)) %>% ungroup() %>% 
  summarize(median(max_2period))

robust_year_min <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr" & secondary_period > 1) %>% 
  group_by(site_id) %>% 
  summarize(max_2period = max(secondary_period)) %>% ungroup() %>% 
  summarize(min(max_2period))

robust_year_max <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr" & secondary_period > 1) %>% 
  group_by(site_id) %>% 
  summarize(max_2period = max(secondary_period)) %>% ungroup() %>% 
  summarize(max(max_2period))
```

Reintroductions of *R. sierrae* were conducted in Yosemite National Park (California, USA) as part of an effort to reestablish populations of this endangered species.
These reintroductions involved the collection of adult frogs from large, Bd-positive donor populations that have recovered in recent years following Bd-caused declines [@knapp2016] and translocating them to nearby recipient sites [@joseph2018].
We refer to these reintroductions as "translocations" [@seddon2014].
During the period `r min(d1$year)`--`r max(d1$year)`, we conducted `r n_distinct(d1$site_id, d1$date)` translocations of frogs to `r n_distinct(d1$site_id)` recipient sites (`r transno_min`--`r transno_max` translocations per site; @fig-yosemap SI) <!--# How to restart numbering for figures in SI, add "S" prefix? -->.
Each translocated cohort included `r transfrogs_min` to `r transfrogs_max` frogs collected from one of `r n_distinct(d1$donor)` donor populations (@fig-yosemap SI).
Estimates of frog survival during the one year following translocation indicate that survival was highly variable between recipient sites, but relatively constant within recipient sites (for the subset of sites that received multiple translocations; @fig-translocation-survival).
As such, the average survival of frogs in the first translocation to a site was an important predictor of survival of frogs in subsequent translocations to that site (@fig-transsurvival-postdens SI).
This suggests an important effect of site characteristics and/or (site \* frog) interactions on frog survival.
In addition, 1-year survival was higher for frogs translocated later in the study period than earlier (@tbl-viability-earlylate SI) <!--# Should be Table 1, not Table 2. Rearrange table order in manuscript. -->, which we suggest resulted primarily from our improved ability over time to choose recipient sites with higher habitat quality for *R. sierrae*.
Finally, Bd loads changed little between immediately prior to translocation versus during the one year after translocation (@fig-bdload-beforeafter SI), and loads were nearly always well below the level indicative of severe chytridiomycosis and associated frog mortality (5.8 ITS copies on a log~10~ scale) [@vredenburg2010; @joseph2018].
<!--# Add median Bd load, proportion below 5.8 copies threshold? --> <!--# The Methods -- Translocation section contains text that repeats similar results. Edit to reduce repetition. --> <!--# To allow comparison with other translocation studies, consider adding information about the proportion of translocated populations at which evidence of successful reproduction was observed in the 1-2 years following translocation. -->

To identify site and frog-specific drivers of post-translocation survival, we developed a multilevel model in a Bayesian framework.
Our specific interest was whether Bd load had an important effect on adult survival, as would be expected if the effect of frog resistance was insufficient to overcome the negative effect of Bd.
The best model of 1-year frog survival identified several important predictors, but Bd load at the time of translocation was not among them (bd_load; @fig-survival-postdens SI).
Important predictors included winter severity in the year following translocation (snow_t1), site elevation (elevation), and donor population (donor_72996; @fig-cond-effects).
Males had somewhat higher survival than females, but this effect was only marginally important (sex_male; @fig-cond-effects).
In addition to Bd load, effects of frog size (length), winter severity in the year of translocation (snow_t), day of year on which a translocation was conducted (day), and translocation order (order_first) were all unimportant (@fig-survival-postdens SI).
Collectively, these results show that Bd load is not a predictor of the survival of translocated frogs, consistent with frogs in recovering populations having evolved sufficient resistance to minimize the adverse effects of Bd infection.
In addition, the low-moderate Bd loads that characterized frogs following translocation and relatively high survival of frogs in the majority of more-recently translocated cohorts suggests that resistant frogs from recovering populations can maintain the benefits of resistance in non-natal habitats.
Finally, in three locations where longer mark-recapture time series allowed us to assess the survival of adults recruited to the population (i.e., progeny of translocated individuals), naturally recruited adults had equivalent or higher survival probabilities than the originally translocated adults (@fig-compare_surv_probs SI).
Therefore, frog resistance in non-natal habitats is maintained across generations.

<!--# Mark W: Regarding the above paragraph, what evidence do we have of "tolerance"? Are Bd loads significantly reduced in populations compared to epizootic levels? Can we convincingly state that the likely mechanism is more consistent with resistance or tolerance?  I ask because it would be nice to just use one of these terms in the paper and not get caught up in any resistance vs. tolerance discussions with reviewers.  Roland: Have a look at this paragraph of Knapp et al. PeerJ 2022: https://peerj.com/articles/12712/#p-61. We argue that there is evidence for resistance and tolerance. -->

<!--# Drive home the point that i) Acquired, heritable resistance in donor populations could promote the long-term viability of translocated populations and b) but survival is still highly dependent on environment independent of Bd. -->

#### Population viability modeling

To test whether the observed yearly survival probabilities of resistant adults in translocated populations was sufficient to promote long-term viability, we built a stage-structured matrix model that captured known frog demography and included demographic and environmental stochasticity.
We parameterized the model using mark-recapture data and known life history values in this system (@tbl-param_values SI).
Given observed yearly adult survival probabilities and a baseline yearly juvenile recruitment probability of 0.3 (i.e., the probability that, following survival, juveniles successfully recruit to the adult class), our stage-structured model predicted that six of twelve translocated populations should experience a long-run growth rate $\lambda$ greater than 1 in the presence of Bd (@fig-viability A; median predicted $\lambda$ ranges from 1.15-1.36 for these six populations).
When we reduced the probability of successful recruitment $\omega$ to 0.1 (a highly conservative estimate), long-run growth rate of five of these six populations was still greater than 1 (@fig-viability A).
In contrast, for the other six translocated populations where yearly adult survival was less than 0.5, our model predicted that it was unlikely that long-run growth rate in these populations was greater than 1, even when the probability of successful recruitment to the adult life stage was near 0.4 (@fig-viability A).
A sensitivity analysis of long-run growth rate $\lambda$ to recruitment probability $\omega$ and yearly adult survival probability of naturally recruited individuals $\sigma_{A_R}$ showed that $\lambda$ was most sensitive to small changes in $\sigma_{A_R}$ (@fig-viability-supp A).

Given demographic stochasticity (i.e., probabilistic births and deaths of individuals in a population) and environmental variability in the probability of juvenile recruitment to the adult class $\omega$ (the transition that we expect to be the most subject to environmental stochasticity in the presence of Bd), our model predicted that, following a single introduction of 40 adult individuals into a population, the six populations with the highest adult survival probabilities had 50-year extinction probabilities of less than 1 (@fig-viability B).
Therefore, even under conservative assumptions about recruitment probability $\omega$, these populations had some probability of persistence over 50 years following a single introduction.
For these six populations, as $\omega$ increased from 0.1 to 0.3, 50 year extinction probability was less 0.5, indicating strong potential for long-term persistence in the presence of Bd and environmental variability in recruitment success (@fig-viability B).
In contrast, for the six populations where $\sigma_{A_R} < 0.5$, extinction probability over 50 years was always predicted to be 1.
To test the validity of our model predictions, we demonstrated that our stochastic model could describe the general recovery trajectory of our translocated population with the longest survey history (@fig-viability-supp B; population 70550, surveyed for 16 years).
Overall, our model demonstrates that given observed yearly adult survival probabilities of resistant frogs in non-natal locations, 50% of our translocated populations have a high probability of population growth and long-term viability in the presence of Bd.
Moreover, five of the seven populations translocated after 2013 were predicted to have long-term viability, compared to only one of five populations translocated prior to 2013.
Thus, there is some trend in our improved ability to select appropriate habitat to promote long-term viability of translocated, resistant populations.
<!--# Mark here: Just figured I would point this out. This is trending in the right direction, but there isn't strong statistical evidence that these two proportions differ.  I tried to phrase this carefully. -->

### Discussion

<!--# Consider broadening the scope of this sentence and the paragraph to be developed from it to encompass all of our findings -->

TO BE WRITTEN.

We provide five lines of evidence that evolutionary rescue is a strong candidate mechanism for emergence of this resistance.

1.  Epidemiological conditions in MYL-Bd systems X, Y, and Z are conducive to evolutionary rescue.
2.  Differences in allele frequencies exist between adjacent Bd-naive and recovering populations.
3.  There is a shared selective response across recovering populations.
4.  Changes in population growth rate due to evolution extend beyond the natal environment.
5.  In populations established via translocation of frogs from recovering populations, Bd load is not an important predictor of post-translocation frog survival (in contrast to important relationship between Bd load and frog survival during Bd-caused epizootics).

<!--# Discuss effect of donor population on survival of translocated frogs. Differences in Bd resistance? -->

<!--# The following point was made in the original Introduction outline but ultimately not addressed there: Translocation of individuals from putatively resistant populations to sites that were formerly occupied but from which populations were extirpated by disease provides a way to disentangle the contributions of natal environment and host genetics on host persistence and recovery. -->

<!--# Translocation "success" often defined as evidence of reproduction by translocated individuals (see IUCN and related references in notes). We used a much higher and more conservative standard to assess success, i.e.,  -->

### Materials and Methods

#### Frog evolution in response to Bd

To identify putative genes under selection that are linked to reduced frog susceptibility to Bd-caused lethal and sub-lethal effects, we used exome capture methods to compare genomes of frogs in populations with different histories of Bd exposure.
Specifically, we compared genomes of frogs from populations that have never experienced a Bd-caused epizootic ("Bd-naive") versus frogs from populations that experienced a Bd epizootic during the past several decades and have since recovered to varying degrees ["recovering": @knapp2016; @vredenburg2010].
These latter populations persist in an enzootic state [@briggs2010], characterized by high Bd prevalence and Bd infection loads in adults that are typically well below the level expected to cause frog mortality [@knapp2011; @vredenburg2010].
Bd-exposure histories of these populations are based on a decade or more of data from VES and Bd surveillance using skin swabbing [e.g., @knapp2016].

DNA samples were collected via buccal swabbing [@broquet2007] of xx-xx frogs from four Bd-naive populations and five recovering populations.
These populations represented a wide latitudinal range in the southern Sierra Nevada, from northern Yosemite National Park to northern Sequoia National Park (@fig-selectionresults A).
To minimize potential confounding effects caused by variation in frog genotypes across latitude [e.g., @rothstein2020], we selected sampling sites such that both population types were represented across similar latitudinal ranges.

#### Frog translocations

##### Field methods

Translocations of *R. sierrae* were conducted in Yosemite National Park (California, USA) as part of an effort to reestablish populations of this endangered species.
We identified donor populations from which adult frogs ($\geq$ 40 mm snout-vent length) could be collected using several years of visual encounter surveys (VES) and skin swab collections [@knapp2016], as well as landscape genetic information [@poorten2017].
Results indicated that these populations contained several hundred *R. sierrae* adults and hundreds or thousands of tadpoles, and that frog abundances were recovering from previous Bd-caused declines [@knapp2016].
As is typical for recovering MYL frog populations, Bd prevalence in the donor populations was high (`r round(bdprev_min, 2)`--`r round(bdprev_max, 2)`) and Bd infection intensity ("load", expressed as median by site: `r round(bdload_min, 2)`--`r round(bdload_max, 2)`) was two or more orders of magnitude below the level at which frog mortality is expected (log~10~(load) $\approx$ 5.78 ITS copies per swab)[@vredenburg2010; @joseph2018].
Recipient sites to which frogs were translocated were chosen based on previous *R. sierrae* presence (determined from VES and/or museum records) or characteristics that suggested high quality habitat for this species [@knapp2005].

During each translocation, adult frogs were collected from the donor population and measured, weighed, swabbed, and PIT tagged.
Frogs were transported to the recipient site either on foot or via helicopter.
Following release, translocated populations were visited approximately once per month during the summer active season (median number of visits per summer = `r round(surveys_yr_median)`; range = `r surveys_yr_min`--`r surveys_yr_max`) and assessed using diurnal capture-mark-recapture (CMR) surveys and VES.
CMR surveys allowed quantification of adult survival, adult population size, and recruitment of new adults, and VES provided estimates of tadpole and subadult abundance.
During 2006-2012, CMR surveys were conducted on a single day (primary period) per site visit, during which all habitats were searched repeatedly for adult frogs.
Frogs were captured using handheld nets, identified via their PIT tag (or tagged if they were untagged), measured, weighed, swabbed, and released at the capture location.
During 2013-2022, we generally used a robust design in which all habitats were searched during several consecutive days (median number of secondary periods per primary period = `r round(robust_year_median)`; range = `r robust_year_min`--`r robust_year_max`), and frogs were processed as described above.
However, when the number of frogs detected on the first survey day was low, we conducted only a single-day CMR survey.
Within a primary period, frogs that were captured during more than one secondary period were measured, weighed, and swabbed only during the first capture, and during subsequent captures were only identified and released.
Frog skin swabs were analyzed using standard Bd DNA extraction and qPCR methods.
For details of survey and swab analysis methods, see @joseph2018.

During each site visit, we conducted VES either immediately before CMR surveys or during the first day of CMR surveys.
VES was conducted by walking the entire water body perimeter, first 100 m of each inlet and outlet stream, and any fringing ponds and wetlands, and counting all *R. sierrae* tadpoles and subadults.
These *R. sierrae* life stages have high detectablility, and counts are highly repeatable and provide estimates of relative abundance [@knapp2000].

##### Estimation of frog survival and abundance

We estimated survival of translocated frogs, recruitment of new frogs into the adult population, and adult population size using a Bayesian open-population Jolly-Seber mark-recapture model with known additions to the population (i.e., translocated cohorts), as implemented by the mrmr package [@joseph2019] and using R Statistical Software [v4.4.4, @rsoftware2022].
The model tracks the states of *M* individuals that comprise a superpopulation made up of real and pseudo-individuals [see @joseph2018 for details].
The possible states of individuals include "not recruited", "alive", and "dead".
The possible observations of individuals include "detected" and "not detected".
We assume that individuals that are in the "not recruited" or "dead" states are never detected (i.e., there are no mistakes in the individual PIT tag records).
For all models, we used mrmr defaults for priors, number of chains (4), and warmup and post-warmup iterations (2000 for each).
We evaluated convergence of the Markov chain Monte Carlo (MCMC) algorithm using trace plots and Gelman-Rubin statistics (Rhat).
Code used to construct CMR datasets and fit models for each translocated population is available from the following GitHub repository: [https://github.com/SNARL1/cmr-analysis](https://github.com/SNARL1/cmr-analysis.).

##### Predictors of post-translocation frog survival

To identify important predictors of frog survival following translocation, we developed multilevel models in a Bayesian framework.
We performed all analyses using the rstanarm package [@rstanarm2022] and R Statistical Software [v4.4.4, @rsoftware2022] .
We used default, weakly informative priors.
Starting models included all relevant population-level effects and, when data contained possible hierarchical structure, we included group-level effects in the full model.
Within each analysis, we evaluated model fit using visualizations of leave-one-out ("LOO") probability integral transformations [@gelman2013, @gabry2019], and compared fits of models with and without group-level effects, and with differing group-level effects using LOO cross-validation [@vehtari2016].
Both approaches utilized the loo package [@vehtari2022].

We conducted two analyses.
In the first, we addressed the more general question of whether survival of frogs in the first translocation to a site predicted the survival of frogs in subsequent translocations.
Group-level effects included site_id or translocation_id.
The second analysis focused specifically on identifying predictors of post-translocation frog survival using a wide range of variables describing characteristics of frogs, sites, and translocations [@fig-survival-postdens] SI.
Group-level effects included site_id, translocation_id, or translocation_id nested within site_id.
In both analyses, the response variable, probability of 1-year frog survival, was rounded to integer values to produce a binomial outcome, and all models used a bernoulli family.
For all models, we used default priors, four chains, and 5000 iterations each for warmup and post-warmup.
We checked convergence of the MCMC algorithm using trace plots and Rhat.
Code to create the datasets and fit models is available from the following GitHub repository: <https://github.com/SNARL1/translocation>.

##### Changes in Bd load following translocation

To assess the magnitude of changes in Bd load on frogs following translocation, we compared Bd loads measured before versus after translocation.
Before-translocation loads were quantified using skin swabs collected from all to-be-translocated frogs at the donor site on the day before or the day of the translocation.
After-translocation Bd loads were based on all swabs collected from translocated frogs at the recipient site in the year of and the year following translocation.
Individual frogs and their associated Bd loads were included in the dataset only if frogs were captured at the recipient site at least once during the year 0-1 period.

To quantify changes in Bd load, we used a workflow similar to that described above (in "Predictors of post-translocation frog survival").
Briefly, the response variable was before-translocation load and the predictor variable was after-translocation load.
We compared models with and without group-level effects, and group-level effects included pit_tag_id, pit_tag_id nested within site_id, or pit_tag_id nested within translocation_id.
All models used a negative binomial family.
Priors, chains, iterations, and checks were as described above.
Code to create the dataset and fit models is available from the following GitHub repository: <https://github.com/SNARL1/translocation>.

#### Population viability modeling

##### Model description

To determine the implications of observed yearly adult survival on the long-term population viability of putatively resistant frogs in non-natal habitats, we developed a population model for *R. sierrae*.
Our central question was: how does the magnitude and variation in observed adult survival probability across translocated populations affect the long-term persistence probability of populations?
We developed a model that tracked seven state variables of an *R. sierrae* population: the density of translocated adults ($A_T$), the density of adults naturally recruited into the population ($A_R$), the density of first year larvae ($L_1$), the density of second year larvae ($L_2$), the density of third year larvae ($L_3$), the density of first year juveniles ($J_1$), and the density of second year juveniles ($J_2$).
We divided adults into two classes $A_T$ and $A_R$ because there was evidence that the survival probability of translocated adults and naturally recruited adults differed.

We modeled the dynamics of these seven state variables using a discrete-time, stage-structured model where a time step is one year.
The dynamics are given by

$$
\begin{bmatrix}
L_1 \\
L_2 \\
L_3 \\
J_1 \\
J_2 \\
A_R \\ 
A_T
\end{bmatrix}(t + 1) = 
\begin{bmatrix}
  0 & 0 & 0 & 0 & 0 & \sigma_{A_R} p_F F & \sigma_{A_T} p_F F \\
  \sigma_{L1} p_{L1} & 0 & 0 & 0 & 0 & 0 & 0 \\
  0 & \sigma_{L2} p_{L2} & 0 & 0 & 0 & 0 & 0\\
  \sigma_{L1} (1 - p_{L1}) & \sigma_{L2} (1 - p_{L2}) & \sigma_{L3} & 0 & 0 & 0 & 0 \\
  0 & 0 & 0 & \sigma_{J1} p_{J1} & 0 & 0 & 0 \\
  0 & 0 & 0 & \sigma_{J1} (1 - p_{J1}) \omega & \sigma_{J2} \omega & \sigma_{A_R} & 0 \\
  0 & 0 & 0 & 0 & 0 & 0 & \sigma_{A_T} 
\end{bmatrix} \begin{bmatrix}
L_1 \\
L_2 \\
L_3 \\
J_1 \\
J_2 \\
A_R \\
A_T \\
\end{bmatrix}(t) 
$$

The parameters in this model are yearly survival probability $\sigma_{\cdot}$, where the "$\cdot$" indicates a particular state variable, the probability that a female frog reproduces in a given year $p_F$, the number of eggs produced by a female frog in a year that successfully hatch $F$, the probability of a first year tadpole remaining as a tadpole $p_{L1}$, the probability of a second year tadpole remaining as a tadpole $p_{L2}$, and the probability of a first year juvenile remaining as a juvenile $p_{J1}$.
The parameter $\omega$ gives the probability of a juvenile successfully recruiting to an adult.

In this model we ignore density-dependent recruitment because we were interested in the growth of the population from an initial reintroduction and whether this growth was sufficient to prevent extinction over 50 years following the introduction.
We also did not directly consider the dynamics of Bd in this model.
We made this decision because i) translocated populations are infected with Bd at high prevalence [@joseph2018], and ii) host density does not seem to play a significant role in multi-year Bd infection dynamics in this system [@wilber2022].
Thus, ignoring Bd infection dynamics and instead assuming all host vital rates are in the presence of high Bd prevalence significantly simplifies the model without much loss of realism.

##### Model analysis

###### Incorporating yearly variability in vital rates

We extracted yearly survival probabilities for translocated adults $\sigma_{A_T}$ and naturally recruited adults $\sigma_{A_R}$ from the CMR model used to estimate population-level parameters of translocated populations (see "Estimation of adult survival and population size" above).
Although we observed yearly variability in adult survival *within* a population, the magnitude of this variability was small compared to *among* population variability.
Thus, we did not include yearly within-population variability in adult survival in this analysis.

However, within a population there was substantial yearly variability in the successful recruitment of adults, greater than what we would expect from Poisson variability about a mean value.
Therefore, we allowed for yearly variability in the probability of successful recruitment $\omega$ (additional details provided in "Estimating model parameters" below).

###### Estimating model parameters

The baseline parameter values for the model and how they were estimated are given in @tbl-param_values : SI.
Parameters $\sigma_{A_T}$ and $\sigma_{A_R}$ were extracted directly from our CMR model described previously in this study.
For populations where we had a sufficient number of pit-tagged, naturally recruited adults, we observed that $\sigma_{A_T}$ and $\sigma_{A_R}$ could be notably different, with $\sigma_{A_R} > \sigma_{A_T}$.
For populations lacking any naturally-recruited adults, we were unable to directly estimate $\sigma_{A_R}$, and instead set $\sigma_{A_R} = \sigma_{A_T}$.

###### Model analysis and simulation

We performed four analyses on our model.
First, we considered a deterministic version of our model with no yearly heterogeneity in probability of successful recruitment $\omega$, and calculated the predicted long-run growth rate $\lambda$ of a population for different values of $\sigma_{A_R}$ and $\omega$.
We then fixed $\omega = 0.3$ and calculated the predicted growth rate of our 12 populations.

Second, we performed a local elasticity analysis on $\lambda$ with respect to parameters $\omega$, $\sigma_{A_R}$, and $F$ to determine how small changes in these parameters could influence the long-run deterministic growth rate of populations.

Third, we defined a version of the model with demographic and environmental stochasticity, where environmental stochasticity was represented by among-year variability in $\omega$.
We used this model to simulate a one-time introduction of 40 translocated adult frogs.
We ran this simulation 1000 times for each population and computed the probability of a population becoming extinct after 50 years given the observed parameter values and environmental stochasticity in $\omega$.
We varied the mean recruitment probability $\omega$ from 0.15 and 0.4 and drew values of $\omega$ each year from a beta distribution with a dispersion parameter of $\phi = 2$ (when $\omega = 0.5$ and $\phi = 2$ the beta distribution is uniform between 0 and 1).

Finally, we assessed whether our stochastic model could reproduce observed trajectories of population recovery.
We focused on population 70550 because this was our longest CMR time series for a translocated population and because this population shows evidence of substantial post-translocation increases in adult abundance associated with population establishment and recovery.
We simulated our model for 16 years, repeating the simulation 50,000 times.
For each run and each year, we drew $\omega$ from a uniform distribution between 0 and 1 (or a beta distribution with mean 0.5 and $\phi = 2$).
Using Approximate Bayesian Computing and rejection sampling (@kosmala2016), we identified the top 2% of trajectories (i.e., 1000 trajectories) that minimized the sum of squared errors between the observed and predicted data.
The $\omega$ values associated with these "best" trajectories represented an approximate posterior distribution [@beaumont2010].
Using these best fit trajectories, we assessed whether our model could qualitatively describe the patterns of recovery in the observed data for population 70550.

### Acknowledgements and funding sources

### References

::: {#refs}
:::

### Figures

<!--# Discuss referencing and in-text generation of figures.  We don't want to re-run the population viability modeling figures each time we compile the manuscript.  A makefile could be a good solution. Caching figures could be another option (https://quarto.org/docs/computations/caching.html) -->

![Allele frequencies of the outlier SNPs for each of the sampled naive and recovering *R. sierrae* populations. The results for RIN3 are consistent with strong selection on this gene following the arrival of Bd, and suggests a role for RIN3 in frog resistance against Bd infection and associated population recovery.](figures/outlier_map_draft_figure.png){#fig-allelefrequencies}

![Median survival of each cohort of translocated frogs to the 12 recipient sites included in this study. Sites are arranged along the x-axis using the average of the median 1-year survival per translocation at each site. Frog survival was estimated using capture-mark-recapture methods. Dot colors indicate the donor population from which frogs in each translocated cohort were collected. Error bars show the 2.5% and 97.5% posterior quantiles (i.e., 95% uncertainty intervals). When multiple points exist for a site, points/error bars are slightly offset to avoid overlap.](figures/translocation_survival_bysiteid.png){#fig-translocation-survival width="450"}

![Conditional effects of the important predictors of 1-year frog survival: (A) elevation of recipient site, (B) winter severity in the year following translocation, (C) donor population, and (D) sex. In (A) and (B), blue lines are medians and gray ribbons are 95% uncertainty intervals. In (C) and (D), box plots show medians, first and third quartiles, largest and smallest values within 1.5x interquartile range, and values outside the 1.5x interquartile range.](figures/cond_effects_plot.png){#fig-cond-effects}

![A population viability analysis of translocated *R. sierrae* that quantifies the long-run growth rate and extinction probability of frog populations. A. The predicted long-run growth rate $\lambda$ for different values of yearly adult survival probability $\sigma_{A_R}$ and probability of recruitment $\omega$, given the parameterized, deterministic model for translocated *R. sierrae*. Colored points show the predicted $\lambda$ values for the twelve translocated populations when probability of recruitment into the adult class is $\omega = 0.3$ (given by the dashed line). B. The predicted fifty-year extinction probabilities of the twelve translocated populations given demographic stochasticity, environmental variability in $\omega$, and different mean values of $\omega$. The dashed line shows where $\omega = 0.3$, for comparison to A.](figures/pop_viability_figures_for_manuscript_v2.jpg){#fig-viability}

![Sensitivity analysis and empirical validation of the stage-structured *R. sierrae* model. A. The elasticity of $\lambda$ with changes in three parameters: $\sigma_{A_R}$, $\omega$, and *F (*number of eggs produced by a female frog in a year that successfully hatch). Elasticity is calculated at the default parameter values for each population and $\omega$ = 0.3\$. B. The 100 simulated trajectories (black lines) that most closely matched the observed abundance trajectory of adult amphibians at site 70550 (light orange).](figures/pop_viability_figures_for_supp.jpg){#fig-viability-supp}

<!-- Left panel color palette is 12-color color-blind friendly, but is not the most friendly color palette to look at.  We can discuss. Roland says: Maybe try a palette that you like better and for those colors that will be hard for eyeballs like mine to distinguish, we can either tweak the colors or add diagnonal lines (ggpattern::geom_bar_pattern). -->

## Supporting Information

### References

### Figures

![<!--# Add first sentence that provides a brief overview of the entire figure. --> (a) Map of sampling locations showing Bd-naive populations in red and persistent populations in blue. (b) PCA calculated from binary SNPs showing genomic relationship of samples. Labels match sampling location in (a) and are colored as in (a). (c) qqplot showing observed and expected p-values for 148,307 SNPs and indels as calculated in GEMMA. Dashed line shows Bonferroni corrected p-value for threshold of 0.01. (d) Manhattan plot showing p-values for each SNP as calculated by GEMMA. SNPs are sorted by genomic position and Contigs are sorted by size. Red line shows Bonferroni corrected p-value for threshold of 0.01. Outlier SNPs above this threshold are highlighted and labeled. Bold labels indicate at least one non-synonymous SNP is present in that gene.](figures/mylf_selection_figure_5Dec2022.png){#fig-selectionresults}

![Map showing the locations of translocated and donor populations in Yosemite National Park (park boundary indicated by black polygon). Symbol shapes indicate the donor population used for each translocation site. To obscure the exact locations of populations, random noise was added to all point coordinates. Inset map shows the location of Yosemite within California. In both maps, elevation is indicated by the colored hillshade layer (dark green = lowest elevation, white = highest elevation).](figures/map_translocation_points.png){#fig-yosemap width="450"}

![For the best model describing the average 1-year survival of frogs in the first translocation to a site as a predictor of survival of frogs in all subsequent translocations to that site, estimated posterior density curves and shaded 95% uncertainty intervals for the intercept and single predictor variable. A variable is considered an important predictor if the associated uncertainty interval does not overlap zero (dashed red line).](figures/mcmc_areas_m2b.png){#fig-transsurvival-postdens width="450"}

![For frogs translocated to each recipient site, Bd loads for the period immediately prior to translocation versus during the 1-year period after translocation.](figures/bdload_beforeafter.png){#fig-bdload-beforeafter width="450"}

![For the best model identifying predictors of 1-year frog survival following translocation, estimated posterior density curves and shaded 95% uncertainty intervals for the intercept and all predictor variables. Variables are considered important predictors if the associated uncertainty interval does not overlap zero (dashed red line).](figures/mcmc_areas_m1d.png){#fig-survival-postdens width="450"}

![A comparison of the average yearly adult survival probabilities for translocated adults versus adults that were naturally recruited to a population as a result of reproduction by translocated frogs. In contrast to @fig-translocation-survival, these are not survival probabilities from the first year following translocation, but averaged survival probabilities across multiple years. Points are median estimates and error bars give the 90% uncertainty intervals around the estimates. All estimates are derived from the mrmr package [see @joseph2018 for details]. <!--# Should we use 95% intervals to be consistent with other analyses?  -->](figures/compare_surv_probs.jpg){#fig-compare_surv_probs}

### Tables

```{r}
#| label: tbl-viability-earlylate
#| tbl-cap: "Proportion of populations translocated early versus late in the study period (< 2013 or >= 2013), respectively) for which the probability of extinction over 50 years was 1 or < 1 (viability = 0 or 1, respectively)."
#| echo: FALSE
#| message: FALSE

viability_table <- read_csv(here::here("data", "clean", "frog_translocation_final.csv")) %>% 
  distinct(site_id, year) %>% 
  group_by(site_id) %>% 
  filter(year == min(year)) %>% 
  mutate(site_id = as.character(site_id),
         viability = if_else(site_id %in% c(70414, 70370, 70134, 70641, 70619, 70505), 0, 1),
         period = if_else(year < 2013, "early", "late")) %>% 
  group_by(period) %>% 
  count(period, viability) %>% 
  mutate(n_period = sum(n), 
         prop = (n / n_period)) %>% 
  mutate(across(prop, round, 3)) %>% 
  arrange(viability, period)
  
  knitr::kable(viability_table, "pipe", align = "lcccr", caption = ) 
```

| **Parameter**                                                             | **Value**            | **Source**                                                         |
|:---------------------------|:------------------|:-------------------------|
| $\sigma_{L_1}$, Yearly survival probability of year 1 tadpoles            | 0.7                  | Estimated from field data, observations, natural history knowledge |
| $\sigma_{L_2}$, Yearly survival probability of year 2 tadpoles            | 0.7                  | Estimated from field data, observations, natural history knowledge |
| $\sigma_{L_3}$, Yearly survival probability of year 3 tadpoles            | 0.7                  | Estimated from field data, observations, natural history knowledge |
| $\sigma_{J_1}$, Yearly survival probability of year 1 juveniles           | 0.25                 | Estimated from field data, observations, natural history knowledge |
| $\sigma_{J_2}$, Yearly survival probability of year 2 juveniles           | 0.5                  | Estimated from field data, observations, natural history knowledge |
| $\sigma_{A_R}$, Yearly survival probability of naturally recruited adults | Varies by population | Estimated from mark-recapture studies                              |
| $\sigma_{A_T}$, Yearly survival probability of translocated adults        | Varies by population | Estimated from mark-recapture studies                              |
| $p_{L_1}$, Probability of a year 1 tadpoles remaining as a tadpoles       | 1                    | Estimated from field data, observations, natural history knowledge |
| $p_{L_2}$, Probability of a year 2 tadpoles remaining as a tadpoles       | 0.25                 | Estimated from field data, observations, natural history knowledge |
| $p_{J_1}$, Probability of a year 1 juvenile remaining as a juvenile       | 0.25                 | Estimated from field data, observations, natural history knowledge |
| $p_F$, Probability of a adult female reproducing in a year                | 0.5                  | Could be as high at 1, based on field observations                 |
| $F$, Number of surviving eggs produced by an adult female                 | 100                  | From observations of captive frogs                                 |
| $\omega$, Probability of juvenile successfully recruiting to an adult     | Varies yearly        | Explored different values of this parameter                        |

: Description and values of parameters used in the model. All survival probabilities are in the presence of the fungal pathogen Bd. {#tbl-param_values}

<!--# Include table of translocation sites, years, frog numbers? -->

<!--# Include Allie's table describing outlier SNPs? -->

### 
