---
bibliography: translocation.bib
csl: pnas.csl
format: pdf
editor: visual
---

<!--# Intended outlet is PNAS. Preferred length of research reports is 6 pages, but up to 12 pages is allowed. A standard 6-page article is approximately 4,000 words, 50 references, and 4 medium-size graphical elements (i.e., figures and tables) -->

## Title: Reintroduction of resistant frogs allows recovery in the presence of a lethal fungal disease

<!--# Title should be less than 135 characters -->

### Authors:

Roland A. Knapp^1,2^\*, Mark Q. Wilber^3^, Allison Q. Byrne^4,5^, Maxwell B. Joseph^6,7^, Thomas C. Smith^1,2^, Andrew P. Rothstein^4,5^, Rob L. Grasso^8^, Erica Bree Rosenblum^4,5^

<!--# Exact author order to be determined -->

### Affiliations:

^1^Sierra Nevada Aquatic Research Laboratory, University of California, Mammoth Lakes, CA, 93546

^2^Earth Research Institute, University of California, Santa Barbara, CA, 93106-3060

^3^School of Natural Resources, University of Tennessee Institute of Agriculture, Knoxville, TN, 37996

^4^Department of Environmental Science, Policy, and Management, University of California - Berkeley, Berkeley, CA, 94720-3114

^5^Museum of Vertebrate Zoology, University of California - Berkeley, Berkeley, CA, 94720-3160

^6^Earth Lab, University of Colorado, Boulder, CO, 80303

^7^NCX, San Francisco, CA, 94115

^8^Resources Management and Science, Yosemite National Park, El Portal, CA, 95318

\*Corresponding author: Sierra Nevada Aquatic Research Laboratory, 1016 Mount Morrison Road, Mammoth Lakes, CA 93546, phone: 760.647.0034, email: roland.knapp\@ucsb.edu

**Preprint Server:**

**Classification:** Biological Sciences - Ecology

### Abstract

<!--# No more than 250 words -->

### Significance Statement

<!--# Less than 120 words. Explain the significance of the research at a level understandable to an undergraduate-educated scientist outside their field of specialty.  -->

### Introduction

Emerging infectious diseases are increasing [@daszak2000] and pose a severe threat to human, livestock, and wildlife populations.
In wildlife populations, disease is responsible for dramatic declines, extirpations, and even extinctions in mammals, birds, amphibians, echinoderms, and other taxa [@hewson2014; @samuel2015; @scheele2019; @cunningham2021].
Although the dramatic and immediate nature of disease-induced declines in wildlife has led to intensive focus on documenting and understanding disease impacts [@langwig2015], this information is generally insufficient to allow the development of effective methods to recover disease-impacted species [@scheele2021; @fellers2007].
Importantly, some populations previously decimated by disease have subsequently recovered to varying degrees [@newell2013; @voyles2018; @knapp2016] and might provide critical insights for species conservation efforts.
Unfortunately, given the nascent nature of wildlife recoveries in the presence of disease, we know far less about the mechanisms driving population recovery [@brannelly2021; @russell2020] and how these mechanisms can be leveraged to re-establish host populations across large spatial scales [@mendelson2019].
At a time when novel infectious diseases are emerging at an unprecedented rate [@fisher2012], this knowledge gap has far-reaching consequences for global biodiversity.

Host evolution following disease-induced declines is one mechanism by which hosts can recover [@carlson2014; @searle2020].
"Evolutionary rescue" describes rapid evolutionary change originating from variation within a host population that reverses ongoing population declines [@carlson2014] <!--# Bree comment: Nuances of word choice -->.
Evolutionary changes following disease-induced declines are known from a diversity of wildlife populations [@savage2016; @epstein2016; @gignoux-wolfsohn2021; @holland2022], but linking these selective responses directly to population recoveries, as necessary for evolutionary rescue, is difficult and remains rare.
In addition to promoting recovery of remnant populations after disease-induced declines, selection that promotes recovery of focal populations can also provide sources of individuals with increased resistance and/or tolerance (hereafter, "resistance") to reestablish populations extirpated following disease emergence [e.g., @mendelson2019].
To date, the use of resistant individuals for species recovery is rare [but see @joseph2018], and studies that evaluate its potential utility are critically needed.
<!--# Other examples in addition to Joseph and Knapp (2018)? In other taxa besides amphibians? -->

There are four conditions that will often determine the feasibility of using individuals from populations that are recovering following disease-induced declines to reestablish extirpated populations.
First, populations that survived following disease arrival have evolved increased resistance.
Second, increased resistance contributes to higher individual fitness in the presence of disease <!--# "disease" or "the pathogen"? Resistance could convey complete prevention of disease, so I often struggle with which word to use -->and allows positive population growth.
Third, the fitness benefits gained through selection for increased resistance manifest beyond the natal environment and are maintained across generations, allowing for the establishment of reintroduced populations.
And fourth, reintroduced populations show long-term growth and persistence.
To date, few of these conditions are documented for any single species impacted by disease, and as a result the reintroduction of resistant hosts as a means to recover species of conservation concern remains largely an untested idea [@scheele2021; @mendelson2019].

The mountain yellow-legged (MYL) frog, comprised of the sister species *Rana muscosa* and *Rana sierrae* [@vredenburg2007], is emblematic of the unprecedented global declines of amphibians caused by the recently-emerged amphibian chytrid fungus [*Batrachochytrium dendrobatidis,* @scheele2019].
Once the most common amphibian in the high elevation portion of the Sierra Nevada mountains [California, USA, @grinnell1924], this frog has disappeared from more than 90% of its historical range during the past century [@vredenburg2007].
Due to the severity of its decline and the increasing probability of extinction, both species are now listed as "endangered" under the U.S.
Endangered Species Act.
Although this decline was initiated by the introduction of non-native trout into the once-fishless portion of the Sierra Nevada [@bradford1989; @knapp2000], the arrival of *B. dendrobatidis* (Bd) in the 1960s and its subsequent spread [@vredenburg2019] caused additional widespread population extirpations [@vredenburg2010; @rachowicz2006].
These Bd-caused declines are fundamentally different from the fish-caused declines because removal of fish populations is feasible and results in the rapid recovery of frogs [@knapp2007; @vredenburg2004].
In contrast, Bd appears to persist in habitats even in the absence of amphibian hosts [@walker2007], and therefore represents a permanent alteration of invaded ecosystems that amphibians will need to overcome to reestablish populations.

Despite the catastrophic impact of Bd on MYL frogs in which most Bd-naive populations are driven to extirpation following Bd arrival [@vredenburg2010], some populations have persisted after epizootics [@briggs2010] and are now recovering [@knapp2016].
Frogs in these recovering populations show reduced susceptibility to Bd infection, consistent with selection for more resistant genotypes [@knapp2016].
Furthermore, exploratory efforts to reintroduce frogs collected from recovering populations to sites from which frogs were previously extirpated indicate that population establishment is possible despite ongoing Bd infection [@joseph2018].

In the current study, we combine results from novel <!--# better adjective than "novel". Mark here: Reads OK without an adjective --> genomic analyses, a 15-year landscape-scale frog reintroduction effort, and detailed population models to provide an unprecedented example of how evolutionary rescue in response to disease allows the successful reestablishment of extirpated MYL frog populations, and that these populations have a high probability of long-term persistence despite ongoing Bd infection.
Specifically, we used exome capture methods to show that MYL frogs have undergone substantial evolutionary change associated with Bd exposure and that may provide a genetic basis for the observed reduced susceptibility/increased resistance to Bd infection observed in recovering populations [@vredenburg2010] <!--# How best to make the jump from "reduced susceptibility" to "increased resistance"? Mark here:  This transition reads fine to me. -->.
We then reintroduce these resistant frogs to 24 sites and describe the dynamics and drivers of population establishment using results from long-term capture-mark-recapture (CMR) surveys.
These data are also used to parameterize population models that extend our inferences of population viability well beyond the multi-year extent of the CMR data.
Collectively, these results provide one of the first examples of how evolutionary rescue and the reintroduction of resistant animals can allow the large-scale recovery of disease-impacted species, with potentially broad implications for the many taxa worldwide that are threatened with extinction by emerging diseases.

<!--# Bree comment: All of this was written before we knew the genomics had something cool to say. But now that the genomics feel a bit more elevated, I think this paragraph should be more bold. For example, we might want more here or in the Discussion to make it clear how the results of each of these sections are super important on their own. And that combining them is really amazing. I think the current tone is appropriately factual, but at some point we need to be louder about the novelty involved and the incredible amount of work that this represents.  This version feels like, we did a,b,c and so MYLF meet the conditions for feasibility (which makes it sound like we've only just gotten to the *starting* point of the study. I think we need to go further than saying they meet the four conditions of a specific conservation action. This is where beefing up the conclusions about evolutionary rescue seem appropriate. Also, genomic analyses typically come last in our papers as a dive into possible mechanistic explanations for observed patterns in the field. I think the order of the primary sections makes sense given the way you set up the argument about evolutionary rescue. That said, the order combined with the tone feels like it's contributing to the sense that the genomics are a box to be checked rather than contributing something exciting to the overall story. Now that Allie has added new analyses, I think we could make a slightly bolder set-up about how the parts fit together to make a surprisingly strong whole -->

<!--# Original pre-edit text from the above paragraph: Here, we report the results of (i) an analysis to evaluate genomic signatures of selection following MYL frog exposure to Bd, (ii) a greatly expanded MYL frog reintroduction effort, and (iii) population viability modeling to assess the long-term persistence of reintroduced populations. Collectively, the results show that MYL frogs generally meet the four conditions for reintroduction feasibility outlined above, and provide a novel example of how the reintroduction of resistant frogs can allow the large-scale recovery of this disease-impacted amphibian. These results may have broad implications for the many species worldwide that are threatened with extinction by emerging diseases. -->

### Results

#### Frog evolution in response to Bd

To assess whether frog populations have displayed an evolutionary response following Bd invasion, we used exome capture methods to compare genomes of frogs in populations with different histories of Bd exposure.
Our goals were to identify putative genes and regions of the genome under selection and assess whether these genes could be related to increased frog resistance to Bd.
Specifically, we compared genomes of frogs sampled in four populations that have never experienced a Bd-caused epizootic ("naive") versus in five populations that experienced a Bd epizootic during the past several decades and have since recovered to varying degrees ("persistent"; @fig-selectionresults A: SI) [@knapp2016; @vredenburg2010].
These recovering populations persist in an enzootic state [@briggs2010], characterized by high Bd prevalence and Bd infection intensities ("load") in adults that are typically well below the level expected to cause frog mortality [@knapp2011, @vredenburg2010].
<!--# Modify text/remove text here/elsewhere to avoid repetition of this. --> Bd-exposure histories of these populations are based on a decade or more of data from visual encounter surveys and Bd surveillance using skin swabbing [e.g., @knapp2016].
Comparing populations with different infection histories allowed larger sample sizes and replication across the landscape.
The alternative approach of comparing samples from the same populations before and after Bd exposure isn't feasible in this system because Bd arrived in most MYL frog populations decades ago and population persistence/recovery is rare and unpredictable.
As a result, samples from recovering populations collected from before and after Bd exposure are not available and are unlikely to be available in the future.

To detect regions of the genome under selection, we used three complementary approaches that together describe the evolutionary response of frog populations to Bd.
First, we used a multivariate linear mixed model to evaluate associations between population type (i.e., naive vs. persistent) and individual variants.
Second, we used a splined window approach to evaluate larger genomic regions showing differentiation between population types <!--# Reverse the order of these two objectives? Mark here:  I am actually liking the ordering of the objectives now. -->.
Finally, we assessed whether putative genes or genomics regions that showed strong evidence of selection were associated with biological functions consistent with possible resistance mechanisms.
<!--# Could we move the following sentence to the methods? --> We then identified outlier variants using a Bonferroni-corrected significance level for single SNPs (liberal = 0.05, stringent = 0.01) , or a threshold for the W-statistic for the splined window analysis (4 standard deviations from the mean).

Regarding our first analysis, individual frogs clustered into three separate groups in principal component space, each containing at least one population from both the naive and recovering groups (@fig-selectionresults B: SI).
The clustering reflected the strong signature of spatial proximity on genetic structure, as has been previously documented with this species <!--# Mark wrote this...confirm validity -->.
We then identified outlier variants using a Bonferroni-corrected significance level for single SNPs (liberal = 0.05, stringent = 0.01).
The liberal set of variants contained 38 outliers (35 SNPs and 3 structural variants) from 30 distinct genes across 16 contigs (@fig-selectionresults C, D: SI).
The stringent set contained 11 outlier SNPs from 7 distinct genes across 4 contigs (@fig-selectionresults C, D: SI, Table 2: SI <!--# add this table -->).
For these SNPs, frequency differences between the persistent and recovering groups ranged from 41% to 86%.
Most SNPs in the stringent set showed frequency differences in only a subset of the sampled populations (@fig-allelefrequencies A, B) and the patterns may be confounded by frog relatedness and/or population structure despite the methods we used to account for these factors.
However, the outlier SNP in the RIN3 gene showed consistent differences in frequencies across all populations (@fig-allelefrequencies C), suggestive of strong selection at this locus associated with exposure of frogs to Bd.
No biological functions, molecular functions, or cellular processes (i.e., GO terms) were significantly overrepresented in the set of 29 annotated outlier genes.

Our second analysis used a splined window to identify regions of the genome under selection.
We found regions of the genome that had both high differentiation (*F~ST~*) and a large difference in nucleotide diversity between naive and persistent populations ($\pi_{diff}$ = $\pi_{naive}$ - $\pi_{recovering}$) <!--# Change to delta pi instead of pi_diff? -->.
We identified 58 outlier windows for F~ST~ and 33 outlier windows for $\pi_{diff}$ (@fig-spline-manhattan A, B).
Of these, 9 regions were outliers for both metrics and two of these shared regions also contained outlier SNPs.
<!--# SNPs from the liberal or strict set? Also "windows" and "regions" are used interchangeably in this paragraph. To avoid confusion, use "windows" throughout? -->.
A total of 33 annotated genes were found in the 9 shared regions and no GO terms were overrepresented within this set of genes (SI Dataset 2).
The strongest signal for a reduction in $\pi$ in recovering populations (highest $\pi_{diff}$), occurred in a 163kb region on Contig19, 12.9Mb upstream of the RIN3 outlier SNP (@fig-spline-manhattan C).
This region contains \~500 SNPs and one annotated gene called "interferon-induced very large GTPase 1-like" (GVINP1).
Additionally, a shared outlier region on Contig1 contained two complement factor genes (C6 and C7).
Interestingly, this region had higher $\pi$ in the recovering populations, consistent with a pattern of balancing selection.
Finally, one shared outlier region on Contig8 contained one outlier SNP (TCF19) and was within 360kb of another outlier SNP (VARS) (@fig-spline-manhattan D).
This region (854kb from the beginning of the outlier windows to the VARS SNP) contained a total of 10 annotated genes.
In *Xenopus*, four of these genes occur in the extended MHC Class I region (FLOT1, TUBB, MDC1,TCF19) and three occur in the extended MHC Class II region (HSP70, LSM2, VARS) [@ohta2006].
<!--# Here or in the discussion we will need to place this result in the context of previous work on MHC in amphibians.  Readers not familiar with amphibian-Bd systems might miss the importance of this result --> Therefore, we conclude that this candidate region under selection is a part of the extended MHC Class I and II complex and shows synteny with other amphibian genomes.

Together, our genomics results show that persistent populations have strong signatures of selection on multiple genes and regions of the genome.
While the functions of all genes and regions showing signatures of selection were not clearly interpretable biologically, regions of the genome associated with resistant phenotypes in other amphibian systems such as the extended MHC Class I, showed strong evidence of selection in persistent MYL frog populations.

<!--# Need summary of importance of genetic results, also need better integration of different selection  results.  Mark W: I took a crack at this to test my naive understanding of the genetic results. Figured I could be a good test case of how a genetically naive reader would summarize the key results, to see if our genomic message is coming across clearly. -->

<!--# Based on Fig. 2, there are numerous outlier windows. How were the specific windows mentioned above chosen? Seems important to mention to avoid the perception that mentioned patterns were chosen haphazardly or cherry-picked based on unknown criteria. -->

<!--# In reading through these results, I am not seeing a compelling theme that emerges from all of the different results, both within and between the SNP and splined window sections. The results show differences in various metrics between population types, but are these changes large or small? I lack the context to understand that. Similarly, is the magnitude of differences consistent with strong or weak selection? In addition to the above, this section needs a concluding summary of the key genetic results. -->

#### Frog translocations

```{r}
#| include: false
#| code-fold: true

library(readr)
library(dplyr)
library(magrittr)
library(stringr)
library(lubridate)
d1 <- read_csv(here::here("data", "clean", "frog_translocation_final.csv"))

transno_min <- d1 %>% 
  distinct(site_id, date) %>% 
  group_by(site_id) %>% 
  count() %>% 
  ungroup() %>% 
  summarize(min(n))

transno_max <- d1 %>% 
  distinct(site_id, date) %>% 
  group_by(site_id) %>% 
  count() %>% 
  ungroup() %>% 
  summarize(max(n))

transfrogs_max <- d1 %>% 
  group_by(site_id, date) %>% 
  count() %>% 
  ungroup() %>% 
  summarize(max(n))

transfrogs_min <- d1 %>% 
  group_by(site_id, date) %>% 
  count() %>% 
  ungroup() %>% 
  summarize(min(n))

transfrogs_median <- d1 %>% 
  group_by(site_id, date) %>% 
  count() %>% 
  ungroup() %>% 
  summarize(median(n))

bdload_min <- d1 %>% 
  filter(!is.na(bd_load)) %>% 
  group_by(site_id) %>% 
  mutate(bdload_ave = median(log10(bd_load + 1))) %>% 
  ungroup() %>% 
  summarize(min(bdload_ave))

bdload_max <- d1 %>% 
  filter(!is.na(bd_load)) %>% 
  group_by(site_id) %>% 
  mutate(bdload_ave = median(log10(bd_load + 1))) %>% 
  ungroup() %>% 
  summarize(max(bdload_ave))

bdprev_max <- d1 %>% 
  filter(!is.na(bd_load)) %>% 
  mutate(bdload01 = if_else(bd_load > 0, 1, 0)) %>% 
  group_by(site_id) %>% 
  summarize(prev = sum(bdload01)/n()) %>% 
  ungroup() %>% select(prev) %>% 
  slice_max(prev, n = 1, with_ties = FALSE)

bdprev_min <- d1 %>% 
  filter(!is.na(bd_load)) %>% 
  mutate(bdload01 = if_else(bd_load > 0, 1, 0)) %>% 
  group_by(site_id) %>% 
  summarize(prev = sum(bdload01)/n()) %>% 
  ungroup() %>% select(prev) %>% 
  slice_min(prev, n = 1, with_ties = FALSE)

files <- fs::dir_ls(here::here("data", "raw", "cmr-analysis", "survey"))  
surveys <- read_csv(files, id = "path") %>% 
  filter(survey_date < "2023-01-01") %>% 
  select(-path)
rm(files)

surveys_yr_median <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr") %>% 
  distinct(site_id, primary_period, .keep_all = TRUE) %>% 
  group_by(site_id) %>% 
  count(yr) %>% ungroup() %>% 
  summarize(median(n))

surveys_yr_max <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr") %>% 
  distinct(site_id, primary_period, .keep_all = TRUE) %>% 
  group_by(site_id) %>% 
  count(yr) %>% ungroup() %>% 
  summarize(max(n))

surveys_yr_min <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr") %>% 
  distinct(site_id, primary_period, .keep_all = TRUE) %>% 
  group_by(site_id) %>% 
  count(yr) %>% ungroup() %>% 
  summarize(min(n))

robust_year_median <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr" & secondary_period > 1) %>% 
  group_by(site_id) %>% 
  summarize(max_2period = max(secondary_period)) %>% ungroup() %>% 
  summarize(median(max_2period))

robust_year_min <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr" & secondary_period > 1) %>% 
  group_by(site_id) %>% 
  summarize(max_2period = max(secondary_period)) %>% ungroup() %>% 
  summarize(min(max_2period))

robust_year_max <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr" & secondary_period > 1) %>% 
  group_by(site_id) %>% 
  summarize(max_2period = max(secondary_period)) %>% ungroup() %>% 
  summarize(max(max_2period))
```

<!--# Bree comment: Here or elsewhere - it feels like we need a more specific bridge from the genomics section into the translocation section. I think we need to be explicit about how the populations we chose for the genomics section relate to the populations used in translocations. Even if they aren't matched in any way, we should reiterate something about how the experimental designs comport. For example, we could explain how the genomic analyses were conducted with multiple recovering populations so we know evidence for selection is strong in multiple areas of the genome in multiple recovering populations. -->

Comparison of genomes of frogs from Bd-naive versus recovering populations (described above) indicated substantial differences in allele frequencies, consistent with strong selection following Bd exposure.
Here we describe the outcome of frog reintroductions in which animals for reintroduction were collected from recovering populations, including two of the five recovering populations in the frog genome study (@fig-selectionresults: population 1 and 4).
<!--# Check this --> Reintroductions of *R. sierrae* were conducted in Yosemite National Park (California, USA) as part of an effort to reestablish populations of this endangered species.
These reintroductions involved the collection of adult frogs from large, Bd-positive donor populations that have recovered in recent years following Bd-caused declines [@knapp2016] and translocating them to nearby recipient sites [@joseph2018].
We refer to these reintroductions as "translocations" [@seddon2014].
During the period `r min(d1$year)`--`r max(d1$year)`, we conducted `r n_distinct(d1$site_id, d1$date)` translocations of frogs to `r n_distinct(d1$site_id)` recipient sites (`r transno_min`--`r transno_max` translocations per site; @fig-yosemap SI).
Each translocated cohort included `r transfrogs_min` to `r transfrogs_max` frogs (median = `r transfrogs_median`) collected from 1 of `r n_distinct(d1$donor)` donor populations (@fig-yosemap SI).
Estimates of frog survival during the one year following translocation indicate that survival was highly variable between recipient sites, but relatively constant within recipient sites (for the subset of sites that received multiple translocations; @fig-translocation-survival).
As such, the average survival of frogs in the first translocation to a site was an important predictor of survival of frogs in subsequent translocations to that site (@fig-transsurvival-postdens SI).
This suggests an important effect of site characteristics and/or (site \* frog) interactions on frog survival.
In addition, 1-year survival was higher for frogs translocated later in the study period than earlier (@tbl-viability-earlylate SI) <!--# Mention of viability is confusing because this issue isn't dealt with until the third section. Perhaps make the categorization based on frog survival? -->, which we suggest resulted primarily from our improved ability over time to choose recipient sites with higher habitat quality for *R. sierrae*.
Finally, Bd loads changed little between immediately prior to translocation versus during the one year after translocation (@fig-bdload-beforeafter SI), and loads were nearly always well below the level indicative of severe chytridiomycosis and associated frog mortality (5.8 ITS copies on a log~10~ scale) [@vredenburg2010; @joseph2018].
<!--# Add median Bd load, proportion below 5.8 copies threshold? --> <!--# The Methods-Translocation section contains text that repeats similar results. Edit to reduce repetition. --> <!--# To allow comparison with other translocation studies, adding information about the proportion of translocated populations at which evidence of successful reproduction was observed in the 1-2 years following translocation. -->

To identify site and frog-specific drivers of post-translocation survival, we developed linear and multilevel models in a Bayesian framework.
Our specific interest was whether Bd load had a negative effect on adult survival, as would be expected if frogs were highly susceptible to Bd infection.
The best model of 1-year frog survival identified several important predictors, but Bd load at the time of translocation was not among them (bd_load; @fig-survival-postdens SI).
Important predictors included winter severity in the year following translocation (snow_t1), site elevation (elevation), and donor population (donor_72996; @fig-cond-effects).
Males had somewhat higher survival than females, but this effect was only marginally important (sex_male; @fig-cond-effects, @fig-survival-postdens SI).
In addition to Bd load, effects of frog size (length), winter severity in the year of translocation (snow_t), day of year on which a translocation was conducted (day), and translocation order (order_first) were all unimportant (@fig-survival-postdens SI).
Collectively, these results show that Bd load is not a predictor of the survival of translocated frogs, consistent with frogs in recovering populations having evolved sufficient resistance to minimize the adverse effects of Bd infection.
<!--# Allie comment: Maybe tolerance is a better word here? Since they were infected. Or maybe we need to explicitly define what we mean with resistance vs tolerance. Bree comment: I noticed this also – in the Intro, you do say that you are lumping both resistance and tolerance into the phrase "resistance" but you are talking about multiple things (being able to avoid infection, being able to fight infection, being able to tolerate infection, etc). Maybe at some point we want to address specifically that the genomic changes we are observing could contribute to multiple facets of resistance / tolerance.  And that we are lumping those together in an "evolved resistance" bin. --> In addition, the moderate Bd loads that characterized frogs following translocation (@fig-bdload-beforeafter SI) and relatively high survival of frogs in numerous translocated cohorts (@fig-translocation-survival) suggests that resistant frogs from recovering populations can maintain the benefits of resistance in non-natal habitats.
Finally, in three locations where longer CMR time series allowed us to assess the survival of adults recruited to the population (i.e., progeny of translocated individuals), naturally recruited adults had equivalent or higher survival probabilities than the originally translocated adults (@fig-compare_surv_probs SI).
Therefore, frog resistance in non-natal habitats is maintained across generations.

#### Population viability modeling

Results from frog translocations indicated that adult survival was often relatively high and that nearly all populations showed evidence of successful reproduction (described above).
Although suggestive of population establishment, a decade or more of surveys may be necessary to confirm that populations are in fact self-sustaining [@joseph2018].
To extend our inferences of population establishment beyond those possible from the CMR data, we developed a population viability model.
Specifically, to test whether the observed yearly adult survival probabilities in translocated populations were sufficient to promote long-term viability, we built a stage-structured matrix model that captured known frog demography and included demographic and environmental stochasticity.
We parameterized the model using CMR data from translocated populations and known life history values in this system (@tbl-param_values SI).
Given observed yearly adult survival probabilities (from analysis of CMR data) and a baseline yearly recruitment probability of 0.3 (i.e., the probability that, following survival, juveniles successfully recruit to the adult class), our stage-structured model predicted that six of twelve translocated populations should experience a long-run growth rate $\lambda$ greater than 1 in the presence of Bd (@fig-viability A; median predicted $\lambda$ ranges from 1.15-1.36 for these six populations).
These six populations all had observed yearly adult survival greater than 0.5.
When we reduced the probability of successful recruitment $\omega$ to 0.1 (a highly conservative estimate), long-run growth rate of five of these six populations was still greater than 1 (@fig-viability A).
In contrast, for the other six translocated populations, where yearly adult survival was substantially less than 0.5, our model predicted that it was unlikely that long-run growth rate in these populations was greater than 1, even when the probability of successful recruitment to the adult life stage was near 0.4 (@fig-viability A).
A sensitivity analysis of long-run growth rate $\lambda$ to recruitment probability $\omega$ and yearly adult survival probability of naturally recruited individuals $\sigma_{A_R}$ showed that $\lambda$ was most sensitive to small changes in $\sigma_{A_R}$ (@fig-viability-supp A).
<!--# May need to add some text early in this section to clarify why so far in this manuscript we have focused on yearly survival probability of translocated adults, but the model focuses more on yearly survival probability of naturally recruited adults (e.g., in the sensitivity analysis, etc.). It is my understanding that you are generally assuming that these two probabilities are equivalent and interchangable, but additional clarification would be helpful. Might also require some clarification in Table 2. You describe this in more detail in the Methods, but it seems that some additional clarification might be helpful here. -->

Given demographic stochasticity (i.e., probabilistic births and deaths of individuals in a population) and environmental variability in the probability of juvenile recruitment to the adult class $\omega$ (the transition that we expect to be the most subject to environmental stochasticity in the presence of Bd), our model <!--# Is this a different model, now with demographic stochasiticity and environmental variability in juvenile recruitment, or the same model used to estimate lamda in the previous paragraph? Clarify. I know from the Methods that you developed different versions of the model, some deterministic and some stochastic, but some clarification might be useful here too. -->predicted that, following a single introduction of 40 adult individuals into a population, the six populations with the highest adult survival probabilities had 50-year extinction probabilities of less than 1 (@fig-viability B).
Therefore, even under conservative assumptions about recruitment probability $\omega$, these populations had some probability of persistence over 50 years following a single introduction.
<!--# The wording in the previous two sentences seems overly cautious in describing the extinction probability (i.e., "less than 1", "some probability of persistence". In fact, as the next sentence makes clear, for reasonable probabilities of recruitment, the extinction probabilities are substantially less than 1. Consider tweaking wording to clarify this. I'd argue for a relatively strong initial statement that we then fine tune only slightly, instead of a weak statement that we then strengthen a lot. Initial impressions can be hard to change. --> For these six populations, as $\omega$ increased from 0.1 to 0.3, 50 year extinction probability was less 0.5 <!--# Based on Fig. 5, looks like extinction propability is < 0.5 for all six populations only for omega > ~0.22. Reword as necessary. -->, indicating strong potential for long-term persistence in the presence of Bd and environmental variability in recruitment success (@fig-viability B).
In contrast, for the six populations where $\sigma_{A_R} < 0.5$, extinction probability over 50 years was always predicted to be 1.
To test the validity of our model predictions, we demonstrated that our stochastic model could describe the general recovery trajectory of our translocated population with the longest survey history (@fig-viability-supp B; population 70550, surveyed for 16 years).
Overall, our model demonstrates that given observed yearly adult survival probabilities of resistant frogs in non-natal locations, 50% of our translocated populations have a high probability of population growth and long-term viability in the presence of Bd.
Moreover, five of the seven populations translocated after 2013 were predicted to have long-term viability, compared to only one of five populations translocated prior to 2013.
Thus, there is some trend in our improved ability to select appropriate habitat to promote long-term viability of translocated, resistant populations.

<!--# Is it worth mentioning that predictions may be conservative because yearly survival probability of naturally recruited animals may be higher than yearly survival probability of the translocated animals that was used to parameterize model (Figure 12). -->

### Discussion

Disease-induced declines are a central conservation challenge for wildlife and identifying biologically grounded strategies for assisted recovery is a critical need across systems.
One such strategy requires leveraging disease-induced genetic changes in host populations and so-called evolutionary rescue to recover a species at the landscape scale.
While this strategy has been previously proposed, it has never been successfully implemented.
Here, we provide strong evidence that selection for resistance and evolutionary rescue has led to the recovery of some populations of the endangered MYL frog in the presence of a deadly fungal pathogen.
Leveraging individuals from these resistant and rescued populations, we use 15 years of translocation effort and population viability modeling to demonstrate that populations that have experienced evolutionary rescue can be used to successfully establish and recover this species across the landscape.
These results not only provide a rare example of species recovery following disease-induced declines due heritable changes in resistance, but provide the first demonstration of how natural evolutionary rescue can be leveraged to successfully recover a species in the presence of disease.

While evolutionary rescue is not the only mechanism that can lead to host recovery in the presence of disease (Brannely), there are four lines of evidence that evolutionary rescue is a strong candidate mechanism for observed resistance and recoveries in MYL frog systems.
First, in host-pathogen systems, evolutionary rescue is more likely to occur when hosts experience repeated selection events (e.g., through re-infections), there is a persistent pathogen reservoir or frequency-dependent infection dynamics, and populations experience increased per capita birth rates at small population sizes [@golas2021].
At least two of these conditions are met in the MYL frog system: i) individuals that recover from Bd infection do not gain long-lasting immunity to Bd and are susceptible to re-infection and ii) there are potential Bd reservoirs (Briggs, Reeder) and seasonal frequency-dependent transmission dynamics (Wilber) in the MYL frog system.
Thus, MYL frog-Bd systems have the epidemiological characteristics necessary for evolutionary rescue to occur.
Second, our genomic analyses comparing naive and recovering MYL frog populations identified notable differences in allele frequencies between adjacent Bd-naive and recovering population.
These differences are consistent with strong signatures of selection and regions of the genome under selection are associated with known mechanisms of resistance to Bd infection.
Third, our translocations allowed us to conclude that observed resistance and recovery of resistant populations is not strictly due to local environmental conditions (though environment does play a role).
Changes in population growth rate due to evolved resistance extend beyond the natal environment and into subsequent generations, as demonstrated by high survival rates of translocated frog populations in new habitats in the presence of Bd.
Finally, evolved resistance has a quantifiable phenotype.
In persistent populations and those established via translocation of frogs from recovering populations, Bd load is not an important predictor of post-translocation frog survival (in contrast to important relationship between Bd load and frog survival during Bd-caused epizootics).
While other studies have documented signatures of selection in hosts following disease-induced declines (Epstein, white-nose, Byrne), our study is unique in that we show that evolved resistance is associated with population recovery in both natal and non-natal habitats, providing an actionable strategy for promoting the long-term viability of a species.

Despite the results we present here, recovery of Bd-impacted amphibians worldwide is expected to be challenging at best.

-   Hundreds of declining species, but mostly failed attempts at recovery.
    Failures may be due in part to a general inability by amphibians to mount an effective immune response against Bd infection.
    In contrast, in our study MYL frogs (a species highly susceptible to chytridiomycosis) showed relatively high frog survival despite ongoing Bd infection and low probability of extinction for many reestablished populations.

Could include mention here of why MYL frogs show evidence of evolutionary rescue in response to Bd exposure when so few other amphibians do.
Text from Introduction describing the conditions under which evolutionary rescue is most likely?

3.  Translocations (including viability results?):
    -   Recovery of Bd-impacted amphibians worldwide is expected to be challenging at best. Hundreds of declining species, but mostly failed attempts at recovery. Failures may be due in part to a general inability by amphibians to mount an effective immune response against Bd infection. In contrast, in our study MYL frogs (a species highly susceptible to chytridiomycosis) showed relatively high frog survival despite ongoing Bd infection and low probability of extinction for many reestablished populations.
    -   Observed reduced susceptibility is likely due to evolutionary rescue, and this outcome is likely facilitating both natural recovery (Knapp et al. 2016) and success of translocation efforts described in this study. MYL frogs provide a rare example of highly susceptible species recovering in the presence of pathogen/disease. Recovery would likely not be possible without evolutionary rescue, as suggested by numerous failed attempts to reestablish populations of Bd-impacted amphibians. Although evolution of resistance in donor populations facilitates the long-term viability of translocated populations, survival is still highly dependent on environment independent of Bd.
    -   IUCN guidance for reintroduction/translocation success is the presence of reproduction. Based on this standard, X of our 12 translocation efforts would be deemed successful. We used a higher standard - 1 year survival and probability of extinction over 50 years. More relevant standard because of the bottleneck to population establishment posed by subadult survival which despite evolutionary rescue is strongly influenced by disease.
    -   This success in the face of a highly virulent pathogen is all the more surprising given the generally low rate of translocation/reintroduction success (several review papers). Despite relatively high success rate in MYL frogs, it is apparent that not all such recovery efforts will succeed. In addition to ongoing effects of Bd, several deterministic and stochastic factors likely also play an important role: (1) effects of elevation, winter severity, and donor population on frog survival, and (2) variability in recruitment driven by Bd-caused mortality of subadults.
    -   Now have a means of reestablishing MYL frog populations across their historical range, including in areas in which natural recolonization is precluded by dispersal limitation. Nonetheless, recovery across this large landscape will require substantial resources over many decades.
    -   Big picture: May also have implications for the recovery of the many Bd-impacted amphibians worldwide. Remains to be seen how many other amphibians respond to Bd exposure via evolutionary rescue.

### Materials and Methods

#### Frog evolution in response to Bd

##### Sampling and Sequencing

We collected DNA samples via buccal swabbing [@broquet2007] from 53 *Rana muscosa*/*Rana sierrae* individuals: 24 from four Bd-naive populations, and 29 from five recovering populations.
These populations were located in the southern Sierra Nevada, from northern Yosemite National Park to northern Sequoia National Park (@fig-selectionresults A).
Samples were collected from 5-6 frogs per population.
To minimize potential confounding effects caused by variation in frog genotypes across latitude [e.g., @rothstein2020], we selected sampling sites such that both population types were represented across similar latitudinal ranges (@fig-selectionresults A).
DNA was extracted following Qiagen DNEasy manufacturer's protocols.
We sequenced the samples using an exome capture assay as described in Byrne et al. (in review).
Briefly, genomic libraries were prepared and captured using a custom Nimblegen capture pool.
Capture baits were designed based on the coding regions of the *Rana muscosa* transcriptome (GenBank accession GKCT00000000).
Captured libraries were then pooled and sequenced on a NovaSeq 6000 150PE Flow Cell S1 at the Vincent J. Coates Genomics Sequencing Lab at UC Berkeley.

##### Data Pre-processing and Cleaning

Raw reads were filtered for adapters and contaminants using fastp [@chen2018] and aligned to the *Rana muscosa* genome [NCBI SRA: SRS6533475, reference = @hon2020] with repetitive elements masked using bwa ("mem" mode) [@li2013].
Exact PCR Duplicates were marked using Picard.
Variants were then called following GATK best practices (v.XX, GATK ref).
Briefly, raw variants were called for each sample using HaplotypeCaller and combined using CombineGVCFs.
Next, genotypes were jointly called using GenotypeGVCFs.
Variants were then hard filtered using gatk VariantFiltration using the following parameters to remove low-quality sites: QD \< 2.0, FS \> 40.0, SOR \> 3.0, MQ \< 50.0, MQRankSum \> 3.0, MQRankSum \< -3.0, ReadPosRankSum \> 3.0, ReadPosRankSum \< -3.0.
This initial filter resulted in 1,595,206 variant sites across 53 individuals.

We then further filtered our dataset at the individual and variant level.
First, we trimmed our variants to only include those with minor allele frequency \> 0.03, a maximum depth of 250 and minimum depth of 5, a minimum genotype quality of 20, and a maximum missing proportion of 0.5.
This filter resulted in 427,038 sites, of which 353,172 were single nucleotide polymorphisms (SNPs) and 73,866 were insertions or deletions (INDELS).
Finally, we trimmed samples with an average depth across filtered sites \< 7x.
Our final dataset included 50 samples (23 from naive and 27 from recovering populations) with an average depth of 16.7x (range = 7.4x -- 26.1x).

##### Data Analysis

To visualize the genomic relationship of our populations we created a PCA using the glPCA function in the adegenet R package [@jombart2008].
Next, to detect regions of the genome under selection between naive and recovering populations, we used a multivariate linear mixed model to evaluate individual variants and a splined window analysis to evaluate larger genomic regions.
<!--# Improve wording in this and subsequent paragraph to clarify that the two approaches were used to identify regions under selection.  For example, for use genomics dummies it would be useful to see something like: "We used two related approaches to identify regions under selection. The first approach A, did X, Y, and Z but had C limitations.  The second approach B, did X, Y, and Z and D limitations."-->.
GEMMA calculates and incorporates a relatedness matrix for input samples, allowing us to account for relatedness and population structure when calculating likelihood ratio statistics for our association test.
For this analysis we used a stringent data filter to include only SNPs with \<5% missing data (missing for no more than 2 individuals).
GEMMA then calculated the likelihood ratio statistic <!--# What is the null with this test (or what are the two models being compared with the LRT)?  Probably obvious for genomics folks, but not totally clear to me.  I am guessing we are testing whether the `population` coefficient (naive vs. persistent) is significant for describing variability in a SNP? --> for the set of 148,307 high quality SNPs across 127 contigs.
We then calculated two statistical thresholds for identifying outliers using a Bonferroni correction for a significance level of 0.05 (liberal set) and 0.01 (stringent set).
<!--# Could we say something like: "Outliers indicated that a SNP was differently represented in naive vs. persistent populations" or whatever is the correct language." --> Results were visualized using a Manhattan plot and qqplot.

To detect regions of the genome under selection <!--# Improve wording.  Also, can we contrast this with the previous approach? What is this analysis doing that the other one didn't? -->, we conducted a splined window analysis for two statistics: *F~ST~* and differential nucleotide diversity ($\pi_{diff}$) between frogs from naive and recovering populations.
First, we calculated per-site F~ST~ between the naïve and recovering individuals for all bi-allelic SNPs in the 30 largest contigs (98% of all SNPs) using VCFtools [@danecek2011].
Next, we calculated per-site $\pi$ separately for individuals from the naive and recovering populations individuals using VCFtools, then calculated $\pi_{diff}$ for each population ($\pi_{diff} = \pi_{naive} - \pi_{recovering}$).
We concatenated the values for F~ST~ and $\pi_{diff}$ in order of size-sorted chromosome number and adjusted the SNP position based on the relative position in the genome.

We then used the package WINGEN R package [@bishop2023] <!--# Correct citation? --> to conduct a splined discrete window analysis for *F~ST~* and $\pi_{diff}$.
This method calculates where non-overlapping window boundaries should occur by identifying inflection points in the fitted spline, therefore balancing false positive and false negative results that occur using other window-based methods [@beissinger2015] .
This method also calculates a W-statistic allowing for outlier identification.
We identified outliers as those with a W-statistic \> overall mean + 4 \* standard deviation for F~ST~ or overall mean +/- 4 \* standard deviation for $\pi_{diff}$.
Outlier regions were then identified as those that were outliers in both analyses, meaning that they showed 1) high differentiation between naive and recovering populations, and 2) differential patterns of nucleotide diversity in the same region.
We then extracted gene transcripts mapped within each region and retrieved annotation for that region using blast.

Using the liberal set of variant outliers, we performed a statistical overrepresentation test (Fisher's exact test) using Blast2GO [@conesa2005] .
First, we retrieved the Blast hits and mapped GO terms for each gene in our targeted transcriptome.
Next, we compared the 29 unique outlier genes from the liberal set of outlier variants to the complete set of genes in our target transcriptome to determine if any GO biological functions, molecular functions, or cellular processes were overrepresented.
We then repeated this process for the set of 33 genes located in the overlapping *F~ST~* and $\pi_{diff}$ splined windows.
For the stringent set of SNP outliers, we determined whether the variant was synonymous (protein sequence the same for each variant) or non-synonymous (protein sequence differs between variants), and where in the gene it was located.
To do this, we first extracted the reference genome sequence surrounding the variant using bedtools "getfasta" [@quinlan2010].
Next, we re-annotated each sequence using blast to get the predicted gene location based on the closest annotated reference [@altschul1997].
We then translated each variant to amino acids and aligned this translation to that of the gene annotation to ensure proper frame of reference using Geneious [@kearse2012].
After ensuring proper translation, we characterized variants as within or outside the coding sequence of the gene and as either synonymous or non-synonymous.

#### Frog translocations

##### Field methods

Translocations of *R. sierrae* were conducted in Yosemite National Park (California, USA) as part of an effort to reestablish populations of this endangered species (@fig-yosemap).
We identified donor populations from which adult frogs ($\geq$ 40 mm snout-vent length) could be collected using several years of visual encounter surveys (VES) and skin swab collections [@knapp2016], as well as landscape genetic information [@poorten2017].
Results indicated that these populations contained several hundred *R. sierrae* adults and hundreds or thousands of tadpoles, and that frog abundances were recovering from previous Bd-caused declines [@knapp2016].
As is typical for recovering MYL frog populations, Bd prevalence in the donor populations was high (`r round(bdprev_min, 2)`--`r round(bdprev_max, 2)`) and Bd infection intensity ("load", expressed as median log~10~(load) by site: `r round(bdload_min, 2)`--`r round(bdload_max, 2)`) was two or more orders of magnitude below the level at which frog mortality is expected (log~10~(load) $\approx$ 5.78 ITS copies per swab)[@vredenburg2010; @joseph2018].
Recipient sites to which frogs were translocated were chosen based on previous *R. sierrae* presence (determined from VES and/or museum records) or characteristics that suggested high quality habitat for this species [@knapp2005].

During each translocation, adult frogs were collected from the donor population and measured, weighed, swabbed, and PIT tagged.
Frogs were transported to the recipient site either on foot or via helicopter.
Following release, translocated populations were visited approximately once per month during the summer active season (median number of visits per summer = `r round(surveys_yr_median)`; range = `r surveys_yr_min`--`r surveys_yr_max`) and assessed using diurnal CMR surveys and VES.
CMR surveys allowed quantification of adult survival, adult population size, and recruitment of new adults, and VES provided estimates of tadpole and subadult abundance.
During 2006-2012, CMR surveys were conducted on a single day (primary period) per site visit, during which all habitats were searched repeatedly for adult frogs.
Frogs were captured using handheld nets, identified via their PIT tag (or tagged if they were untagged), measured, weighed, swabbed, and released at the capture location.
During 2013-2022, we generally used a robust design in which all habitats were searched during several consecutive days (median number of secondary periods per primary period = `r round(robust_year_median)`; range = `r robust_year_min`--`r robust_year_max`), and frogs were processed as described above.
However, when the number of frogs detected on the first survey day was zero or near zero, we typically conducted only a single-day CMR survey.
When using a robust design, within a primary period, frogs that were captured during more than one secondary period were measured, weighed, and swabbed during the first capture, and during subsequent captures were only identified and released.
Frog skin swabs were analyzed using standard Bd DNA extraction and qPCR methods.
For details of survey and swab analysis methods, see @joseph2018.

During each site visit, we conducted VES either immediately before CMR surveys or during the first day of CMR surveys.
VES was conducted by walking the entire water body perimeter, first 100 m of each inlet and outlet stream, and any fringing ponds and wetlands, and counting all *R. sierrae* tadpoles and subadults.
These *R. sierrae* life stages have high detectablility, and counts are highly repeatable and provide estimates of relative abundance [@knapp2000].

##### Estimation of frog survival and abundance

We estimated survival of translocated frogs, recruitment of new frogs into the adult population, and adult population size using a Bayesian open-population Jolly-Seber mark-recapture model with known additions to the population (i.e., translocated cohorts), as implemented by the mrmr package [@joseph2019] and using R Statistical Software [v4.4.4, @rsoftware2022].
The model tracks the states of *M* individuals that comprise a superpopulation made up of real and pseudo-individuals [see @joseph2018 for details].
The possible states of individuals include "not recruited", "alive", and "dead".
The possible observations of individuals include "detected" and "not detected".
We assume that individuals that are in the "not recruited" or "dead" states are never detected (i.e., there are no mistakes in the individual PIT tag records).
For all models, we used mrmr defaults for priors, number of chains (4), and warmup and post-warmup iterations (2000 for each).
We evaluated convergence of the Markov chain Monte Carlo (MCMC) algorithm using trace plots and Gelman-Rubin statistics (Rhat).
Code used to create CMR datasets and fit models for each translocated population is available from the following GitHub repository: [https://github.com/SNARL1/cmr-analysis](https://github.com/SNARL1/cmr-analysis.).

##### Predictors of post-translocation frog survival

To identify important predictors of frog survival following translocation, we developed multilevel models in a Bayesian framework.
We performed all analyses with the rstanarm package [@rstanarm2022] and R Statistical Software [v4.4.4, @rsoftware2022], and used default, weakly informative priors.
Starting models included all relevant population-level effects and, when data contained possible hierarchical structure, we included group-level effects in the full model.
Within each analysis, we evaluated model fit using visualizations of leave-one-out ("LOO") probability integral transformations [@gelman2013, @gabry2019], and compared fits of models with and without group-level effects, and with differing group-level effects using LOO cross-validation [@vehtari2016].
Both approaches utilized the loo package [@vehtari2022].

We conducted two analyses.
In the first, we addressed the more general question of whether survival of frogs in the first translocation to a site predicted the survival of frogs in subsequent translocations.
Group-level effects included site_id or translocation_id.
The second analysis focused specifically on identifying predictors of post-translocation frog survival using a wide range of variables describing characteristics of frogs, sites, and translocations ([@fig-survival-postdens] : SI).
Group-level effects included site_id, translocation_id, or translocation_id nested within site_id.
In both analyses, the response variable, probability of 1-year frog survival, was rounded to integer values to produce a binomial outcome, and all models used a bernoulli family.
For all models, we used default priors, four chains, and 5000 iterations each for warmup and post-warmup.
We checked convergence of the MCMC algorithm using trace plots and Rhat.
Code to create the datasets and fit models is available from the following GitHub repository: <https://github.com/SNARL1/translocation>.
<!--# Mention that in both analyses we used 1-year survival as a single metric of survival that was available for all translocated populations regardless of how many years of post-translocation data were available. -->

##### Changes in Bd load following translocation

To assess the magnitude of changes in Bd load on frogs following translocation, we compared Bd loads measured before versus after translocation.
Before-translocation loads were quantified using skin swabs collected from all to-be-translocated frogs at the donor site on the day before or the day of the translocation.
After-translocation Bd loads were based on all swabs collected from translocated frogs at the recipient site in the year of and the year following translocation.
Individual frogs and their associated Bd loads were included in the dataset only if frogs were captured at the recipient site at least once during the year 0-1 period.

To quantify changes in Bd load, we used an analysis workflow similar to that described above.
Briefly, the response variable was before-translocation load and the predictor variable was after-translocation load.
We compared models with and without group-level effects, and group-level effects included pit_tag_id, pit_tag_id nested within site_id, or pit_tag_id nested within translocation_id.
All models used a negative binomial family.
Priors, chains, iterations, and checks were as described above.
Code to create the dataset and fit models is available from the following GitHub repository: <https://github.com/SNARL1/translocation>.
<!--# Does it make more sense to only analyze the model with pit_tag_id as the group-level effect. That is the model that is most analogous to a paired t-test. -->

#### Population viability modeling

##### Model description

To determine the implications of observed 1-year adult survival on the long-term viability of populations established via translocation, we developed a population model for MYL frogs.
Our central question was: How does the magnitude and variation in observed adult survival probability across translocated populations affect the long-term persistence probability of populations?
We developed a model that tracked seven state variables of a frog population: density of translocated adults ($A_T$), density of adults naturally recruited into the population ($A_R$), density of first-year tadpoles ($L_1$), density of second-year tadpoles ($L_2$), density of third-year tadpoles ($L_3$), density of first-year juveniles ($J_1$), and density of second-year juveniles ($J_2$).
We divided adults into two classes $A_T$ and $A_R$ because there is evidence that the survival probability of translocated adults and naturally recruited adults differs ([@fig-compare_surv_probs] : SI).

We modeled the dynamics of these seven state variables using a discrete-time, stage-structured model where a time step is one year.
The dynamics are given by

$$
\begin{bmatrix}
L_1 \\
L_2 \\
L_3 \\
J_1 \\
J_2 \\
A_R \\ 
A_T
\end{bmatrix}(t + 1) = 
\begin{bmatrix}
  0 & 0 & 0 & 0 & 0 & \sigma_{A_R} p_F F & \sigma_{A_T} p_F F \\
  \sigma_{L1} p_{L1} & 0 & 0 & 0 & 0 & 0 & 0 \\
  0 & \sigma_{L2} p_{L2} & 0 & 0 & 0 & 0 & 0\\
  \sigma_{L1} (1 - p_{L1}) & \sigma_{L2} (1 - p_{L2}) & \sigma_{L3} & 0 & 0 & 0 & 0 \\
  0 & 0 & 0 & \sigma_{J1} p_{J1} & 0 & 0 & 0 \\
  0 & 0 & 0 & \sigma_{J1} (1 - p_{J1}) \omega & \sigma_{J2} \omega & \sigma_{A_R} & 0 \\
  0 & 0 & 0 & 0 & 0 & 0 & \sigma_{A_T} 
\end{bmatrix} \begin{bmatrix}
L_1 \\
L_2 \\
L_3 \\
J_1 \\
J_2 \\
A_R \\
A_T \\
\end{bmatrix}(t) 
$$

The parameters in this model are yearly survival probability $\sigma_{\cdot}$ ("$\cdot$" indicates a particular state variable), probability that a female frog reproduces in a given year $p_F$, number of eggs produced by a female frog in a year that successfully hatch $F$, probability of a first-year tadpole remaining as a tadpole $p_{L1}$, probability of a second-year tadpole remaining as a tadpole $p_{L2}$, and probability of a first-year juvenile remaining as a juvenile $p_{J1}$.
The parameter $\omega$ gives the probability of a juvenile successfully recruiting to an adult.
<!--# Can the omega parameter be added to the list of other parameters: "..., probability of a first-year juvenile remaining as a juvenile pj1, and probability of a juvenile successfully recruiting to an adult omega." Listing omega separately seems unnecessary, at least to me. -->

In this model we ignore density-dependent recruitment because we were interested in the growth of the population from an initial reintroduction and whether this growth was sufficient to prevent extinction over 50 years following the introduction.
We also did not directly consider the dynamics of Bd in this model.
We made this decision because (i) translocated populations are infected with Bd at high prevalence [@joseph2018], and (ii) host density does not seem to play a significant role in multi-year Bd infection dynamics in this system [@wilber2022].
Thus, ignoring Bd infection dynamics and instead assuming all host vital rates are in the presence of high Bd prevalence significantly simplifies the model without much loss of realism.

##### Model analysis

###### Incorporating yearly variability in vital rates

We extracted yearly survival probabilities for translocated adults $\sigma_{A_T}$ and naturally recruited adults $\sigma_{A_R}$ from the CMR model used to estimate population-level parameters of translocated populations (see "Estimation of adult survival and population size" above).
Although we observed yearly variability in adult survival *within* a population, the magnitude of this variability was small compared to *among* population variability ([@fig-translocation-survival]).
Thus, we did not include yearly within-population variability in adult survival in this analysis.
However, within a population there was substantial yearly variability in the successful recruitment of adults, greater than what we would expect from Poisson variability around a mean value.
Therefore, we allowed for yearly variability in the probability of successful recruitment $\omega$ (additional details provided in "Estimating model parameters" below).

###### Estimating model parameters

The baseline parameter values for the model and how they were estimated are given in @tbl-param_values : SI.
Parameters $\sigma_{A_T}$ and $\sigma_{A_R}$ were extracted directly from our CMR model described previously in this study.
For populations where we had a sufficient number of PIT-tagged, naturally-recruited adults, we observed that $\sigma_{A_T}$ and $\sigma_{A_R}$ could be notably different, with $\sigma_{A_R} > \sigma_{A_T}$ ([@fig-compare_surv_probs] : SI).
For populations lacking sufficient numbers of naturally-recruited adults, we were unable to directly estimate $\sigma_{A_R}$, and instead set $\sigma_{A_R} = \sigma_{A_T}$.

###### Model analysis and simulation

We performed four analyses on our model.
First, we considered a deterministic version of our model with no yearly heterogeneity in probability of successful recruitment $\omega$, and calculated the predicted long-run growth rate $\lambda$ of a population for different values of $\sigma_{A_R}$ and $\omega$.
We then fixed $\omega = 0.3$ and calculated the predicted growth rate of our 12 populations.

Second, we performed a local elasticity analysis on $\lambda$ with respect to parameters $\omega$, $\sigma_{A_R}$, and $F$ to determine how small changes in these parameters could influence the long-run deterministic growth rate of populations.

Third, we defined a version of the model with demographic and environmental stochasticity, where environmental stochasticity was represented by among-year variability in $\omega$.
We used this model to simulate a one-time introduction of 40 translocated adult frogs.
We ran this simulation 1000 times for each population and computed the probability of a population becoming extinct after 50 years given the observed parameter values and environmental stochasticity in $\omega$.
We varied the mean recruitment probability $\omega$ from 0.15 and 0.4 and drew values of $\omega$ each year from a beta distribution with a dispersion parameter of $\phi = 2$ (when $\omega = 0.5$ and $\phi = 2$ the beta distribution is uniform between 0 and 1).

Finally, we assessed whether our stochastic model could reproduce observed trajectories of population recovery.
We focused on population 70550 because this was our longest CMR time series for a translocated population and because this population shows evidence of substantial post-translocation increases in adult abundance associated with population establishment and recovery.
We simulated our model for 16 years, repeating the simulation 50,000 times.
For each run and each year, we drew $\omega$ from a uniform distribution between 0 and 1 (or equivalently a beta distribution with mean 0.5 and $\phi = 2$).
Using Approximate Bayesian Computing and rejection sampling [@kosmala2016], we identified the top 2% of trajectories (i.e., 1000 trajectories) that minimized the sum of squared errors between the observed and predicted data.
The $\omega$ values associated with these "best" trajectories represented an approximate posterior distribution [@beaumont2010].
Using these best fit trajectories, we assessed whether our model could qualitatively describe the patterns of recovery in the observed data for population 70550.

### Acknowledgments and funding sources

### Figures

<!--# Bree comment: At some point it might be good to talk about what figures go in main text and which stay in supplement (or if we might make some multi-panel figures so that we can fit more in the main text). One thing I'm struck by in the figures is that each of the three main sections (genetics, translocations, PVA) are each so information rich but also somewhat disconnected. Not sure if we want to create some kind of synthetic or hybrid conceptual/data figure that pulls the main point of each analysis out and puts it in visual/conceptual relationship with the others. The results are so cool and we don't want readers to gloss over the details or have to work too hard to put it all back together again. -->

<!--# Standardize panel labels. I suggest capital letters with no punctuation, e.g., "A", not "A.". -->

![For each of the naive and recovering MYL frog populations, pie charts showing allele frequencies for the stringent set of 11 outlier SNPs from 7 distinct genes: (A) LOC108802036, (B) TCF19, VARS, MAP1S, TYRP1, SRFBP1, and (C) RIN3. Charts are superimposed on a map of the Sierra Nevada study area, with Yosemite, Kings Canyon, and Sequoia National Parks (from north to south) shown in dark gray, and the range boundary for MYL frogs in this portion of the Sierra Nevada shown in light gray. The inset map locates the study area in California. <!--# Allie: Add compass rose and format distance bar to match that in Yosemite map. Relocate pie charts to avoid the perception that all recovering populations are west of naive populations. In legend, provide interpretation of "Homozygous reference", "Homozygous alternate", and "Heterozygous". How about identifying each population as R. muscosa or R. sierrae using squares and dots, respectively? The current version of the figure ignores the species identities, but including them might help with interpretation of results. Try moving inset map to lower left portion of panel - more room there to enlarge inset?  Remove distance scale from inset map. --> <!--# Bree comment: Minor but the little set-in map in the central panel may be too small to be useful. Roland response:  -->](figures/allele_maps.jpg){#fig-allelefrequencies}

![Results from the splined window analyses showing outlier windows for $\pi_{diff}$ (A) and *F~ST~* (B). Outlier windows for either $\pi_{diff}$ or *F~ST~* are shown in blue and outlier windows for both $\pi_{diff}$ and *F~ST~* are shown in red. (C) Magnified Contig19 from (A) showing two adjacent outlier windows for $\pi_{diff}$ 12.9Mb upstream of the RIN3 outlier SNP (indicated with a dashed vertical blue line). (D) Magnified Contig8 from from (B) showing the *F~ST~* outlier window that includes the outlier SNPs TCF19 and VARS. This region of the genome contains 10 annotated genes known to occur in the extended MHC Class I and II regions. <!--# As I commented in the Results text, numerous outlier regions were identified, so how and why were the regions highlighted in C and D chosen? Legend text that describes the outlier categories needs additional editing to clarify the categories. My attempt feels only partially successful. Capitalize plot identifiers (A, B, C, D). -->](figures/spline_manhattan_figure.jpg){#fig-spline-manhattan}

![Median 1-year survival of each cohort of translocated frogs at the 12 recipient sites included in this study. Sites are arranged along the x-axis using the average of the median 1-year survival per translocation at each site. Frog survival was estimated using CMR methods. Dot colors indicate the donor population from which frogs in each translocated cohort were collected. Error bars show the 2.5% and 97.5% posterior quantiles (i.e., 95% uncertainty intervals). When multiple translocations were conducted to a site, points and error bars are slightly offset to avoid overlap.](figures/translocation_survival_bysiteid.png){#fig-translocation-survival width="450"}

![Conditional effects of the important predictors of 1-year frog survival. (A) winter severity in the year following translocation, (B) elevation of recipient site, (C) donor population, and (D) sex. In (A) and (B), blue lines are medians and gray ribbons are 95% uncertainty intervals. In (C) and (D), box plots show medians, first and third quartiles, largest and smallest values within 1.5x interquartile range, and values outside the 1.5x interquartile range.](figures/cond_effects_plot.png){#fig-cond-effects}

![A population viability analysis of translocated MYL frogs that quantifies the long-run growth rate and extinction probability of frog populations: (A) Predicted long-run growth rate $\lambda$ for different values of yearly adult survival probability $\sigma_{A_R}$ and probability of recruitment $\omega$, given the parameterized, deterministic model for translocated frogs. Colored points show the predicted $\lambda$ values for the twelve translocated populations when probability of recruitment into the adult class is $\omega = 0.3$ (given by the dashed line). (B) Predicted 50-year extinction probabilities of the 12 translocated populations given demographic stochasticity, environmental variability in $\omega$, and different mean values of $\omega$. The dashed line shows where $\omega = 0.3$, for comparison to (A). There are 6 lines at extinction probability = 1, 5 of which (70414--70619) are hidden beneath the line for 70505. <!--# Remove period after plot labels, e.g., "A", not "A.". -->](figures/pop_viability_figures_for_manuscript_v2.jpg){#fig-viability}

![Sensitivity analysis and empirical validation of the stage-structured MYL frog model. (A) Elasticity of $\lambda$ with changes in three parameters: $\sigma_{A_R}$, $\omega$, and *F (*number of eggs produced by a female frog in a year that successfully hatch) <!--# Necessary to define F (when sigma and omega are not defined)? Maybe instead just cite "Table 2 SI"?) -->. Elasticity is calculated at the default parameter values for each population and $\omega$ = 0.3. (B) 100 simulated trajectories (black lines) that most closely matched the observed abundance trajectory of adult amphibians at site 70550 (light orange).](figures/pop_viability_figures_for_supp.jpg){#fig-viability-supp}

<!--# Remove period after plot labels -->

<!--# Left panel color palette is 12-color color-blind friendly, but is not the most friendly color palette to look at.  We can discuss. Roland says: Maybe try a palette that you like better and mention that order of bars (left to right) is the same as in the legend (top to bottom). In addition, if you think it is necessary, for those colors that will be hard for eyeballs like mine to distinguish, we can either tweak the colors or add diagnonal lines (ggpattern::geom_bar_pattern). -->

## Supporting Information

### Figures

![Results from the exome capture selection study. (A) Map of sampling locations in the Sierra Nevada study area showing Bd-naive populations in red and recovering populations in blue. Yosemite, Kings Canyon, and Sequoia National Parks (from north to south) are shown in dark gray, and the range boundary for MYL frogs in this portion of the Sierra Nevada is shown in light gray. (B) PCA calculated from binary SNPs showing genomic relationship of samples. Numeric labels and colors match those in (A). (C) qqplot showing observed and expected p-values for 148,307 SNPs and indels as calculated in GEMMA. Dashed line shows Bonferroni corrected p-value for threshold of 0.05 (liberal) and 0.01 (stringent) outliers <!--# Add line for 0.05, if that distinction would be useful. -->. (D) Manhattan plot showing p-values for each SNP as calculated by GEMMA. SNPs are sorted by genomic position and contigs are sorted by size. Red line shows Bonferroni corrected p-value for threshold of 0.01. <!--# Show line for threshold of 0.05 also? -->Outlier SNPs above this threshold are highlighted and labeled. Bold labels indicate at least one non-synonymous SNP is present in that gene. <!--# Map (panel A) is duplicated here and in the mapped allele frequency figure. I think it can be removed from this figure. Capitalize panel labels. Label/identify both the liberal and stringent outlier sets? -->](figures/map_qqplot_manhattan.jpg){#fig-selectionresults}

![Map showing the locations of translocated and donor MYL frog populations in Yosemite National Park (park boundary indicated by black polygon). Symbol shapes indicate the donor population used for each translocation site. To obscure the exact locations of populations, random noise was added to all point coordinates. Inset map shows the location of Yosemite within California. In both maps, elevation is indicated by the colored hillshade layer (dark green = lowest elevation, white = highest elevation). <!--# Make sure all points are inside YOSE polygon -->](figures/map_translocation_points.png){#fig-yosemap width="450"}

![For the best model describing the average 1-year survival of frogs in the first translocation to a site as a predictor of survival of frogs in all subsequent translocations to that site, estimated posterior density curves and shaded 95% uncertainty intervals for the intercept and single predictor variable. A variable is considered an important predictor if the associated uncertainty interval does not overlap zero (indicated by the dashed red line).](figures/mcmc_areas_m2b.png){#fig-transsurvival-postdens width="450"}

![For frogs translocated to each of the 12 recipient sites, Bd loads for the period immediately prior to translocation versus during the 1-year period after translocation. Box plot details are as in @fig-cond-effects. ](figures/bdload_beforeafter.png){#fig-bdload-beforeafter width="450"}

![For the best model identifying predictors of 1-year frog survival following translocation, estimated posterior density curves and shaded 95% uncertainty intervals for the intercept and all predictor variables. Variables are considered important predictors if the associated uncertainty interval does not overlap zero (indicated by the dashed red line).](figures/mcmc_areas_m1d.png){#fig-survival-postdens width="450"}

![Comparison of the average yearly adult survival probabilities for adults translocated to each of three sites versus adults that were naturally recruited at each site (as a result of reproduction by translocated frogs). In contrast to @fig-translocation-survival, these are not survival probabilities from the first year following translocation, but instead represent averaged survival probabilities across multiple years. Points are median estimates and error bars give the 90% uncertainty intervals around the estimates. All estimates are derived from the mrmr package. <!--# Use 95% uncertainty intervals for consistency? Also, use dark blue instead of gray for naturally-recruited? Might show up better. -->](figures/compare_surv_probs.jpg){#fig-compare_surv_probs}

### Tables

```{r}
#| label: tbl-viability-earlylate
#| tbl-cap: "Proportion of populations translocated early versus late in the study period (< 2013 or >= 2013, respectively) for which the probability of extinction over 50 years was 1 or < 1 (i.e., viability = 0 or 1, respectively)."
#| echo: FALSE
#| message: FALSE

viability_table <- read_csv(here::here("data", "clean", "frog_translocation_final.csv")) %>% 
  distinct(site_id, year) %>% 
  group_by(site_id) %>% 
  filter(year == min(year)) %>% 
  mutate(site_id = as.character(site_id),
         viability = if_else(site_id %in% c(70414, 70370, 70134, 70641, 70619, 70505), 0, 1),
         period = if_else(year < 2013, "early", "late")) %>% 
  group_by(period) %>% 
  count(period, viability) %>% 
  mutate(n_period = sum(n), 
         prop = (n / n_period)) %>% 
  mutate(across(prop, round, 3)) %>% 
  arrange(viability, period)
  
  knitr::kable(viability_table, "pipe", align = "lcccr", col.names = c("Period", "Viability category", "Number of translocations", "Total translocations", "Proportion of translocations")) 
```

| **Parameter**                                                             | **Value**            | **Source**                                                         |
|:--------------------------|:------------------|:-------------------------|
| $\sigma_{L_1}$, Yearly survival probability of year 1 tadpoles            | 0.7                  | Estimated from field data, observations, natural history knowledge |
| $\sigma_{L_2}$, Yearly survival probability of year 2 tadpoles            | 0.7                  | Estimated from field data, observations, natural history knowledge |
| $\sigma_{L_3}$, Yearly survival probability of year 3 tadpoles            | 0.7                  | Estimated from field data, observations, natural history knowledge |
| $\sigma_{J_1}$, Yearly survival probability of year 1 juveniles           | 0.25                 | Estimated from field data, observations, natural history knowledge |
| $\sigma_{J_2}$, Yearly survival probability of year 2 juveniles           | 0.5                  | Estimated from field data, observations, natural history knowledge |
| $\sigma_{A_R}$, Yearly survival probability of naturally recruited adults | Varies by population | Estimated from mark-recapture studies                              |
| $\sigma_{A_T}$, Yearly survival probability of translocated adults        | Varies by population | Estimated from mark-recapture studies                              |
| $p_{L_1}$, Probability of a year 1 tadpoles remaining as a tadpoles       | 1                    | Estimated from field data, observations, natural history knowledge |
| $p_{L_2}$, Probability of a year 2 tadpoles remaining as a tadpoles       | 0.25                 | Estimated from field data, observations, natural history knowledge |
| $p_{J_1}$, Probability of a year 1 juvenile remaining as a juvenile       | 0.25                 | Estimated from field data, observations, natural history knowledge |
| $p_F$, Probability of a adult female reproducing in a year                | 0.5                  | Could be as high at 1, based on field observations                 |
| $F$, Number of surviving eggs produced by an adult female                 | 100                  | From observations of captive frogs                                 |
| $\omega$, Probability of juvenile successfully recruiting to an adult     | Varies yearly        | Explored different values of this parameter                        |

: Description and values of parameters used in the model. All survival probabilities are in the presence of the fungal pathogen Bd. {#tbl-param_values}

### Datasets

Dataset 1: gemma_outliers.csv

Dataset 2: spline_window_shared_outliers.csv

### References

## 
