---
format: docx
editor: visual
csl: nature.csl
bibliography: translocation.bib
---

## Title: Reintroduction of resistant frogs allows recovery in the presence of a lethal fungal disease

<!--# 75 characters. Suggestions to make this more compelling? Use "Evolutionary rescue"? -->

### Authors:

Roland A. Knapp^1,2^\*, Allison Q. Byrne^3,4^, Mark Q. Wilber^5^, Maxwell B. Joseph^6,7^, Thomas C. Smith^1,2^, Andrew P. Rothstein^3,4^, Rob L. Grasso^8^, Erica Bree Rosenblum^3,4^

<!--# Exact author order to be determined -->

### Affiliations:

^1^Sierra Nevada Aquatic Research Laboratory, University of California, Mammoth Lakes, CA, USA

^2^Earth Research Institute, University of California, Santa Barbara, CA, USA

^3^Department of Environmental Science, Policy, and Management, University of California. Berkeley, Berkeley, CA, USA

^4^Museum of Vertebrate Zoology, University of California, Berkeley, Berkeley, CA, USA

^5^Department of Forestry, Wildlife, and Fisheries, University of Tennessee Institute of Agriculture, Knoxville, TN, USA

^6^Earth Lab, University of Colorado, Boulder, CO, USA

^7^NCX, San Francisco, CA, USA

^8^Resources Management and Science, Yosemite National Park, El Portal, CA, USA

\*Corresponding author: roland.knapp\@ucsb.edu

### Abstract

<!--# No more than 200 words, follow structure described in instructions for authors. 2-3 sentences of basic-level introduction to the field; a brief account of the background and rationale of the work; a statement of the main conclusions (introduced by the phrase 'Here we show' or its equivalent); and finally, 2-3 sentences putting the main findings into general context so it is clear how the results described in the paper have moved the field forwards. See [annotated example](https://www.nature.com/documents/nature-summary-paragraph.pdf).  -->

### Main

<!--# subheadings can be used, can be "teasers" of results (up to 40 characters) -->

Severe effects of disease on global biodiversity (bats, primates, amphibians, corals, sea stars, etc.)

Host evolution and population persistence/recovery

-   Species recovery in the face of disease will depend on the ability of a species to move or change.

-   Rapid evolution termed "evolutionary rescue" can reverse ongoing declines in some species (Golas et al. 2021; Searle et al. 2021). (Not sure if Allie wants to go the ER direction)

-   ER has been proposed as a mechanism of recovery in wildlife populations decimated by disease (Epstein et al. 2016, Schiebelhut et al. 2018).

    -   Tasmanian devils: Epstein et al. (2016), Wells et al., Ecology (2019)

    -   Frogs in Australia: Grogan et al. (2018), McKnight et al. (2020), Newell et al. (2013), Scheele et al., (2014)

    -   Frogs in Panama: Voyles et al. (2018)

    -   Frogs in the Sierra Nevada: Knapp et al. (2016) @knapp2016

    -   Sea Star Wasting Disease: Menge et al., Plos One (2016), Schiebelhut et al. PNAS (2018)

    -   White-nose syndrome: Gignoux-Wolfsohn et al. (2021) Molecular ecology.

-   Potential knowledge gap: Subsequent recoveries from disease-induced declines following observed genetic changes have rarely \[never?\] been documented in wildlife populations (Sea stars: Moritsch and Raimondi 2018?).

Population reestablishment using reintroduction of resistant hosts

-   "Rescued" populations provide a source of animals from which to repatriate extirpated populations and disentangle the effects of the local environment and genetics on host recoveries.

-   Mendelson et al. (2019)

-   Scheele et al. (2021)

-   Joseph and Knapp (2018)

-   Potential knowledge gap/opportunity/selling point: For rescued populations to contribute to landscape-scale species recovery, the fitness benefits gained through selection should manifest beyond the natal environment, allowing for the long-term growth and persistence of incipient populations in novel environmental conditions. In this study, we first document evidence of strong selection in host populations that are persisting with a deadly fungal pathogen, indicative that evolutionary rescue in response to previous pathogen invasion has occurred. We then repeatedly translocate these "rescued" populations to non-natal habitats on the landscape and use longitudinal surveys, mark-recapture analysis, and population modeling to demonstrate that "rescued" populations recover and persist across a landscape of disease. Thus, natural selection combined with translocations promotes the recovery of a species on the brink of disease-induced extinction. These results may have broad implications for the many species worldwide that are threatened with extinction by the emergence of novel diseases.

Description of MYL frog species complex, impacts of Bd, and evidence of recent recovery.

Describe expectations if frog populations are recovering due to evolutionary rescue.

-   Differences in allele frequencies exist between Bd-naive and recovering populations

-   Shared selective response across recovering populations

-   Changes in population growth rate due to evolution extend beyond the natal environment

-   In populations established via translocation of frogs from recovering populations, Bd load is not an important predictor of post-translocation frog survival (in contrast to important relationship between Bd load and frog survival during Bd-caused epizootics)

-   Population viability: Yearly population growth rate of translocated populations in similar environments to the natal environment should be greater than 1, indicating the capacity of the selected frogs to recover in the presence of the pathogen. 

Then show how each result is consistent with those expectations.

#### Frog evolution in response to Bd

@fig-exomemap

#### Translocations allow population reestablishment

```{r}
#| include: false

library(readr)
library(dplyr)
library(magrittr)
library(stringr)
library(lubridate)
d1 <- read_csv(here::here("data", "clean", "frog_translocation_final.csv"))

transno_min <- d1 %>% 
  distinct(site_id, date) %>% 
  group_by(site_id) %>% 
  count() %>% 
  ungroup() %>% 
  summarize(min(n))

transno_max <- d1 %>% 
  distinct(site_id, date) %>% 
  group_by(site_id) %>% 
  count() %>% 
  ungroup() %>% 
  summarize(max(n))

transfrogs_max <- d1 %>% 
  group_by(site_id, date) %>% 
  count() %>% 
  ungroup() %>% 
  summarize(max(n))

transfrogs_min <- d1 %>% 
  group_by(site_id, date) %>% 
  count() %>% 
  ungroup() %>% 
  summarize(min(n))

bdload_min <- d1 %>% 
  filter(!is.na(bd_load)) %>% 
  group_by(site_id) %>% 
  mutate(bdload_ave = median(log10(bd_load + 1))) %>% 
  ungroup() %>% 
  summarize(min(bdload_ave))

bdload_max <- d1 %>% 
  filter(!is.na(bd_load)) %>% 
  group_by(site_id) %>% 
  mutate(bdload_ave = median(log10(bd_load + 1))) %>% 
  ungroup() %>% 
  summarize(max(bdload_ave))

bdprev_max <- d1 %>% 
  filter(!is.na(bd_load)) %>% 
  mutate(bdload01 = if_else(bd_load > 0, 1, 0)) %>% 
  group_by(site_id) %>% 
  summarize(prev = sum(bdload01)/n()) %>% 
  ungroup() %>% select(prev) %>% 
  slice_max(prev, n = 1, with_ties = FALSE)

bdprev_min <- d1 %>% 
  filter(!is.na(bd_load)) %>% 
  mutate(bdload01 = if_else(bd_load > 0, 1, 0)) %>% 
  group_by(site_id) %>% 
  summarize(prev = sum(bdload01)/n()) %>% 
  ungroup() %>% select(prev) %>% 
  slice_min(prev, n = 1, with_ties = FALSE)

files <- fs::dir_ls(here::here("data", "raw", "cmr-analysis", "survey"))  
surveys <- read_csv(files, id = "path") %>% 
  filter(survey_date < "2023-01-01") %>% 
  select(-path)
rm(files)

surveys_yr_mean <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr") %>% 
  distinct(site_id, primary_period, .keep_all = TRUE) %>% 
  group_by(site_id) %>% 
  count(yr) %>% ungroup() %>% 
  summarize(mean(n))

surveys_yr_max <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr") %>% 
  distinct(site_id, primary_period, .keep_all = TRUE) %>% 
  group_by(site_id) %>% 
  count(yr) %>% ungroup() %>% 
  summarize(max(n))

surveys_yr_min <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr") %>% 
  distinct(site_id, primary_period, .keep_all = TRUE) %>% 
  group_by(site_id) %>% 
  count(yr) %>% ungroup() %>% 
  summarize(min(n))

robust_year_mean <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr" & secondary_period > 1) %>% 
  group_by(site_id) %>% 
  summarize(max_2period = max(secondary_period)) %>% ungroup() %>% 
  summarize(mean(max_2period))

robust_year_min <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr" & secondary_period > 1) %>% 
  group_by(site_id) %>% 
  summarize(max_2period = max(secondary_period)) %>% ungroup() %>% 
  summarize(min(max_2period))

robust_year_max <- surveys %>% 
  mutate(yr = year(ymd(survey_date))) %>%
  filter(survey_type == "cmr" & secondary_period > 1) %>% 
  group_by(site_id) %>% 
  summarize(max_2period = max(secondary_period)) %>% ungroup() %>% 
  summarize(max(max_2period))
```

During the period `r min(d1$year)`--`r max(d1$year)`, we conducted `r n_distinct(d1$site_id, d1$date)` translocations of adult frogs to `r n_distinct(d1$site_id)` recipient sites in Yosemite (`r transno_min`--`r transno_max` translocations per site; @fig-yosemap). Each translocated cohort included `r transfrogs_min` to `r transfrogs_max` frogs collected from one of `r n_distinct(d1$donor)` donor populations (@fig-yosemap). <!--# Create table of translocation sites, years, frog numbers? --> Estimates of frog survival during the one year following translocation indicate that survival was highly variable between sites, but relatively constant within sites (for the subset of sites that received multiple translocations; @fig-translocation-survival). As such, for sites that received more than one translocation, the average survival of frogs in the first translocation to a site was an important predictor of survival of frogs in all subsequent translocations to that site (median of posterior distribution = 4.14, uncertainty interval = 3.06\--5.28; see Supporting Information for analysis details). The relatively constant within-site frog survival across all translocations suggests an important effect of site characteristics on frog survival. <!--# How to extract these values from table produced by line 934 in translocation_survival_analysis.Rmd (or from associated code output)? -->

The best model of 1-year frog survival identified several important predictors, but Bd load at the time of translocation was not among them (bd_load; @fig-survival-postdens). This result is consistent with the exome capture results suggesting that frogs in recovering populations have evolved increased resistance and/or tolerance to Bd infection. Important predictors are shown in @fig-cond-effects, and included winter severity in the year following translocation (snow_t1), site elevation (elevation), and donor population (donor_72996). Males had higher survival than females, but this effect was only marginally important (sex_male; @fig-cond-effects). In addition to Bd load, effects of frog size (length), winter severity in the year of translocation (snow_t), day of year on which translocation was conducted (day), and translocation order (order_first) were all unimportant (@fig-survival-postdens).

<!--# Present results indicating that Bd loads on frogs changed little following translocation? -->

#### Long-term persistence of reestablished populations

Given observed yearly adult survival probabilities, our stage-structured model predicted that six of twelve translocated populations should experience a long-run growth rate $\lambda$ near or greater than 1 in the presence of Bd (@fig-viability A; median predicted $\lambda$ ranges from 0.95-1.16 for these six populations). When the probability of successful recruitment $\omega$ was greater than 0.1 (the conservative estimate we used as a baseline value in our model), long-run growth rate of these populations could be notably greater than 1 (@fig-viability A). In contrast, for the other six translocated populations where yearly adult survival was less than 0.5, our model predicted that it was unlikely that long-run growth rate in these populations was greater than 1, even when the probability of successful recruitment to the adult life stage was near 0.4 (@fig-viability A). A sensitivity analysis of long-run growth rate $\lambda$ to recruitment probability $\omega$ and yearly adult survival probability of naturally recruited individuals $\sigma_{A_R}$ showed that $\lambda$ was most sensitive to small changes in $\sigma_{A_R}$ (@fig-viability-supp A).

Given demographic stochasticity (i.e., probabilistic births and deaths of individuals in a population) and environmental variability in the probability of adult recruitment $\omega$, our model predicted that, following a a single introduction of 40 adult individuals into a population, the six populations with the highest adult survival probabilities had 50-year extinction probabilities of less than 1 (@fig-viability B). Therefore, even under conservative assumptions about recruitment probability $\omega$, these populations had some probability of persistence over 50 years following a single introduction. For these six populations, as $\omega$ increased from 0.1 to 0.3, 50 year extinction probability was less 0.5, indicating strong potential for long-term persistence in the presence of Bd and environmental variability in recruitment success (@fig-viability B). In contrast, for the six populations where $\sigma_{A_R} < 0.5$, extinction probability over 50 years was always predicted to be 1. To test the validity of our model predictions, we demonstrated that our stochastic model could describe the general recovery trajectory of our translocated population with the longest surveillance history (@fig-viability-supp B; population 70550, surveyed for 16 years). <!--# To help constrain the possible values of omega, add information about recruitment probability observed in the study populations? Because we don't have accurate counts of subadults, can we even provide rough estimate of recruitment probability? -->

### Figures and figure legends

<!--# Discuss referencing and in-text generation of figures.  We don't want to re-run the population viability modeling figures each time we compile the manuscript.  A makefile could be a good solution. Caching figures could be another option (https://quarto.org/docs/computations/caching.html) -->

![Map showing the locations of Bd-naive or recovering frog populations from which DNA samples were collected for the exome capture analyses. Black polygons indicate the boundaries of Yosemite, Kings Canyon, and Sequoia National Parks (from top to bottom). To obscure the exact locations of populations, random noise was added to all point coordinates. Inset map shows the location of the study area within California. In both maps, elevation is indicated by the colored hillshade layer (dark green = lowest elevation, white = highest elevation)](figures/map_exome_points.png){#fig-exomemap width="450"}

![Map showing the locations of translocated and donor populations in Yosemite National Park (park boundary indicated by black polygon). To obscure the exact locations of populations, random noise was added to all point coordinates. Elevations are indicated as in @fig-exomemap. See inset map in @fig-exomemap to locate Yosemite National Park within California. <!--# Add park boundaries to inset map? Also, for this and previous figure, is basemap necessary/helpful -->](figures/map_translocation_points.png){#fig-yosemap width="450"}

![Median survival of each cohort of translocated frogs to the 12 recipient sites included in this study. Sites are arranged along the x-axis using the average of the median survival per translocation at each site. Frog survival for the one-year period following each translocation was estimated using capture-mark-recapture methods. Dot colors indicate the donor population from which frogs in each translocated cohort were collected.](figures/translocation_survival_bysiteid.png){#fig-translocation-survival width="450"}

![For the best model identifying predictors of 1-year frog survival following translocation, estimated posterior density curves and shaded 95% uncertainty intervals for all predictor variables. Variables are considered important predictors if the associated uncertainty interval does not overlap zero (dashed red line).](figures/mcmc_areas_m1d.png){#fig-survival-postdens width="450"}

<!--# Change @fig-survival-postdens from mcmc_areas to mcmc_intervals? -->

![Conditional effects of the important predictors of 1-year frog survival: (A) elevation of recipient site, (B) winter severity in the year following translocation, (C) sex, and (D) donor population. In (A) and (B), blue lines are medians and gray ribbons are 95% uncertainty intervals. In (C) and (D), box plots show medians, first and third quartiles, largest and smallest values within 1.5x interquartile range, and values outside the 1.5x interquartile range.](figures/cond_effects_plot.png){#fig-cond-effects}

![A. The predicted long-run growth rate $\lambda$ for different values of yearly adult survival probability $\sigma_{A_R}$ and probability of recruitment $\omega$, given the parameterized, deterministic model for translocated *R. sierrae*. Colored points show the predicted $\lambda$ values for the twelve translocated populations when probability of recruitment into the adult class is $\omega = 0.3$ (given by the dashed line). B. The predicted fifty-year extinction probabilities of the twelve translocated populations given demographic stochasticity, environmental variability in $\omega$, and different mean values of $\omega$. The dashed line shows where $\omega = 0.3$, for comparison to A.](figures/pop_viability_figures_for_manuscript_v2.jpg){#fig-viability}

![A. The elasticity of $\lambda$ with changes in three parameters: $\sigma_{A_R}$, $\omega$, and *F (*number of eggs produced by a female frog in a year that successfully hatch). Elasticity is calculated at the default parameter values for each population and $\omega$ = 0.3. B. The 100 simulated trajectories (black lines) that most closely matched the observed abundance trajectory of adult amphibians at site 70550 (light orange).](figures/pop_viability_figures_for_supp.jpg){#fig-viability-supp}

<!-- Left color palette as is as 12-color color-blind friendly is not the most friendly color palette to look at.  We can discuss. -->

### References

::: {#refs}
:::

### Methods

<!-- Typically less than 3000 words -->

#### Frog evolution in response to Bd

<!-- subheadings can be used, up to 40 characters -->

To identify putative genes under selection that are linked to reduced frog susceptibility to Bd-caused lethal and sub-lethal effects, we used exome capture methods to compare genomes of frogs in populations with different histories of Bd exposure. Specifically, we compared genomes of frogs from populations that have never experienced a Bd-caused epizootic ("Bd-naive") versus frogs from populations that experienced a Bd epizootic during the past several decades and have since recovered to varying degrees ["recovering"; @knapp2016, @vredenburg2010]. These latter populations persist in an enzootic state [@briggs2010], characterized by high Bd prevalence and Bd infection intensities (load) in adults that are typically well below the level expected to cause frog mortality [@knapp2011, @vredenburg2010]. Bd-exposure histories of these populations are based on a decade or more of data from VES and Bd surveillance using skin swabbing [e.g., @knapp2016].

<!--# Mention that we sampled declining populations also? Will need to update map and text to show only the samples that were sequenced. -->

DNA samples were collected via buccal swabbing [@broquet2007] of xx-xx frogs from seven Bd-naive populations and seven recovering populations. These populations represented a wide latitudinal range in the southern Sierra Nevada, from northern Yosemite National Park to south of Sequoia National Park (@fig-exomemap). To minimize potential biases associated with different latitudinal ranges of the two population types, we selected populations to represent similar latitudinal ranges.

#### Frog translocations

##### Field methods

Translocations of *R. sierrae* were conducted in Yosemite National Park (California, USA) as part of an effort to reestablish populations of this endangered species. We identified donor populations from which adult frogs ($\geq$ 40 mm snout-vent length) could be collected using several years of visual encounter surveys (VES) and skin swab collections [@knapp2016]. Results indicated that these populations contained several hundred *R. sierrae* adults and hundreds or thousands of tadpoles, and that frog abundances were recovering from previous Bd-caused declines [@knapp2016]. As is typical for recovering MYL frog populations, Bd prevalence in the donor populations was high (`r round(bdprev_min, 2)`--`r round(bdprev_max, 2)`) and Bd infection intensity ("load", expressed as median by site: `r round(bdload_min, 2)`--`r round(bdload_max, 2)`) was two or more orders of magnitude below the level at which frog mortality is expected (log~10~(load) $\approx$ 5.78 ITS copies per swab)[@vredenburg2010; @joseph2018]. Recipient sites to which frogs were translocated were chosen based on previous *R. sierrae* presence (determined from VES and/or museum records) or characteristics that suggested high quality habitat for this species [@knapp2005].

During each translocation, adult frogs were collected from the donor population and measured, weighed, swabbed, and PIT tagged. Frogs were transported to the recipient site either on foot or via helicopter. Following release, translocated populations were visited approximately once per month during the summer active season (mean number of visits per summer = `r round(surveys_yr_mean)`; range = `r surveys_yr_min`--`r surveys_yr_max`) and assessed using diurnal capture-mark-recapture (CMR) surveys and VES. CMR surveys allowed quantification of adult survival, adult population size, and recruitment of new adults, and VES provided estimates of tadpole and subadult abundance. During 2006-2012, CMR surveys were conducted on a single day (primary period) per site visit, during which all habitats were searched repeatedly for adult frogs. Frogs were captured using handheld nets, identified via their PIT tag (or tagged if they were untagged), measured, weighed, swabbed, and released at the capture location. During 2013-2022, we generally used a robust design in which all habitats were searched during several consecutive days (mean number of secondary periods per primary period = `r round(robust_year_mean)`; range = `r robust_year_min`--`r robust_year_max`)<!--# calculate min, max, median-->), and frogs were processed as described above. However, when the number of frogs detected on the first survey day was low, we conducted only a single-day CMR survey. Within a primary period, frogs that were captured during more than one secondary period were measured, weighed, and swabbed only during the first capture, and during subsequent captures were only identified and released. Frog skin swabs were analyzed using standard Bd DNA extraction and qPCR methods. For details of survey and swab analysis methods, see @joseph2018.

During each site visit, we conducted VES either immediately before CMR surveys or during the first day of CMR surveys. VES was conducted by walking the entire water body perimeter, first 100 m of each inlet and outlet stream, and any fringing ponds and wetlands, and counting all *R. sierrae* tadpoles and subadults. These *R. sierrae* life stages have high detectablility, and counts are highly repeatable and provide estimates of relative abundance [@knapp2000].

##### Estimation of frog survival and abundance

We estimated survival of translocated frogs, recruitment of new frogs into the adult population, and population size using a Bayesian open-population Jolly-Seber mark-recapture model with known additions (i.e., translocated cohorts) to the population, as implemented by the mrmr package [@joseph2019] and using R Statistical Software [v4.4.4; \@rsoftware2022]. The model tracks the states of *M* individuals that comprise a superpopulation made up of real and pseudo-individuals [see @joseph2018 for details]. The possible states of individuals include "not recruited", "alive", and "dead". The possible observations of individuals include "detected" and "not detected". We assume that individuals that are in the "not recruited" or "dead" states are never detected (i.e., there are no mistakes in the individual PIT tag records). For all models, we used mrmr defaults for priors, number of chains (4), and warmup and post-warmup iterations (2000 for each). We evaluated convergence of the Markov chain Monte Carlo (MCMC) algorithm using trace plots and Gelman-Rubin statistics (Rhat). Code used to construct CMR datasets and fit models for each translocated population is available from a GitHub repository ([https://github.com/SNARL1/cmr-analysis](https://github.com/SNARL1/cmr-analysis.)).

##### Predictors of post-translocation frog survival

To identify important predictors of frog survival following translocation, we developed multilevel models in a Bayesian framework. We performed all analyses using R Statistical Software [v4.4.4; @rsoftware2022] and the rstanarm package [@rstanarm2022]. We started with models that included all relevant population-level effects and, when data contained possible hierarchical structure, we included group-level effects in the full model. Within each analysis, we evaluated model fit using visualizations of leave-one-out ("LOO") probability integral transformations [@gelman2013, @gabry2019], and compared fits of models with and without group-level effects, and with differing group-level effects using LOO cross-validation [@vehtari2016]. Both approaches utilized the loo package.

Of the two analyses we conducted, one focused on identifying predictors of post-translocation frog survival from a wide range of variables describing characteristics of frogs, habitat, and translocations [@fig-survival-postdens]. The other analysis focused specifically on whether survival of frogs in the first translocation to a site predicted the survival of frogs in subsequent translocations. In both analyses, the response variable, probability of frog survival, was rounded to integer values to produce a binomial outcome, and all models used a bernoulli family. For all models, we used default priors, four chains, and 5000 iterations each for warmup and post-warmup. We checked convergence of the MCMC algorithm using trace plots and Rhat. Code to create datasets and fit models is available from the following GitHub repository: <https://github.com/SNARL1/translocation>.

#### Population viability modeling

##### Model description

To determine the implications of acquired Bd resistance on long-term population viability, we developed a dynamic, population model for *R. sierrae*. Our central question was: how does the magnitude and variation in observed adult survival probability across translocated populations affect the long-term persistence probability of populations? We developed a model that tracked seven state variables of an *R. sierrae* population: the density of translocated adults ($A_T$), the density of adults naturally recruited into the population ($A_R$), the density of first year larvae ($L_1$), the density of second year larvae ($L_2$), the density of third year larvae ($L_3$), the density of first year juveniles ($J_1$), and the density of second year juveniles ($J_2$). We divided adults into two classes $A_T$ and $A_R$ because there was evidence that the survival probability of translocated adults and naturally recruited adults differed.

We modeled the dynamics of these seven state variables using a discrete-time, stage-structured model where a time step is one year. The dynamics are given by

$$
\begin{bmatrix}
L_1 \\
L_2 \\
L_3 \\
J_1 \\
J_2 \\
A_R \\ 
A_T
\end{bmatrix}(t + 1) = 
\begin{bmatrix}
  0 & 0 & 0 & 0 & 0 & \sigma_{A_R} p_F F & \sigma_{A_T} p_F F \\
  \sigma_{L1} p_{L1} & 0 & 0 & 0 & 0 & 0 & 0 \\
  0 & \sigma_{L2} p_{L2} & 0 & 0 & 0 & 0 & 0\\
  \sigma_{L1} (1 - p_{L1}) & \sigma_{L2} (1 - p_{L2}) & \sigma_{L3} & 0 & 0 & 0 & 0 \\
  0 & 0 & 0 & \sigma_{J1} p_{J1} & 0 & 0 & 0 \\
  0 & 0 & 0 & \sigma_{J1} (1 - p_{J1}) \omega & \sigma_{J2} \omega & \sigma_{A_R} & 0 \\
  0 & 0 & 0 & 0 & 0 & 0 & \sigma_{A_T} 
\end{bmatrix} \begin{bmatrix}
L_1 \\
L_2 \\
L_3 \\
J_1 \\
J_2 \\
A_R \\
A_T \\
\end{bmatrix}(t) 
$$

The parameters in this model are yearly survival probability $\sigma_{\cdot}$, where the "$\cdot$" indicates a particular state variable, the probability that a female frog reproduces in a given year $p_F$, the number of eggs produced by a female frog in a year that successfully hatch $F$, the probability of a first year tadpole remaining as a tadpole $p_{L1}$, the probability of a second year tadpole remaining as a tadpole $p_{L2}$, and the probability of a first year juvenile remaining as a juvenile $p_{J1}$. The parameter $\omega$ gives the probability of a juvenile successfully recruiting to an adult.

In this model we ignore density-dependent recruitment because we were interested in the growth of the population from an initial reintroduction and whether this growth was sufficient to prevent extinction over 50 years following the introduction. We also did not directly consider the dynamics of Bd in this model. We made this decision because i) all translocated populations are infected with Bd at high prevalence, and ii) host density does not seem to play a significant role in multi-year Bd infection dynamics in this system [@wilber2022]. Thus, ignoring Bd infection dynamics and instead assuming all host vital rates are in the presence of high Bd prevalence significantly simplifies the model without much loss of realism.

##### Model analysis

###### Incorporating yearly variability in vital rates

We extracted yearly survival probabilities for translocated adults $\sigma_{A_T}$ and naturally recruited adults $\sigma_{A_R}$ from the CMR model used to estimate population-level parameters of translocated populations (see "Estimation of adult survival and population size" above). Although we observed yearly variability in adult survival *within* a population, the magnitude of this variability was small compared to *among* population variability. Thus, we did not include yearly within-population variability in adult survival in this analysis.

However, within a population there was substantial yearly variability in the successful recruitment of adults, greater than what we would expect from Poisson variability about a mean value. Therefore, we allowed for yearly variability in the probability of successful recruitment $\omega$ (additional details provided in "Estimating model parameters" below).

###### Estimating model parameters

The baseline parameter values for the model and how they were estimated are given in Table 1. Parameters $\sigma_{A_T}$ and $\sigma_{A_R}$ were extracted directly from our CMR model described previously in this study. For populations where we had a sufficient number of pit-tagged, naturally recruited adults, we observed that $\sigma_{A_T}$ and $\sigma_{A_R}$ could be notably different, with $\sigma_{A_R} > \sigma_{A_T}$. For populations lacking any naturally-recruited adults, we were unable to directly estimate $\sigma_{A_R}$, and instead set $\sigma_{A_R} = \sigma_{A_T}$.

Table 1: Description and values of parameters used in the model. All survival probabilities are in the presence of the fungal pathogen Bd. <!--# Roland, please add any additional details needed to the "Source" column.  We ideally want to keep this succinct.  -->

| **Parameter**                                                             | **Value**            | **Source**                                                         |
|:---------------------------|:-----------------|:-------------------------|
| $\sigma_{L_1}$, Yearly survival probability of year 1 tadpoles            | 0.7                  | Estimated from field data, observations, natural history knowledge |
| $\sigma_{L_2}$, Yearly survival probability of year 2 tadpoles            | 0.7                  | Estimated from field data, observations, natural history knowledge |
| $\sigma_{L_3}$, Yearly survival probability of year 3 tadpoles            | 0.7                  | Estimated from field data, observations, natural history knowledge |
| $\sigma_{J_1}$, Yearly survival probability of year 1 juveniles           | 0.25                 | Estimated from field data, observations, natural history knowledge |
| $\sigma_{J_2}$, Yearly survival probability of year 2 juveniles           | 0.5                  | Estimated from field data, observations, natural history knowledge |
| $\sigma_{A_R}$, Yearly survival probability of naturally recruited adults | Varies by population | Estimated from mark-recapture studies                              |
| $\sigma_{A_T}$, Yearly survival probability of translocated adults        | Varies by population | Estimated from mark-recapture studies                              |
| $p_{L_1}$, Probability of a year 1 tadpoles remaining as a tadpoles       | 1                    | Estimated from field data, observations, natural history knowledge |
| $p_{L_2}$, Probability of a year 2 tadpoles remaining as a tadpoles       | 0.25                 | Estimated from field data, observations, natural history knowledge |
| $p_{J_1}$, Probability of a year 1 juvenile remaining as a juvenile       | 0.25                 | Estimated from field data, observations, natural history knowledge |
| $p_F$, Probability of a adult female reproducing in a year                | 0.5                  | Could be as high at 1, based on field observations                 |
| $F$, Number of surviving eggs produced by an adult female                 | 100                  | From observations of captive frogs                                 |
| $\omega$, Probability of juvenile successfully recruiting to an adult     | Varies yearly        | Explored different values of this parameter                        |

###### Model analysis and simulation

We performed four analyses on our model. First, we considered a deterministic version of our model with no yearly heterogeneity in probability of successful recruitment $\omega$, and calculated the predicted long-run growth rate $\lambda$ of a population for different values of $\sigma_{A_R}$ and $\omega$. We then fixed $\omega = 0.3$ and calculated the predicted growth rate of our 12 populations.

Second, we performed a local elasticity analysis on $\lambda$ with respect to parameters $\omega$, $\sigma_{A_R}$, and $F$ to determine how small changes in these parameters could influence the long-run deterministic growth rate of populations.

Third, we defined a version of the model with demographic and environmental stochasticity, where environmental stochasticity was represented by among-year variability in $\omega$. We used this model to simulate a one-time introduction of 40 translocated adult frogs. We ran this simulation 1000 times for each population and computed the probability of a population becoming extinct after 50 years given the observed parameter values and environmental stochasticity in $\omega$. We varied the mean recruitment probability $\omega$ from 0.15 and 0.4 and drew values of $\omega$ each year from a beta distribution with a dispersion parameter of $\phi = 2$ (when $\omega = 0.5$ and $\phi = 2$ the beta distribution is uniform between 0 and 1).

Finally, we assessed whether our stochastic model could reproduce observed trajectories of population recovery. We focused on population 70550 because this was our longest CMR time series for a translocated population and because this population shows evidence of substantial post-translocation increases in adult abundance associated with population establishment and recovery. We simulated our model for 16 years, repeating the simulation 50,000 times. For each run and each year, we drew $\omega$ from a uniform distribution between 0 and 1 (or a beta distribution with mean 0.5 and $\phi = 2$). Using Approximate Bayesian Computing and rejection sampling (@kosmala2016), we identified the top 2% of trajectories (i.e., 1000 trajectories) that minimized the sum of squared errors between the observed and predicted data. The $\omega$ values associated with these "best" trajectories represented an approximate posterior distribution [@beaumont2010]. Using these best fit trajectories, we assessed whether our model could qualitatively describe the patterns of recovery in the observed data for population 70550.

### Data availability

### References

<!--# References for Methods section only -->

### Acknowledgements

### Author contributions

### Competing interests

The authors declare no competing interests.

### Additional information

Supplementary Information is available for this paper.

Correspondence and requests for materials should be addressed to Roland Knapp.

Peer review information.

Reprints and permissions information is available at www.nature.com/reprints.

### Supplementary Information

### Notes (author use only - not part of manuscript)

Science (impact factor = 63.714)

-   Research articles

    -   Present a major advance.

    -   Up to approximately 4500 words, including references, notes and captions. Corresponds to approximately 5 printed pages in journal (approxiumately 11 manuscript pages).

    -   Up to 6 figures or tables, and about 40 references.

-   Reports

    -   New research results of broad significance.

    -   Up to \~2500 words, including references, notes and captions. Corresponds to approximately 3 printed pages in journal (approximately 6 manuscript pages).

    -   Up to 4 figures or tables, and about 30 references.

Nature (impact factor = 69.8)

-   Articles

    -   Original reports whose conclusions represent a substantial advance in understanding of an important problem and have immediate, far-reaching implications.

    -   Biological sciences papers do not normally exceed 8 pages. This would be approximately 4300 words and 5-6 modest display items.

    -   This [article](https://www.nature.com/articles/s41586-022-04824-9) provides a potentially useful (e.g., recent) example of formatting and structure for a paper focused on ecology/evolution

PNAS (impact factor = 12.77)

-   Research reports

    -   Original research of exceptional importance.

    -   Preferred length is 6 pages, but up to 12 pages is allowed. 6 page article is approximately 4,000 words, 50 references, and 4 medium-size figures or tables.

Nature Communications (impact factor = 17.694)

-   Article

    -   Papers published by the journal represent important advances of significance to specialists within each field. Open access.

    -   The main text (not including abstract, Methods, References and figure legends) is limited to 5,000 words. The maximum title length is 15 words.

Science Advances (impact factor = 14.957)

-   Publishes impactful research papers and reviews in any area of science. Journal extends the capacity of *Science* magazine to identify and promote significant advances in science and engineering. Open access.

-   Research article

    -   Should present a major advance.

    -   Articles can be short of long format, up to 15,000 words, with up to 10 figures and/or tables.

-   
